% Template for PLoS
% Version 3.6 Aug 2022
%
% % % % % % % % % % % % % % % % % % % % % %
%
% -- IMPORTANT NOTE
%
% This template contains comments intended 
% to minimize problems and delays during our production 
% process. Please follow the template instructions
% whenever possible.
%
% % % % % % % % % % % % % % % % % % % % % % % 
%
% Once your paper is accepted for publication, 
% PLEASE REMOVE ALL TRACKED CHANGES in this file 
% and leave only the final text of your manuscript. 
% PLOS recommends the use of latexdiff to track changes during review, as this will help to maintain a clean tex file.
% Visit https://www.ctan.org/pkg/latexdiff?lang=en for info or contact us at latex@plos.org.
%
%
% There are no restrictions on package use within the LaTeX files except that no packages listed in the template may be deleted.
%
% Please do not include colors or graphics in the text.
%
% The manuscript LaTeX source should be contained within a single file (do not use \input, \externaldocument, or similar commands).
%
% % % % % % % % % % % % % % % % % % % % % % %
%
% -- FIGURES AND TABLES
%
% Please include tables/figure captions directly after the paragraph where they are first cited in the text.
%
% DO NOT INCLUDE GRAPHICS IN YOUR MANUSCRIPT
% - Figures should be uploaded separately from your manuscript file. 
% - Figures generated using LaTeX should be extracted and removed from the PDF before submission. 
% - Figures containing multiple panels/subfigures must be combined into one image file before submission.
% For figure citations, please use "Fig" instead of "Figure".
% See http://journals.plos.org/plosone/s/figures for PLOS figure guidelines.
%
% Tables should be cell-based and may not contain:
% - spacing/line breaks within cells to alter layout or alignment
% - do not nest tabular environments (no tabular environments within tabular environments)
% - no graphics or colored text (cell background color/shading OK)
% See http://journals.plos.org/plosone/s/tables for table guidelines.
%
% For tables that exceed the width of the text column, use the adjustwidth environment as illustrated in the example table in text below.
%
% % % % % % % % % % % % % % % % % % % % % % % %
%
% -- EQUATIONS, MATH SYMBOLS, SUBSCRIPTS, AND SUPERSCRIPTS
%
% IMPORTANT
% Below are a few tips to help format your equations and other special characters according to our specifications. For more tips to help reduce the possibility of formatting errors during conversion, please see our LaTeX guidelines at http://journals.plos.org/plosone/s/latex
%
% For inline equations, please be sure to include all portions of an equation in the math environment.  For example, x$^2$ is incorrect; this should be formatted as $x^2$ (or $\mathrm{x}^2$ if the romanized font is desired).
%
% Do not include text that is not math in the math environment. For example, CO2 should be written as CO\textsubscript{2} instead of CO$_2$.
%
% Please add line breaks to long display equations when possible in order to fit size of the column. 
%
% For inline equations, please do not include punctuation (commas, etc) within the math environment unless this is part of the equation.
%
% When adding superscript or subscripts outside of brackets/braces, please group using {}.  For example, change "[U(D,E,\gamma)]^2" to "{[U(D,E,\gamma)]}^2". 
%
% Do not use \cal for caligraphic font.  Instead, use \mathcal{}
%
% % % % % % % % % % % % % % % % % % % % % % % % 
%
% Please contact latex@plos.org with any questions.
%
% % % % % % % % % % % % % % % % % % % % % % % %

\documentclass[10pt,letterpaper]{article}
\usepackage{textgreek}

% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}
\usepackage{newunicodechar}
\newunicodechar{≈}{\approx}
\newunicodechar{α}{\alpha}
\newunicodechar{≥}{\geq{}}
\newunicodechar{Δ}{\Delta}
\DeclareUnicodeCharacter{03B2}{\textbeta}
\newunicodechar{₂}{$_2$}
\newunicodechar{ }{~}
\newunicodechar{ }{\,}
\usepackage{siunitx} % optional, for units

% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}

% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{natbib} % for \citep and \citet
% \usepackage{cite} % comment out or remove if using natbib

% Use nameref to cite supporting information files (see Supporting Information section for more info)
\usepackage{nameref,hyperref}

% line numbers
\usepackage[right]{lineno}

% ligatures disabled
\usepackage[nopatch=eqnum]{microtype}
\DisableLigatures[f]{encoding = *, family = * }

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}
\usepackage{enumitem}
\usepackage{booktabs}

% array package and thick rules for tables
\usepackage{array}

% create "+" rule type for thick vertical lines
\newcolumntype{+}{!{\vrule width 2pt}}

% create \thickcline for thick horizontal lines of variable length
\newlength\savedwidth
\newcommand\thickcline[1]{%
  \noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
  \cline{#1}%
  \noalign{\vskip\arrayrulewidth}%
  \noalign{\global\arrayrulewidth\savedwidth}%
}

% \thickhline command for thick horizontal lines that span the table
\newcommand\thickhline{\noalign{\global\savedwidth\arrayrulewidth\global\arrayrulewidth 2pt}%
\hline
\noalign{\global\arrayrulewidth\savedwidth}}


% This is for Supplementary
\usepackage{graphicx}      % already in almost every template
\usepackage{subcaption}    % gives sub-figures + \subref
\usepackage{placeins}      % lets us slam a barrier before the SI
\usepackage[margin=1in]{geometry}

% For table
\usepackage{booktabs}
\usepackage{multirow}
\usepackage{array}
\usepackage{tabularx}
\usepackage{makecell} % in your preamble

% ---------- helper to switch counters to S-numbers ----------
\newcommand{\beginsupplement}{%
  \setcounter{table}{0}%
  \renewcommand{\thetable}{S\arabic{table}}%
  \setcounter{figure}{0}%
  \renewcommand{\thefigure}{S\arabic{figure}}}

% 2) Define a macro \beginsupplement that:
%    • Resets the figure/table counters
%    • Prefixes future figures/tables with “S”
\usepackage{etoolbox} % for \pretocmd and \setcounter


% Remove comment for double spacing
%\usepackage{setspace} 
%\doublespacing

% Text layout
\raggedright
\setlength{\parindent}{0.5cm}
\textwidth 5.25in 
\textheight 8.75in

% Bold the 'Figure #' in the caption and separate it from the title/caption with a period
% Captions will be left justified
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\renewcommand{\figurename}{Fig}

% Use bibliography
\usepackage[numbers]{natbib}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother

% links
\usepackage{hyperref}


% Header and Footer with logo
\usepackage{lastpage,fancyhdr,graphicx}
\usepackage{epstopdf}
%\pagestyle{myheadings}
\pagestyle{fancy}
\fancyhf{}
%\setlength{\headheight}{27.023pt}
%\lhead{\includegraphics[width=2.0in]{PLOS-submission.eps}}
\rfoot{\thepage/\pageref{LastPage}}
\renewcommand{\headrulewidth}{0pt}
\renewcommand{\footrule}{\hrule height 2pt \vspace{2mm}}
\fancyheadoffset[L]{2.25in}
\fancyfootoffset[L]{2.25in}
\lfoot{\today}

%% Include all macros below

\newcommand{\lorem}{{\bf LOREM}}
\newcommand{\ipsum}{{\bf IPSUM}}



%% END MACROS SECTION
\usepackage{graphicx}
\usepackage[aboveskip=1pt,labelfont=bf,labelsep=period,justification=raggedright,singlelinecheck=off]{caption}
\usepackage{placeins}

\begin{document}
\vspace*{0.2in}

% Title must be 250 characters or less.
\begin{flushleft}
{\Large
\textbf\newline{Sorghum Lipidomics Database} % Please use "sentence case" for title and headings (capitalize only the first word in a title (or heading), the first word in a subtitle (or subheading), and any proper nouns).
}
\newline
% Insert author names, affiliations and corresponding author email (do not include titles, positions, or degrees).
\\
Nirwan Tandukar\textsuperscript{1,2\Yinyang},
Ruthie Stokes\textsuperscript{3},
Name4 Surname\textsuperscript{2},
Name5 Surname\textsuperscript{2\ddag},
Name6 Surname\textsuperscript{2\ddag},
Rubén Rellán Álvarez\textsuperscript{1,3*},

\bigskip
\textbf{1} Department of Genetics and Genomics, North Carolina State University, Raleigh, NC, USA
\\
\textbf{2} Department of Bioinformatics, North Carolina State University, Raleigh, NC, USA
\\
\textbf{3} Department of Molecular and Structural Biochemistry,  North Carolina State University, Raleigh, NC, USA
\\
\bigskip


% Insert additional author notes using the symbols described below. Insert symbol callouts after author names as necessary.
% 
% Remove or comment out the author notes below if they aren't used.
%
% Primary Equal Contribution Note
\Yinyang These authors contributed equally to this work.

% Additional Equal Contribution Note
% Also use this double-dagger symbol for special authorship notes, such as senior authorship.
\ddag These authors also contributed equally to this work.

% Current address notes
\textcurrency Current Address: Dept/Program/Center, Institution Name, City, State, Country % change symbol to "\textcurrency a" if more than one current address note
% \textcurrency b Insert second current address 
% \textcurrency c Insert third current address



% Group/Consortium Author Note
\textpilcrow Membership list can be found in the Acknowledgments section.

% Use the asterisk to denote corresponding authorship and provide email address in note below.
* correspondingauthor@institute.edu

\end{flushleft}
% Please keep the abstract below 300 words
\section*{Abstract}
SAP lines



\linenumbers
% Use "Eq" instead of "Equation" for equation citations.

\section*{Introduction}

\subsection*{Lipid remodelling under abiotic constraints}

Plants remodel their membranes in a highly‐orchestrated manner when temperature or nutrient supply is sub‑optimal.  Below we summarise the characteristic fingerprints for \textbf{cold}, \textbf{phosphorus} and \textbf{nitrogen} stress, with emphasis on (i) class ratios that can be used as diagnostic indicators and (ii) individual molecular species that act as markers in lipidomic data sets.

%--------------------------------------------------------------------
\subsubsection*{Cold stress}
\label{sec:cold}

\begin{enumerate}[label=\textbf{\arabic*.}, leftmargin=1.2em]
  \item \textbf{Higher acyl‑chain unsaturation.}  Cold‐tolerant genotypes accumulate poly‑unsaturated fatty acids—principally 18\,:3, 18\,:2 and 18\,:1—leading to a higher double‑bond index (DBI) and preventing membrane rigidification at low temperature \citep[pp.~431–440, 460]{Low_temp_stress_Bhattacharya}.  An increase in DBI is consistently reported in tolerant lines of \textit{Arabidopsis}, maize and peanut \citep[pp.~11–12]{Lipid_transcriptome_Cold_stress_Yu}.

  \item \textbf{Class‑level reshaping.}  
        \begin{itemize}
          \item Poly‑unsaturated PC, PE, PG, MGDG and DGDG species rise, whereas their saturated counterparts decline \citep[pp.~3–4]{Low_temperatures_Wang,Low_temp_stress_Bhattacharya}.  
          \item The bilayer/non‑bilayer ratio, \(\mathrm{(PC+DGDG)/(PE+MGDG)}\), increases, stabilising the lamellar phase of membranes during freezing events \citep[pp.~492–493]{Low_temp_stress_Bhattacharya}.  
          \item Phosphatidic acid (PA) and lysophospholipids (LPC, LPE) surge, reflecting activation of phospholipase D and A, respectively \citep[pp.~456, 472--474]{Low_temp_stress_Bhattacharya}.
        \end{itemize}

  \item \textbf{Species‑level markers.}  In maize, PA\,36:5, PA\,36:6, DAG\,36:5 and DAG\,36:6 are elevated, whereas MGDG\,36:5 and multiple PC species decline \citep[pp.~6–8]{cold_tolerance_maize_Shi}.  Tolerant cultivars show higher TAG and lower DAG/TAG ratios compared with sensitive lines \citep[pp.~11]{Lipid_transcriptome_Cold_stress_Yu}.

  \item \textbf{Lipid signalling.}  PLD- and PLA‑derived PA and lyso‑lipids act as second messengers, triggering cold‐responsive gene networks \citep[pp.~454–456]{Low_temp_stress_Bhattacharya}.

  \item \textbf{Functional outcome.}  Increased unsaturation and altered bilayer propensity maintain a fluid–crystalline phase, securing electron transport and nutrient transport across membranes at low temperature \citep[pp.~463–465]{Low_temp_stress_Bhattacharya}.
\end{enumerate}

%--------------------------------------------------------------------
\subsubsection*{Phosphorus deprivation}
\label{sec:phosphorus}

\begin{enumerate}[label=\textbf{\arabic*.}, leftmargin=1.2em]
  \item \textbf{Phospholipid depletion.}  Major phospholipids (PC, PE, PG, PI, PS, PA) decline sharply as they serve as an internal Pi source; in soybean leaves every phospholipid class decreased under Pi limitation \citep[pp.~1,\,3,\,5]{lipid_remodeling_low_P_Saito}.

  \item \textbf{Compensatory rise of non‑P lipids.}  MGDG, DGDG, SQDG and the diagnostic glucuronosyldiacylglycerol (GlcADG) accumulate to preserve membrane surface area \citep[pp.~3--4]{Phosphate_deficiency_Wang}.  GlcADG can increase up to 14‑fold in soybean \citep{lipid_remodeling_low_P_Saito}.

  \item \textbf{Diagnostic ratio.}  The phospholipid/galactolipid ratio (PL/GL) drops from \(\sim\)0.3 (P‐sufficient) to \(\le 0.05\) under severe P stress in field‐grown camelina \citep[page~4]{Phosphate_deficiency_Wang}.

  \item \textbf{Tissue specificity.}  Older leaves are remodelled first, exporting Pi to developing tissues \citep[pp.~1,\,5]{lipid_remodeling_low_P_Saito}.

  \item \textbf{Enzymatic drivers.}  Phospholipase C/D hydrolyse PC and PE; MGDG/DGDG and SQDG synthases are up‑regulated to supply the replacement lipids \citep[pp.~1–2, 6]{Phosphate_scaracity_Xue}.
\end{enumerate}

%--------------------------------------------------------------------
\subsubsection*{Nitrogen deprivation}
\label{sec:nitrogen}

\begin{enumerate}[label=\textbf{\arabic*.}, leftmargin=1.2em]
  \item \textbf{Chloroplast glycolipids.}  Rapeseed shows an 18 % (leaf) to 35 % (root) reduction in MGDG; DGDG declines by 23 % in roots, resulting in a suppressed \(\mathrm{MGDG/DGDG}\) ratio \citep[pp.~5--9]{nitrogen_deficiency_lipid_Yang}.

  \item \textbf{Phospholipid curtailment.}  PC, PE, PI, PS and PA all decrease markedly, the latter by more than 90 % in both organs \citep{nitrogen_deficiency_lipid_Yang}.

  \item \textbf{Storage lipids.}  TAG remains unchanged in rapeseed but accumulates in mature tea leaves under low N, suggesting carbon re‑allocation from photosynthetic (N‑rich) to storage pools \citep[pp.~6--7]{Nitrogen_fertilizer_Ruan}.

  \item \textbf{Integrated carbon‑nitrogen balance.}  Lower nitrogen leaves a surplus of assimilated carbon; plants divert it into TAG or into highly unsaturated MGDG 36:5/36:6 species observed in tea shoots at high N \citep{Nitrogen_fertilizer_Ruan}.
\end{enumerate}

%--------------------------------------------------------------------
\subsubsection*{Synthesis}

Cold, P and N stress each trigger a distinctive yet overlapping pattern of lipid remodelling:

\begin{itemize}
  \item \textbf{Cold} prioritises \emph{unsaturation} and bilayer‑to‑non‑bilayer balance to maintain fluidity.  
  \item \textbf{Pi starvation} reallocates phosphorus by replacing phospho‑lipids with galacto‑ and sulfo‑lipids, sharply lowering the PL/GL ratio.  
  \item \textbf{N starvation} down‑regulates chloroplast glycolipids and phospholipids, sometimes storing excess carbon as TAG.  
\end{itemize}

These shifts are mirrored in our sorghum data: unsaturation indices rise under early low‑temperature planting; the \(\mathrm{DGDG/MGDG}\) and \(\mathrm{SQDG/PG}\) ratios increase under P‑limited, low‑input conditions; and TAG/PC as well as \(\mathrm{TG/DG}\) ratios escalate when available nitrogen is low (see Sections \ref{sec:cold}, \ref{sec:phosphorus} and \ref{sec:nitrogen}).

- Stress in plants specifically in sorghum

- Cold stress

- low Nitrogen

- low Phosphorus

- Relate to climate change?



%---------------------------------------------------------------
\begin{table}[ht]
\centering
\small
\setlength{\tabcolsep}{6pt}
\renewcommand{\arraystretch}{1.15}
\begin{tabular}{@{}p{2.3cm} p{4.2cm} p{1.3cm} p{4.5cm} p{1.7cm}@{}}
\toprule
\textbf{Stress} & \textbf{Key lipid class / molecular species} & \textbf{Direction\textsuperscript{a}} & \textbf{Diagnostic (ratio) or remark} & \textbf{Ref.} \\
\midrule
\multirow{6}{*}{\textbf{Cold}} 
 & Poly‑unsaturated FA (18:3, 18:2, 18:1)            & $\uparrow$ & Higher double‑bond index (DBI)                                & \citet{Low_temp_stress_Bhattacharya} \\
 & Unsat.\ PC, PE, PG, MGDG, DGDG                     & $\uparrow$ & Bilayer lipids enriched                                        & \citet{Low_temperatures_Wang}        \\
 & PA (incl.\ PA\,36:5;\,36:6)                        & $\uparrow$ & PLD activation; signalling                                     & \citet{cold_tolerance_maize_Shi}     \\
 & LPC, LPE                                           & $\uparrow$ & PLA activity                                                   & \citet{Low_temp_stress_Bhattacharya} \\
 & DAG\,36:5;\,36:6                                   & $\uparrow$ & Mobilisation of PC unsat.\ chains                             & \citet{cold_tolerance_maize_Shi}     \\
 & TAG (total)                                        & $\uparrow$ & \textit{cf.}\ DAG/TAG $\downarrow$ in tolerant lines           & \citet{Lipid_transcriptome_Cold_stress_Yu} \\
 \cmidrule{2-5}
 & \multicolumn{2}{@{}l}{\textit{Cold ratios}}       & (PC\,+\,DGDG)/(PE\,+\,MGDG)\:$\uparrow$; \ DAG/TAG\:$\downarrow$ & \citet{Low_temp_stress_Bhattacharya} \\
\midrule
\multirow{5}{*}{\textbf{P deficiency}} 
 & PC, PE, PG, PI, PS, PA                             & $\downarrow$ & Release of Pi pool                                            & \citet{lipid_remodeling_low_P_Saito} \\
 & MGDG, DGDG                                          & $\uparrow$  & Galacto‑lipid replacement                                     & \citet{Phosphate_deficiency_Wang}    \\
 & SQDG                                               & $\uparrow$  & Sulfo‑lipid substitution                                      & \citet{Phosphate_deficiency_Wang}    \\
 & GlcADG                                             & $\uparrow$  & Pi‑stress biomarker (14‑fold)                                 & \citet{lipid_remodeling_low_P_Saito} \\
 & \multicolumn{2}{@{}l}{\textit{P ratios}}           & PL/GL $\downarrow$ (to $\le$ 0.05); DGDG/MGDG $\uparrow$        & \citet{Phosphate_deficiency_Wang}    \\
\midrule
\multirow{5}{*}{\textbf{N deficiency}} 
 & MGDG (leaf, root)                                  & $\downarrow$ & 18–35 \% reduction                                            & \citet{nitrogen_deficiency_lipid_Yang} \\
 & DGDG (root)                                        & $\downarrow$ & 24 \% reduction                                               & \citet{nitrogen_deficiency_lipid_Yang} \\
 & PC, PE, PI, PS, PA                                 & $\downarrow$ & PA $\downarrow$ > 90 \%                                       & \citet{nitrogen_deficiency_lipid_Yang} \\
 & TAG (mature tea leaves)                            & $\uparrow$  & Carbon sink under low N                                       & \citet{Nitrogen_fertilizer_Ruan}      \\
 & \multicolumn{2}{@{}l}{\textit{N ratios}}           & MGDG/DGDG $\downarrow$; TAG/PC $\uparrow$; TG/DG $\uparrow$     & \citet{nitrogen_deficiency_lipid_Yang} \\
\bottomrule
\multicolumn{5}{l}{\footnotesize \textsuperscript{a}\,$\uparrow$ increase, $\downarrow$ decrease relative to control or sufficient nutrient.}
\end{tabular}
\caption{Core lipid markers and class ratios characterising cold, phosphorus and nitrogen stress as distilled from the literature survey.  Arrows indicate the direction of change in stressed tissues.}
\label{tab:lipid_markers}
\end{table}
%---------------------------------------------------------------



\section*{Materials and methods}

\subsection*{Plant Material and Growth Conditions}
In our study, we used the Sorghum Association Panel (SAP), consisting of 400 accessions designed to cover extensive genetic and phenotypic diversity. This collection includes both temperate-adapted breeding lines and tropical landraces. The panel represents five botanical races, bicolor, caudatum, durra, guinea, and kafir, capturing a diverse range of domestication and adaptation processes.

SAP was first genotyped using simple sequence repeat markers, followed by low-coverage genotyping by sequencing (GBS). For a more comprehensive variation set, Boatwright et al. (ref) resequenced all entries using whole genome sequencing (WGS) with an average depth of 38× (ranging from 25–72×). The variant data from WGS revealed approximately 43.98 million polymorphisms, including roughly 38 million SNPs with 5 million small insertions/deletions. While GBS variants were predominantly located in genic regions, the WGS data were more evenly distributed across genic and intergenic regions. Genome-wide linkage disequilibrium is approximately 20 kb, although there were deviations specific to each chromosome. The consequent high-density variant map establishes the resequenced SAP as a valuable tool for examining diversity and conducting genome wide association studies (GWAS).

We evaluated SAP accessions across two different field settings during two growing seasons (2019 and 2022) at the Pee Dee Research and Education Center, Clemson University, Florence, South Carolina. The "control" condition, herein denoted as C, involved standard agronomic inputs with sufficient levels of nitrogen (N) and phosphorus (P) along with a typical planting schedule. In contrast, the "low input" condition, herein denoted as LI,  featured reduced N and P coupled with earlier planting to mimic a cold stress environment.

\subsection*{Lipidomics Analysis}

\subsubsection*{Sample Preparation and Extraction}
Samples were prepared following standard extraction protocols explained in hpc1 paper (ref).


\subsubsection*{Liquid Chromatography-Mass Spectrometry (LC-MS) Analysis}
Lipid extracts were subjected to high-resolution mass spectrometry employing both positive and negative ionization modes to achieve comprehensive lipid coverage. Chromatographic separation was executed utilizing either reversed-phase columns, such as C18, or hydrophilic interaction chromatography (HILIC) columns, contingent upon the polarity of the lipids. In the case of C18 methodologies, the gradient elution commenced at 1 minute and concluded at 8 minutes, succeeded by an isocratic elution phase from 8 to 9.5 minutes. Data preceding 1 minute and subsequent to 9.5 minutes were omitted to eliminate solvent front and late eluting artifacts. For HILIC methodologies, the gradient initiation occurred at 1 minute, terminating at 16.25 minutes, and was followed by an isocratic elution extending until 18.5 minutes; data before 1 minute and after 18.5 minutes were excluded in a similar manner.

\subsubsection*{Feature Detection and Data Processing}
Raw LC-MS data, in both positive and negative ion modes, were processed utilizing MZmine 2, an open-source software for the analysis of mass spectrometry data (ref). The pipeline encompassed peak detection, chromatogram construction, deconvolution, and isotope filtering, producing a detailed feature table containing mass-to-charge ratio-retention time (mz-rt) pairs. Isotopic peaks were excluded to minimize redundancy, as a single metabolite can yield multiple co-eluting ions, such as adducts and in-source fragments. Therefore, mz-rt duplicates were handled with care, with potential de-adducting considered via MS-FLO when appropriate. We acknowledge that such degeneracy can lead to an inflated number of features compared to the actual number of metabolites present which is we considered during metabolite identification. 


\subsubsection*{Blank/extraction-control filtering, intensity thresholds, and sparsity pruning}
To reduce the background and carryover effects, an extraction control filter was implemented at the feature level. For each feature, the maximum intensity was determined across extraction controls (\(a\)) and biological samples (\(b\)). Features for which \(b < 10\times a\) were eliminated. To preclude the exclusion of borderline yet potentially biological signals, a feature was retained should at least one biological sample exceed the extraction control maximum. Furthermore, a minimum average intensity threshold within the treatment groups of interest (\(\sim 10^{6}\) peak height) was imposed to ensure that downstream analyses would emphasize robust signals. At the sample level, any sample exhibiting \(\geq 70\%\) features as zero (or missing) was excluded prior to normalization and statistical analysis. This pruning of sparsity is essential to prevent unstable scaling and spurious differential signals caused by ultra-sparse profiles. 


\subsubsection*{MS/MS spectral library matching and cross-referencing of IDs}
For each feature analyzed by MS/MS, the most intense fragmentation spectrum was queried against the GNPS database. Library matches resulted in putative identifications (levels 2/3), potentially including isomers or near-mass analogs. Features without direct matches were eliminated. To enable quantification with identifications, tables were linked using the feature \emph{row ID} from the MZmine peak list and the corresponding \#Scan\# key in the GNPS results, ensuring a one-to-one correspondence between intensities and candidate identifications. In cases where a feature yielded multiple GNPS hits, a single primary annotation was designated by retaining the highest MQScore (cosine similarity). Ties in the values were resolved based on a greater number of shared fragment ions and a smaller precursor mass error (ppm). All other sub-threshold or lower-ranked candidates were retained for verification but were excluded from subsequent statistical analyses.

\subsubsection*{Systematic Error Removal Using Random Forest (SERRF)}
Following the cleaning process, the data were then used for SERRF normalization (ref). We used the SERRF server (https://slfan2013.github.io/SERRF-online/\#) to obtain the normalized output. After applying SERRF, only biological samples were preserved. Any zeros were substituted with two-thirds of the minimum nonzero value for that feature to prevent potential infinite logarithmic transformations.

\subsubsection*{Spatial Correction}
Finally, we conducted an additional quality control step specifically aimed at eliminating any spatial patterns across our experimental trials. This was achieved using the R package \texttt{SpATS} (ref), which applies a two-dimensional P-spline ANOVA surface over the field coordinates. For every lipid feature, we characterized its intensity as
\begin{align}
  y_{ij} &= \mu + f_{\mathrm{row}}(i) + f_{\mathrm{col}}(j) + f_{\mathrm{row,col}}(i,j) + \varepsilon_{ij},
\end{align}

where 
\begin{itemize}
  \item \(f_{\mathrm{row}}\) and \(f_{\mathrm{col}}\) represent smooth functions that model systematic effects across rows and columns, respectively, 
  \item \(f_{\mathrm{row,col}}\) is a smooth interaction surface that handles more complex spatial gradients. 
}  

We utilized the residuals, defined as the difference between observed intensity and the fitted spatial trend, as our final intensity data. This methodology effectively corrects for positional artifacts, such as edge effects, that could interfere with subsequent analyses. Detailed smoothing parameters, including the number of knots, penalty orders, and comprehensive model specifications, can be found in our GitHub repository at \texttt{scripts/spats\_qc.R}.


\subsubsection*{Lipid Quantification}
The identified lipid species were organized into lipid classes and subclasses (see Supplementary Table~1) based on Lipid Maps (ref). In each sample, the total intensity for a class was obtained by summing the intensities of the species within that class. To manage variability in signal intensity due to different runs or injections, these class totals were normalized relative to the total ion current (TIC) of the sample. As a result, the relative abundances were presented as percentages of the TIC by adding up intensities across all lipid classes in the sample. For each class and its subclasses, we determined the TIC fraction for each sample and then averaged these percentages across samples for each condition (Control, $n=384$; LowInput, $n=362$). Lipids were categorized into glycerolipid, glycerophospholipid, sphingolipid, sterol, betaine lipid, fatty acid, ether lipid, and terpenoid. Refer to Supplementary Table 2 for the full list. 

\subsubsection*{Lipid Ratio Identification}
To determine the key lipid ratios most significantly influenced by the shift from C to LI, we utilized the cumulative class-level abundances of lipids and calculated all possible pairwise ratios. These calculations were then analyzed through orthogonal partial least squares–discriminant analysis (OPLS-DA) employing the \emph{ropls} package in R (v3.3.2). To identify the most distinguishing ratios, we analyzed the Variable Importance in Projection (VIP) scores generated by OPLS-DA. A threshold of 1 for VIP was used. These high-ranking ratios enhanced the multivariate differentiation between C and LI samples. The model is explained in detail below. 

\subsubsection*{Lipid Ratio Calculation}

To quantify condition specific shifts between lipid classes, we used  log\textsubscript{10} ratios of class‐level relative abundances. Normalization proceeded as follows:

\begin{enumerate}
\item \textbf{Per‐sample TIC normalization.}  
For each sample, preprocessed peak intensities were summed (total ion current, TIC) and each species intensity was divided by the sample TIC to yield a relative abundance:
\[
\text{Relative Abundance} = \frac{\text{Intensity}}{\text{TIC}}.
\]

\item \textbf{Log\textsubscript{10} transformation with a pseudo‐count.}  
Because many relative abundances are very small or zero, we added half of the smallest nonzero value in that sample ($\varepsilon$) and computed 
\[
\log_{10}(x + \varepsilon),
\]
to stabilize variance.

\item \textbf{Class‐level aggregation (mean log\textsubscript{10}).}  
Species were grouped into lipid classes (e.g., PC, PE, DGDG, MGDG, TG, DG; see Supp.\ Table~2).  
For each sample $i$ and class $c$, we averaged the species‐level logs:
\[
\text{class\_log}_{i,c} = \frac{1}{n_c} \sum_{k \in c} \log_{10}\!\left(\frac{\text{Intensity}_{i,k}}{\text{TIC}_i} + \varepsilon_i\right).
\]
\end{enumerate}

\paragraph{Ratios (log scale).}
Pairwise \emph{log‐ratios} were computed as differences of class logs, e.g.
\[
\text{DGDG/PC} = \text{class\_log}_{\text{DGDG}} - \text{class\_log}_{\text{PC}} 
= \log_{10}\!\left(\frac{\text{DGDG}}{\text{PC}}\right).
\]
Positive values indicate enrichment of the numerator class (LI $>$ C if the LI–C effect is positive), and negative values indicate the reverse.

\subsubsection*{Statistical Tests}
For each ratio, we performed a comparison between LI and C utilizing the two-sample Wilcoxon rank-sum test (Mann–Whitney) on the log ratio values. It was selected for its robustness to unequal group sizes and its ability to handle data with heavy tails without assuming normal distribution. Also, for each ratio, we present the following quantities (SuppTable 5):

\begin{itemize}
\item \texttt{n\_C}, \texttt{n\_LI} = sample sizes in C and LI.
\item \texttt{median\_C}, \texttt{median\_LI} =  group medians of the log‐ratio.
\item \texttt{effect\_log10} = median difference on the log scale, defined as $\text{median}_{LI} - \text{median}_{C}$. Positive means the ratio is higher in LI, negative means the ratio is higher in C. 
\item \texttt{effect\_fc} = fold‐change corresponding to \texttt{effect\_log10}, computed as $10^{\text{effect\_log10}}$ (e.g., $+0.70$ implies $\approx 5.0\times$).
\item \texttt{HL\_low}, \texttt{HL\_high} = 95\% confidence interval for the Hodges–Lehmann (HL) location shift (robust estimate of LI–C difference). If the CI excludes 0, the shift is statistically significant.
\item \texttt{p\_wilcox} = Wilcoxon rank–sum $p$‐value for LI vs.\ C.
\item \texttt{p\_adj\_BH} = Benjamini–Hochberg adjusted $p$‐value (FDR correction across $m=43$ ratios, $\alpha=0.05$).
\item \texttt{AUC\_pct} = probability of superiority (ROC AUC $\times 100\%$), i.e.\ the probability a randomly chosen LI value exceeds a Control value.
\item \texttt{cliffs\_delta} = Cliff’s $\delta$ effect size ($[-1,1]$); $\delta \approx +1$ (or $-1$) indicates nearly complete separation (LI $>$ C or LI $<$ C). 
\item \texttt{jackknife\_stability} = leave‐one‐out sign stability of $\text{median}_{LI} - \text{median}_{C}$ (1.0 means direction invariant to any single sample).
\end{itemize}



%\paragraph{Direction cheat sheet.}
%\[
%\text{effect\_log10} > 0 \;\Rightarrow\; \text{ratio higher in LI (fold‐change } = 10^{\text{effect\_log10}}).
%\]  
%\[
%\delta \approx +1 \;(\text{AUC} \approx 100\%) \;\Rightarrow\; \text{almost all LI $>$ C}; \quad
%\delta \approx -1 \;(\text{AUC} \approx 0\%) \;\Rightarrow\; \text{almost all LI $<$ C}.
%\]


\subsection*{Principal Component Analysis (PCA)}  
We performed three complementary PCA workflows in R using \emph{stats} package (ref).  First, we ran PCA on the individual lipid species abundances.  Second, we summed abundances by lipid class (SuppTable1) (e.g. TG, DG, PC, MGDG, SQDG) and repeated PCA to highlight broader shifts in lipids. Third, we computed key log ratios metrics using OPLS-DA (explained below) and carried out PCA on these as well. In all cases, data were mean–centered and scaled prior to analysis. For each PCA, we retained the first two principal components for visualization. 


\subsection*{Genome-wide Association Studies (GWAS)} 
GWAS analyses were carried out for each lipid trait under each condition using the mixed linear model (MLM) featured in GEMMA (v2.3) (ref). To address population structure and relatedness, a centered relatedness matrix (kinship) was computed from SNP genotype data. For each lipid trait, the MLM was applied using the kinship matrix to handle population stratification effects. Besides individual traits, GWAS was also applied to summed lipid classes and all possible ratios (refer to Supplementary Table 1), and the first two PCs of the summed classes. GWAS analysis was conducted on the first two PCs for each class. A significance threshold of \(-\log_{10}(p) \geq 7\) was employed in order to account for multiple comparisons. 
%In contrast, a more lenient threshold of \(-\log_{10}(p) \geq 5\) was applied to all other analyses.

\subsection*{Gene Annotation}
SNPs were aligned with the Sorghum bicolor reference genome v3.1 (BTx623). For each marker, a 50 kb segment was designated, spanning 25 kb on either side, and all gene models within this area were retrieved. Functional annotations and homology were obtained from Phytozome (https://phytozome.jgi.doe.gov), SorghumBase (https://sorghumbase.com), and TAIR for corresponding Arabidopsis thaliana orthologs. Genes with known roles in N, P, cold tolerance, or lipid metabolism were specifically noted. We aggregated the frequency of each candidate gene within all lipid GWAS findings and marked those with the highest recurrence showing -log10(p-values) of 7 or greater.

\subsection*{Orthogonal Projections to Latent Structures–Discriminant Analysis (OPLS–DA)}

OPLS–DA was employed to identify the lipid class ratios that most effectively distinguish between C and LI while reducing unrelated variance. The analysis was confined to ratios from glycero- and glycerophospholipid classes, specifically TG, DG, MG, DGDG, MGDG, PC, LPC, PE, LPE, PA, PS, and PG. All possible pairwise ratios between the mean log relative abundances of classes were computed as explained above. OPLS–DA was conducted using the \texttt{ropls} R package (v1.34.0), wherein the ratio matrix was denoted as $\mathbf{X}$ and the response $Y$ was coded as lipid class. A single predictive component (\texttt{predI = 1}) was defined, whereas the number of orthogonal components was determined through cross-validation (\texttt{orthoI = NA}). The decomposition is:
\[
  \mathbf{X} = T_{p} P_{p}^{T} \;+\; T_{o} P_{o}^{T} \;+\; E,
\]
where,
\begin{itemize}
  \item \(T_{p}\) and \(P_{p}\) are the predictive score and loading matrices capturing variation correlated with \(Y\),
  \item \(T_{o}\) and \(P_{o}\) are the orthogonal score and loading matrices capturing structured variation orthogonal to \(Y\),
  \item \(E\) is the residual matrix representing unexplained variation.
\end{itemize}

To address the issue of overfitting and adjust for differing sample sizes (C: $n=394$, LI: $n=363$), we employed a stratified seven-fold cross-validation approach with folds that are balanced to estimate $R^{2}_{Y}$ (the variance in $Y$ explained) and $Q^{2}$ (cross-validated predictivity). The significance of the model was evaluated using 500 label permutations. One-sided exact $p$-values were derived as $(\#\{\text{perm} \geq \text{obs}\} + 1)/(N_{\text{perm}} + 1)$. Furthermore, we present $R^{2}_{X}$ (the fraction of $X$ variance captured on the predictive axis) to enhance the interpretability of the score plots.

Ratios that discriminate between C and LI were prioritized based on Variable Importance in Projection (VIP) scores, with VIP $> 1$ as the threshold. Since VIP signifies contribution rather than direction, the effect size direction was separately summarized through the calculation of group medians of the log ratios. 

\subsubsection*{Lipid Metabolic Network Analysis (LINEX2)}

We used the Lipid Network Explorer (LINEX2; \url{https://exbio.wzw.tum.de/linex/}) to reconstruct lipid networks and to identify condition enriched subgraphs for C vs. LI, considering both the ratio and difference between them. Lipid names were standardized to align with species notation (class plus acyl composition). LINEX2 constructs a global species network based on curated reaction rules, including headgroup interconversions, (de)acylation/editing, elongation, desaturation, and overlays a quantitative association structure (Spearman correlations across samples) onto reaction edges. For enrichment, LINEX2 calculates substrate–product change scores for each reaction reagrading the C vs. LI ratio and difference, employing a greedy local-search procedure to identify subgraphs that optimize the average substrate and product change. Since, \textit{Sorghum bicolor} is not implemented as a default organism in LINEX2, we selected \textit{Oryza sativa} (OSA) as the reference species for network construction.


\subsubsection*{Lipid Ontology Enrichment and Hierarchical Classification (LION/web)}
We conducted functional ontology for lipids using LION/web (Molenaar \emph{et al.}, 2019; \url{http://www.lipidontology.com}). Lipid nomenclature was standardized according to LIPID MAPS annotations. The default settings of LION/web were applied for enrichment statistics and multiple-testing correction. We retained terms for $q \leq 0.10$. A hierarchical classification analysis of individual lipids and functional ontologies was also performed using the LION/web.


\subsection*{Random-forest modeling and TreeSHAP interpretation}
\subsubsection*{Lipid data processing}
To achieve variance stabilization and address zero values, each lipid intensity underwent transformation as outlined in OLPS-DA section. Following this transformation, lipid columns were median-centered across samples through feature-wise subtraction of the column median, effectively removing global offsets while maintaining inter-sample variability.

\subsubsection*{Phenotype and population structure covariates}
The phenotypic traits for SAP include plant height (PT) and days to anthesis (FT) (Supp Table 10), which served as the response variables. To reduce the likelihood of the model highlighting population structure over biological factors, both response variables and lipid predictors underwent adjustment through residualization with respect to the principal components, employing the ordinary least squares (OLS) method.

Let \(y\) denote the phenotype of the SAP and \(X_{\mathrm{PC}}\) the PC design matrix (with intercept). We computed the residual phenotype as
\[
rFT \;=\; y - \hat{y}, 
\qquad 
\hat{y} \;=\; X_{\mathrm{PC}}\,(X_{\mathrm{PC}}^{\top}X_{\mathrm{PC}})^{-1}X_{\mathrm{PC}}^{\top}y,
\]
i.e.\ the residuals from the regression \(y \sim X_{\mathrm{PC}}\).
For the lipid matrix \(L\) (samples \(\times\) features), we removed PC effects feature-wise via the same projection:
\[
L_{\mathrm{adj}} \;=\; L \;-\; P L,
\qquad
P \;=\; X_{\mathrm{PC}}\,(X_{\mathrm{PC}}^{\top}X_{\mathrm{PC}})^{-1}X_{\mathrm{PC}}^{\top}.
\]
The adjusted lipid matrix \(L_{\mathrm{adj}}\) and the residual phenotype \(rFT\) were used for all downstream modeling.

\subsubsection*{Train/test split stratified by genetic background}
In order to maintain a balance of population structure across the splits, we employed \(k\)-means clustering in principal component (PC) space, as referenced in \(k=6\). An 80/20 train/test split was subsequently carried out. To ensure the reproducibility of the findings, random seeds were fixed.

\subsubsection*{Random forest tuning, cross-validation, and training}
We employed a random forest model (\texttt{ranger} in R) to represent \(rFT\) as a function of the adjusted lipid features. The hyperparameters were optimized using \texttt{tuneRanger}, with 1,000 trees and a search conducted over \texttt{mtry}, \texttt{min.node.size}, \texttt{sample.fraction}, to minimize the root-mean-squared error (RMSE) within the training dataset. To assess the expected generalization performance on the training data, we implemented a 5-fold cross-validation using \texttt{mlr} with the optimized parameters and reported the fold-wise RMSE, mean absolute error (MAE), and \(R^2\).

A conclusive forest model (ranger; 1,000 trees) was constructed using the complete training dataset in conjunction with the optimized parameters. Due to the potential issue where excessively large \texttt{min.node.size} values may lead to over-smoothing of trees, resulting in predictions that are nearly constant. Thus, we imposed a practical limitation that if the optimized \texttt{min.node.size} exceeded 15, it was adjusted to 15 for the final construction, while all other optimized values remained unchanged.

\subsubsection*{Model evaluation metrics}
We evaluated the final model once on the test set. The following metrics were computed:
\[
\mathrm{RMSE} \;=\; \sqrt{\frac{1}{n}\sum_{i=1}^{n}\bigl(\hat{y}_i - y_i\bigr)^2}, 
\qquad
\mathrm{MAE} \;=\; \frac{1}{n}\sum_{i=1}^{n}\bigl|\hat{y}_i - y_i\bigr|,
\]
\[
r \;=\; \mathrm{cor}(\hat{\bm{y}}, \bm{y}), 
\qquad
R^2 \;=\; r^2,
\qquad
\mathrm{Bias} \;=\; \frac{1}{n}\sum_{i=1}^{n}\bigl(\hat{y}_i - y_i\bigr),
\]
\[
\mathrm{NRMSE}~(\%) \;=\; \frac{\mathrm{RMSE}}{\mathrm{SD}(rFT)} \times 100\%.
\]

\subsubsection*{TreeSHAP computation and feature ranking}
Exact TreeSHAP values were calculated for the fitted forest using \texttt{treeshap} in R to achieve local attributions for each lipid. The trained model was transformed into a unified tree representation , wherein TreeSHAP was executed on the identical feature frame utilized during the training phase. This procedure results in an \(n \times p\) matrix of SHAP values \(\phi_{ij}\), representing the impact of lipid \(j\) on sample \(i\). The global importance associated with lipid \(j\) was encapsulated as the mean absolute SHAP.\[
\overline{|\phi|}_j \;=\; \frac{1}{n}\sum_{i=1}^{n} \bigl|\phi_{ij}\bigr|,
\]
and features were ranked by \(\overline{|\phi|}_j\) in descending order.




\subsection*{Data Availability}
Data processing and statistical analyzes were performed in R (version 4.3.3) using. All the codes, figures, and pipeline are described in the GitHub repository: github.com/nirwan1265/SoLD\_paper.

% Results and Discussion can be combined.
\section*{Results}

\subsection*{Quality Control and Signal Normalization of Lipidomics Data}
Instrument performance was measured using total ion current (TIC) traces throughout the injection sequence for batches C (Supplementary Fig. \ref{fig:S1}A) and LI (Supplementary Fig. \ref{fig:S1}B). As expected, the blanks (grey) remain near zero TIC, the internal standards (green) cluster tightly around their nominal signal, and the quality controls (red) track reproducibly throughout the run. The injections of samples (blue) exhibit the highest TIC with no isolated outliers or sudden jumps. These profiles confirm that instrument performance was stable over time, with consistent sensitivity and no evidence of progressive or unexpected signal change.

After SERRF, in sets C and LI, the relative standard deviation (RSD) of the sample  for all raw lipid characteristics decreased from 3. 86 \% and 1. 18 \%  to 0. 94 \% and 0. 51 \%, respectively,  (Supplementary Figs. \ref{fig:S2}A and \ref{fig:S2}B, top panel). This reduction in technical variability demonstrates that SERRF effectively removes batch-related effects, producing more consistent peak areas across injections. Similarly, PCA of the signal pre- and post-SERRF shows that the points cluster tightly, consistent with a reduction of technical variance and retention of biological structure(Supplementary Figs. \ref{fig:S2}A and \ref{fig:S2}B, bottom panel).

Following SERRF normalization, spatial mixed modeling was implemented with \texttt{SpATS} to address positional effects due to the field layout. For each lipid trait, a smooth two-dimensional spline was adjusted over the row and column positions (\texttt{SAP(col,row)}, degree = 3, $p$-order = 2, $n_{\mathrm{seg}}=(8,2)$), while each genotype was incorporated as a random effect. This approach facilitated the decomposition of the raw signal into broad spatial trends, marginal row/column effects, and genotype effects, resulting in spatially adjusted values (residuals + genotype BLUPs) for subsequent analysis. Supplementary Fig.~\ref{fig:S3} illustrates TG(10:0/10:0/10:0) under C (A,B) and LI (C,D) conditions. Panels A and C depict the modeled 3D spatial surfaces for the raw data, highlighting low-frequency gradients across the plate (more pronounced in the C (A) compared to the LI (C)). Panels B and D showcase the six-figure SpATS diagnostics, which include raw data, model fitted data, residuals, fitted spatial trend, genotype BLUPs, and BLUP histograms. Residuals in the non-corrected data reveal spatial structures. However, post-modeling, they exhibit a random distribution as the fitted spatial trend. The genotypic BLUPs encapsulate biological variance, with histograms corroborating the anticipated centralized distribution around zero which was used for GWAS analysis. After SERRF normalization and spatial correction (SpATS), instrumental drift, batch effects, and field-position artifacts were minimized leading to less technical and spatial error. The example lipid in Supplementary Fig.~\ref{fig:S3} illustrates the typical pattern of strong raw gradient, smooth corrected surface, and spatially clean residuals. Finally, we propagated the spatially adjusted values (residuals $+$ genotype BLUPs) as the input for downstream analyses including univariate comparisons, PCA/OPLS‐DA, VIP‐ranked ratio screens, and GWAS. 

\subsection*{Overview of lipid count}
The enumeration of various lipid species and their respective classes identified under C and LI conditions is presented in Supplementary Figures \ref{fig:S4}, \ref{fig:S5}, and \href{https://docs.google.com/spreadsheets/d/1SB90-QLYheKEzmHCUIh1UfgkrtbL064s8Oo5BfwFaV0/edit?gid=0#gid=0}{Supplementary Table S2 and Supplementary Table S3}. TG shows the greatest diversity in species, with 71 distinct species identified in C conditions compared to 69 under LI conditions. MG reveals a significant reduction, decreasing from eight species in C to five in LI conditions. PC, SQDG, and DG each experience a reduction of exactly one species under LI. In contrast, PE exhibits an increase, with 15 species identified under LI conditions as opposed to 14 in Control conditions.

Upon aggregating by class Supplementary Figures \ref{fig:S4}, \ref{fig:S5}, and \href{https://docs.google.com/spreadsheets/d/1SB90-QLYheKEzmHCUIh1UfgkrtbL064s8Oo5BfwFaV0/edit?gid=675277745#gid=675277745}{Supplementary Table S2}, the overall composition of the lipidome remained largely stable, despite variations in lipid species. The C condition yields 226 species, whereas the LI  yields 224. Glycerolipids constitute the largest class, comprising 120 species in the C and 114 in the LI , followed by glycerophospholipids, with 60 and 64 species, respectively. All other classes, terpenoids (12 and 14), fatty acids (7 and 11), sphingolipids (9 and 5), ether lipids (7 each), prenols (4 each), sterols (2 and 3), and betaine lipids (1 for LI), collectively represent only a minor portion of the species (Supplementary Fig \ref{fig:S4}, \href{https://docs.google.com/spreadsheets/d/1SB90-QLYheKEzmHCUIh1UfgkrtbL064s8Oo5BfwFaV0/edit?gid=675277745#gid=675277745}{Supplementary Table S3}). These observations indicate that, notwithstanding significant environmental stresses in the form of N, P, and cold, the LI workflow maintains nearly the entire range and balance of lipidome species.

\subsection*{Overview of individual lipid species}
For each lipid species, we computed the mean percentage contribution within each species (the relative abundance of each species divided by the total abundance of that species under a given condition, averaged between samples). \href{https://docs.google.com/spreadsheets/d/1SB90-QLYheKEzmHCUIh1UfgkrtbL064s8Oo5BfwFaV0/edit?gid=675277745#gid=675277745}{Supplementary Table S4} lists the species sorted by their mean percentage contribution under C and LI. In the following, we summarize the dominant species and any moderate compositional changes between the two conditions.

\subsubsection*{Acyl‐ether glycerol (AEG)} 
AEG(o-32:3), AEG(o-34:4), and AEG(o-34:5) are the most prevalent species, together comprising more than 75\% of the overall AEG signal across both conditions. AEG(o-32:3) experiences a significant increase under LI, going from 23.2\% to 35\%, whereas AEG(o-34:5) decreases from 26.9\% to 21.2\%. AEG(o-34:4) remains relatively constant. Minor species like AEG(o-36:6) and AEG(o-30:2) are present at low levels and exhibit minimal changes. In summary, the AEG pool composition undergoes a moderate adjustment under stress, with the significant rise in AEG(o-32:3) being a key factor in this alteration.

\subsubsection*{Diacylglycerol (DG)} 
The dataset reveals a predominant presence of the diacylglycerol (DG) species DG(18:3/18:3), accounting for approximately 78.9 \% of the DG pool within the C and 73.7 \% under LI. The subsequent most prevalent species, DG(16:0/18:2) and DG(12:0/12:0), each comprise about 4–5 \% of the DG signal across both experimental conditions. DG(16:0/18:3) is observed to increase from 2.2 \% in the C condition to 5.2 \%, and DG(18:1/2:0) rises significantly from a minimally detectable 0.4 \% to 6.3 \%. By contrast, the majority of other species of mid-abundance such as DG(18:2/18:2), DG(16:0/18:1), DG(18:1/18:3), DG(18:2/18:3), DG(18:1/18:1), and DG(18:1/18:2) exhibit variations amounting to only a few tenths of a percent across different conditions. Several minor species, detectable during C, namely DG(16:0/16:0) at 1.0 \% and DG(16:0/2:0) and DG(18:0/18:0) at 0.4 \% each, decrease below the detection threshold or are absent under LI. Trace lipids such as DG(16:1/18:2), DG(18:0/20:3), and DG(8:0/8:0) are exclusively detectable in LI  at concentrations of less than 0.1 \%. These findings suggest that despite the DG profile predominantly being defined by its 18:3/18:3 core, the LI condition tends to enrich certain unsaturated and short-chain species.

\subsubsection*{Digalactosyldiacylglycerols (DGDG)} 
Digalactosyldiacylglycerols (DGDG) are predominantly constituted by the species DGDG(18:2/18:4), which comprises 63.9 \% of the DGDG composition within the C, with a minor increase to 65.4 \% under LI. The second most prevalent species, DGDG(16:0/18:3), accounts for 27.6 \% in the C  compared to 24.2 \% in the LI, and is followed by DGDG(18:0/18:3), which exhibits an increase from 4\% to 5.5\%. Minor constituents such as DGDG(16:0/18:2) exhibit significant stability (2.1\% to 2.2\%), whereas DGDG(18:2/18:3) slightly increases from 1\% to 1.4\%. The low abundance species DGDG(16:0/18:1) and DGDG(18:0/18:2) remain near the detection limit, with negligible changes from 0.9\% to 0.8\% and 0.4\% to 0.5\%, respectively. These data indicate that the LI effectively maintains the DGDG composition, with only minor enrichment of fully saturated and monounsaturated species at the periphery.

\subsubsection*{Monogalactosyldiacylglycerol (MGDG)}
The MGDG composition is predominantly characterized by the fully saturated MGDG(18:3/18:3), which slightly increases from 87.3 \% of the monogalactolipids in the C to 89.3 \% under LI. The second most prevalent species, MGDG(16:0/18:3), decreases from 5.5 \% to 4.2 \%, while the di- and tri-unsaturated species MGDG(18:2/18:3) and MGDG(18:2/18:2) experience slight reductions (5 \% to 4.5 \% and 2.1 \% to 2.0 \%, respectively). The minor component, MGDG(16:0/18:1), remains a trace constituent ($<$ 0.1 \%) in both conditions. Thus, the MGDG profile preserves its distinctive polyunsaturated signature, with only a marginal increase in the abundance of the 18:3/18:3 species at the expense of those with moderate abundance.

\subsubsection*{Lysophospholipids (LPC, LPE)}
Lysophospholipids constitute a minor component of the lipidome, dominated by LPC(16:0) and LPC(18:3). Within the C, LPC(16:0) comprises 73 \% of the lysophosphatidylcholines, with LPC(18:3) constituting the remaining 27 \%. Under LI, these proportions are altered to 62.6 \% and 30.5 \%, respectively. An additional presence of LPC(18:2) (6.9 \%) is exclusively identified in the LI, suggesting that LI may reveal trace unsaturated species. Across both conditions, lysophosphatidylethanolamine consists entirely of LPE(16:0), maintaining a constant 100 \% presence within the LPE pool, irrespective of the condition.

\subsubsection*{Monoacylglycerol (MG)} 
Monoacylglycerols are predominantly characterized by the polyunsaturated MG(18:3), which increases from 51.8\% of the MG pool in the C to 58.2\% in LI. MG(18:1) shows an increase from 16.1\% to 18.4\%, and MG(12:0) increases from 12.4\% to 14.6\%, while the saturated MG(16:0) decreases from 8.9\% to 6.9\%. The di-unsaturated MG(18:2) also experiences a slight reduction (3.4\% to 2.0\%). Several minor species detectable in the C,MG(16:1) (1.1\%) and MG(18:0) (6.3\%), alongside trace amounts of MG(20:4) (\textless0.1\%), fall below the detection threshold or are absent under the LI conditions. Overall, shifts in the MG profile towards more highly unsaturated C18 species while reducing the presence of low-abundance saturated and monounsaturated forms are observed in the LI.

\subsubsection*{Phosphatidic acid (PA)} 
PA(34:2) constitutes ~100\% of the PA pool in both conditions, indicating that only one species is reliably detected.

\subsubsection*{Phosphatidylcholine (PC)} 
Phosphatidylcholines (PC) remain dominated by the PC(16:0/18:2) species which drops from 37.9 \% of the CPC pool to 27.7 \% in LI . The overall PC landscape is noticeably reshaped under LI conditions. The saturated PC(16:0/18:0) species rises from 14.6 \% to 19.4 \%, while the mono-unsaturated PC(16:0/18:1) form falls from 8.9 \% to 4.8 \%. The polyunsaturated PC(16:0/18:3) remains roughly constant at ~8–9 \%. Among the C36 lipids, PC(16:0/20:5) declines modestly (7.5 \% to 5.7 \%) and PC(16:0/20:4) collapses (6.9 \% to 0.6 \%), whereas the previously negligible PC(16:1/18:2) species surges from 0.1 \% in C to 7.5 \% in LI, and PC(18:0/18:1) likewise emerges from 0.1 \% to 5.8 \% of the PC signal. A new C38 variant, PC(18:2/20:0), climbs from below detection to 3.1 \%, and a handful of other minor PCs (e.g. PC(16:1/20:4) at 5.3 \%) appear only in the LI run. All other low abundance forms, long-chain di- and tri-unsaturated species, and odd-chain variants—remain at or below 1 \% in both conditions. Thus, while PC(16:0/18:2) continues to dominate, LI selectively unmasks and enriches several minor PC isoforms and shifts the saturated/unsaturated balance across the class.

\subsubsection*{Phosphatidylethanolamine (PE)} 
Under conditions of LI, the phosphatidylethanolamine (PE) pool undergoes a significant transformation, emphasizing species with higher unsaturation levels. Within the C, PE(16:0/18:2) predominates, comprising 50.3\% of the PE pool. However, it diminishes considerably to 20.7\% under LI. PE(16:0/18:3), which is polyunsaturated, escalates from a modest 5.4\% to 30.2\%, consequently becoming one of the most prevalent PEs under LI. In parallel, PE(16:0/20:5) nearly triples its proportion from 3.7\% to 9.5\%, while PE(14:0/22:6) and PE(18:0/18:2), which are either undetectable or minor in the C (at 0.4\% and 1.0\%, respectively), increase significantly to 8.5\% and 7.7\%. In contrast, di-unsaturated species such as PE(16:0/20:4) and PE(18:2/18:2) decrease from 12.1\% to 7.3\% and 8.4\% to 6.4\%, respectively, and the mono-unsaturated PE(16:0/18:1) experiences a drastic reduction from 10.2\% to merely 0.9\%. Minor species including PE(18:3/18:3) and PE(18:2/18:3) display only slight alterations, with several low-abundance forms either emerging or vanishing between analyses. These findings indicate that LI preferentially reveals and enhances the presence of long-chain, polyunsaturated ethanolamines, while concurrently diminishing their mono- and diunsaturated counterparts.

\subsubsection*{Phosphatidylglycerol (PG)} 
Phosphatidylglycerols (PG) undergo a considerable compositional reorganization under LI. In the  C, PG(16:0/18:1) accounts for 60.3\% of the PG pool, yet its proportion decreases to 56.6\% in LI. Similarly, the fully saturated PG(16:0/16:0) experiences a reduction from 33.4\% in the C to merely 17.5\% in LI. PG(16:0/18:0), which is present at 6\% in the C, emerges as a prominent component at 25.9\% under LI conditions. These variations indicate that LI favors the persistence of the unsaturated 34:1 species at the expense of the fully saturated 32:0 form.

\subsubsection*{Phosphatidylglycerol (PS)} 
In C, the partitioning of the Phosphatidylglycerol (PS) pool is characterized by the predominance of two molecular species. PS(18:0/20:4) constitutes the majority at 65.1\%, while PS(18:0/18:2) accounts for the remaining 34.9\%. Upon LI condition, only PS(18:0/18:2) remains detectable, representing 100\% of the PS signal as PS(18:0/20:4) declines below the detection threshold. This transition demonstrates that the LI preferentially retains the PS(18:0/18:2) head-group configuration.

\subsubsection*{Sulfoquinovosyldiacylglycerol (SQDG)} 
Sulfoquinovosyldiacylglycerols (SQDG) experience a substantial compositional transition under LI. In the C, SQDG(16:0/18:3) and SQDG(18:3/18:3) collectively account for 74.6\% of the composition (48.4\% and 26.2\%, respectively). However, in LI, the prevalence of these species declines significantly to 3.3\% and 4.6\%, respectively. Conversely, SQDG(16:1/18:3) increases from a negligible 0.4\% to 59.7\%, while SQDG(16:0/18:1) rises from 0.7\% to 24.4\%. The fully saturated SQDG(16:0/16:0) decreases from 15.6\% to 0.3\%. All other variants of moderate abundance—such as SQDG(18:1/18:3), SQDG(18:2/18:3), and SQDG(18:0/18:3) undergo modest changes to between 0.6\% and 4.1\%, and trace species like SQDG(16:0/14:0) remain below 1\%. Overall, in the LI condition, originally dominant species 16:0/18:3 and 18:3/18:3 replace a new predominance of 16:1/18:3 and enrich the mono-unsaturated 16:0/18:1 species, thereby revealing a substantially different SQDG composition profile.

\subsubsection*{Triacylglycerol (TG)} 
Under LI, the triglyceride (TG) profile experiences substantial reorganization, predominantly focusing on a singular polyunsaturated species, TG(16:1/20:1/20:2), which increases markedly from a mere 0.1\% of the C TG pool to 27.3\% in LI. In contrast, the fully saturated TG(16:0/16:0/16:0) is significantly diminished, approximately halving from 18.4\% to 8.1\%, while the di-unsaturated TG(16:1/16:1/22:5) declines from 20.6\% to 5.3\%. TG(18:2/18:2/22:1), previously undetectable under C conditions, constitutes 15.8\% of the TG signal in the LI. Similarly, the partially unsaturated TG(18:1/18:2/18:3) remains largely stable, registering a minor increase from 5.3\% to 5.9\%. However, all typically mid-abundance forms, TG(16:0/18:2/18:3), TG(18:2/18:2/18:4), TG(16:0/18:1/18:3), and TG(16:0/18:3/18:3) are reduced to near zero. A few minor species, such as TG(18:0/18:2/20:1) (increasing from \textless0.1\% to 3\%) and TG(12:0/12:0/14:0) (decreasing from 1.4\% to 0.6\%), exhibit directional shifts. Thus, the LI condition considerably alters the conventional TG framework, introducing a pronounced peak in the highly unsaturated TG(16:1/20:1/20:2), while also identifying TG(18:2/18:2/22:1) as a prominent new constituent.

\bigskip
In summary, the LI initiates a coordinated reprogramming of the plant lipidome, favoring neutral, energy-storage lipids and highly unsaturated membrane components while simultaneously restructuring the glycerophospholipid profile. TGs, which typically exhibit saturated backbones, are almost entirely supplanted by a novel, polyunsaturated TG(16:1/20:1/20:2) peak (0.1\% to 27.3\%) and the emerging TG(18:2/18:2/22:1) (below detection to 15.8\%), even as TG(16:0/16:0/16:0) reduces its prevalence. DGs remain predominantly composed of DG(18:3/18:3) (78.9\% to 73.7\%) but show enrichment in DG(16:0/18:3) and DG(18:1/2:0) under LI conditions. The MG pool similarly shifts towards its polyunsaturated C18 core (MG(18:3) 51.8\% to 58.2\%), and the galactolipids MGDG and DGDG maintain their highly unsaturated characteristics with only minor subclass variations. SQDG transitions from SQDG(16:0/18:3) and SQDG(18:3/18:3) to a predominant SQDG(16:1/18:3) majority (0.4\% to 59.7\%) with increased SQDG(16:0/18:1). Among lysophospholipids, LPC increasingly favors the saturated LPC(16:0) at the reduction of LPC(18:3), while LPE consistently remains LPE(16:0). Glycerophospholipids experience deliberate remodeling. PC reduces its 16:0/18:2 backbone (37.9\% to 27.7\%) in preference for 16:1/18:2 and 18:0/18:1 variations. PE significantly enriches in PE(16:0/18:3) (5.4\% to 30.2\%) and longer-chain polyunsaturates. PG reveals a novel PG(16:0/18:0) isomer while decreasing PG(16:0/16:0). PS converges to a singular PS(18:0/18:2) species. Thus, these transformations highlight that LI not only preserves but in certain contexts amplifies the core unsaturated and energy-storage lipids, while revealing low-abundance variants and fundamentally redistributing membrane lipid classes.


\subsection*{Lipid Remodeling Revealed by Lipid Class Distribution and Functional Enrichment}
Lowinput (LI) condition induces a reorganization of the lipidome (Fig.\ref{fig:Fig1_lipid_class}; Supplementary Figure \ref{fig:S5}). At the class level, glycerolipids maintain their dominance yet decrease from 64.4\% of TIC in C to 52.3\% in LI. Glycerophospholipids undergo a slight reduction (30.2\% to 25.3\%), whereas sphingolipids significantly increase from a near-baseline of approximately 0.2\% to ~17.3\% (Supplementary Fig. S5A). Minor constituents, including sterols and betaine lipids (Undetectable to 0.2\%), fatty acids (0.7\% to 0.3\%), ether lipids (~1\%), and terpenoids (3.4\% to 3.8\%), together constitute the remaining 5–7\% in both conditions. Neutral storage lipids exhibit expansion (TG ~2.7\% to 5.4\% TIC; DG 13.8\% to 14.8\%), while chloroplast-associated galactolipids show attenuation (MGDG 34.5\% to 32.1\%; SQDG 2.9\% to 1.4\%; DGDG remains relatively stable at 12.7–13.0\%). The apparent reduction in the glycerolipid super-family as depicted in Supplementary Figure \ref{fig:S5}A consequently reflects a redistribution within glycerolipids, characterized by the growth of neutral members (DG/TG) offset by a contraction of the galactolipids.

The analysis of the glycerolipid composition explains the observed redistribution (Supplementary Figure \ref{fig:S5}B). MGDG remains the predominant subclass, accounting for 50.6\% of glycerolipid TIC in the C condition and 47.3\% in the LI. DGDG levels are almost stable (18.8\% to 19.2\%), whereas SQDG shows a decline (4.3\% to 2.0\%). Conversely, DG levels exhibit  a slight increase (20.3\% to 21.9\%) and TG nearly doubles within the subgroup (4.0\% to 7.9\%). MG exhibits a slight reduction (2.0\% to 1.7\%). Overall, there is a shift in the glycerolipid compartment from a profile dominated by galactolipids to one with an increased proportion of neutral lipids.

Within glycerophospholipids, the integrity of the bilayer core is maintained. Phosphatidylcholine (PC) remains predominant, comprising approximately 80\% of the class (79.9\% in C conditions; 76.9\% in LI), while PE exhibits a modest increase (from 18.4\% to 20.8\%), and PG exhibits a slight increase (from 1.3\% to 1.5\%). Lysophospholipids (LPC, LPE) and less common headgroups (PS, PA) persist at levels below or equal to 0.5\% (Supplementary Figure \ref{fig:S5}C). In agreement with Fig.\ref{fig:Fig1_lipid_class}A, the overall share of phospholipids in TIC displays only minor fluctuations (PC from 12.7\% to 13.0\%, PE from 5.8\% to 6.6\%, PG approximately 0.5\%), thereby indicating the preservation of the bulk bilayer despite low P stress.

The findings by TIC lipidomic profile are corroborated using the LION enrichment analysis (Fig \ref{fig:Fig1_lipid_class}B). These lipid transformations manifest as distinct functional motifs. The most pronounced depletions are observed in glycerophospholipids [GP], primarily driven by diminished levels of glycerophosphocholines (PC; GP01 / GP0101) and the category "headgroup with positive charge / zwitter-ion." Several biophysical descriptors are also categorized as down regulated, such as "very low lateral diffusion" and "average bilayer thickness," as well as specific PUFA characteristics (e.g., C20:4), suggesting that the LI state comprises fewer of the most mobile, PC-rich, thin-bilayer species. PE-related descriptors (GP02, diacyl-GPE GP0201) are present among them as well, however, with less prominence and significance than PC, aligning with our ratio/composition analyses where PC exhibits a more consistent decline than PE. [NOTE:so this is not explained till later but makes sense here as well.]

Elevations in LI are also observed. Enrichments are concentrated around neutral glycerolipids, such as glycerolipids [GL], diacylglycerols [GL0201], triacylglycerols [GL0301], and functional categories such as "lipid droplet" and "lipid storage." Biophysical categories include "high / above-average bilayer thickness," which aligns with a transition towards longer and/or more saturated chains that facilitate tighter packing in LI (ref). Thus, these enhanced terms quantitatively reconstruct the TG expansion identified in the class plots (\ref{fig:Fig1_lipid_class}A) (Fig, as well as the reductions in the X/TG ratio (e.g., SQDG/TG, MGDG/TG, PS/TG, PA/TG, LPC/TG)(\ref{fig:S4_lipid_ratio_lowP}). Furthermore, the reduced PC-prevalent signal is consistent with elevated PS-denominated ratios in the Control group (PC/PS, PE/PS, PG/PS, PA/PS↓ in LI), indicating a relative reduction of PC in LI. The UP glycerolipid along with lipid-droplet descriptors reflect the increase in TG composition ($\approx$2.7\% to 5.4\%; subclass $\approx$4.0\%→7.9\%) alongside the decline in DG/TG and X/TG ratios. The biophysical enrichments (increased bilayer thickness, reduced mobility sets) are indicative of more ordered membrane configurations under LI, which correspond with the galactolipid restructuring documented (lower MGDG/, slight DGDG bias) and the sphingolipid increase observed in the global class visualization. 

LI is characterized by (i) a coordinated reduction in PC-rich glycerophospholipids, and (ii) a coordinated augmentation in neutral glycerolipids, specifically DG and TG, along with lipid droplet/storage terms, accompanied by biophysical signatures indicative of thicker, more ordered membranes. These outputs from the LION analysis also align well with the univariate ratio statistics (δ approximating ±1; AUC approaching 0/100\%) and shifts in composition, thus underscoring that the lipidomic profile of LI reflects an expansion in lipid storage with concurrent remodeling of membrane headgroup and biophysical properties.

\begin{figure}[htbp]
  \centering
  \includegraphics[width=\textwidth]{fig/main/Fig1.png}

    \caption{Overview of lipidomics for Control and Lowinput \\
    \textbf{(A)} Percent‐of‐TIC breakdown for individual lipid \emph{species}, averaged across all C (n = 384) and LI (n = 362) samples. Only species whose mean contribution >= 3 \% are labeled in‐bar.  
    \textbf{(B)} We ranked all identified lipid species by their LI–C contrast and ran LION/web in ranking mode. Terms plotted above the midline (“DOWN”) are enriched among lipids that decrease in LI; those below (“UP”) are enriched among lipids that increase in LI. Bubble size reflects the number of annotated species contributing to a term; the x-axis shows enrichment strength (−log10 FDR-q; the dashed line marks q = 0.10). This presentation avoids arbitrary thresholds and captures coordinated movement of chemically/biophysically related lipids.}
    
  \label{fig:Fig1_lipid_class}
\end{figure}

\subsection*{Lipid Remodeling Revealed by PCA and OPLS-DA}

The PCA of species level lipid data shown in Fig \ref{fig:Fig2:OPLS}A, with each point representing a sample, effectively discriminates the two conditions along PC1, which accounts for 54.5\% of the variance, while PC2 contributes 6.7\% to the variance (cumulative $\approx 61.2\%$). The LI samples are positioned at positive PC1 values and form a compact cluster, whereas the C samples are located at negative PC1 values, with no overlap observed between the groups. PC2 represents only minor residual variation, manifested as vertical dispersion within each condition.This pattern indicates a comprehensive, multivariate reorganization of the lipidome, rather than being a consequence of outliers. 

For the class-summed glycerolipids and glycerophospholipids (Fig \ref{fig:Fig2:OPLS}B), the first two components elucidate 54.4\% and 16.8\% of the variance, respectively (cumulative $\approx 71.2\%$), once more distinguishing LI (right/positive PC1) from C (left/negative PC1). The loadings reveal the classes co-varying with each axis. Vectors oriented rightward and upward (aligning with LI) encompass PC, PE, PG, and the plastid galactolipids MGDG and DGDG, while SQDG, DG, LPC, and PA also project rightward with negative PC2, contributing to the LI quadrant. In contrast, MG, PS, and LPE load leftward and upward, associating with C. TG demonstrates only a short down-left vector in this scaling, implying its lesser contribution to the LI–C separation compared to other classes. The direction of PCA loadings here signifies covariance rather than indicating the sign of mean shifts. PC1 encapsulates the coordinated shift across various membrane-associated lipid classes that delineates conditions, whereas PC2 distinguishes chloroplast-associated patterns (MGDG/DGDG and PA/LPC contrasts) from other membrane and neutral lipids. The narrow confidence ellipses and lack of overlap corroborate that these class-level changes are systematic across samples and are not attributed to a few extreme points. 



%Fig \ref{fig:Fig2:OPLS}A presents a PCA of the individual lipid species, where each point represents a sample. The initial two components elucidate approximately 58.5\% of the total variance, with PC1 contributing 38.3\% and PC2 contributing 20.2\%. Samples are distinctly resolved according to condition along PC1. LowInput (LI) scores are displaced towards positive PC1 values and coalesce into a compact cluster, whereas Control (C) scores are located at negative PC1 values with no overlap. PC2 encapsulates secondary structure in the data, as C samples depict an oblique band extending from positive to negative PC2, indicative of coordinated covariation among subsets of lipids, while LI samples remain narrowly concentrated near PC2 $\approx$ 0 with limited dispersion. As separation occurs along the variance-maximizing axis, the figure suggests a comprehensive, multivariate reorganization of the lipidome rather than an influence exerted by a limited number of outliers. Hence, Fig \ref{fig:Fig2:OPLS} Panel A demonstrates that the condition alone accounts for the majority of between-sample variance in the species-level data, and that LI engenders a more homogenous lipid state compared to C.

%Fig \ref{fig:Fig2:OPLS} Panel B provides a summary of a PCA conducted on class level summed glycerolipids and glycerophospholipids, with the loading vectors superimposed as a biplot. Collectively, PC1 (43.7\%) and PC2 (34.3\%) account for approximately 78\% of the variance, effectively distinguishing the two conditions as well. Samples obtained under the LI condition are positioned to the right (positive PC1) and slightly upward, while C samples are situated in the left–down quadrant. The arrows denote the lipid classes that drive the observed separation. Vectors oriented to the right/upward—specifically TG, MG, PE, PG, PC, and PS exhibit positive PC1 loadings, with PS, PE, MG, and TG also displaying positive PC2 loadings. Their orientation suggests that relative enrichment of these classes shifts samples towards the LI aggregation. This observation is consistent with individual class accounting that shows neutral/storage lipids (TG, MG) increase under LI, and the primary zwitterionic bilayer lipids (PC, PE, PG) are either maintained or slightly elevated (Fig \ref{fig:Fig1_lipid_class}B; Supplementary Figure \ref{fig:S5}B;Supplementary Figure \ref{fig:S5}C). Conversely, vectors oriented downwards, particularly PA and SQDG (near-vertical, negative PC2) and the galactolipids MGDG/DGDG along with DG/LPC in the lower-right quadrant represent features that situate C samples in the left–down region of the plane. Most notably, SQDG (strongly negative PC2, $\mathrm{PC1} \approx 0$) and MGDG (negative PC2 with a modest +PC1 component) carry two of the longest loadings, indicating that chloroplast glycolipid variation forms the principal axis orthogonal to the LI–C separation. Functionally, PC1 encapsulates the remodeling from a galactolipid/anionic-inclined configuration towards a storage-enhanced, zwitterionic bilayer state, whereas PC2 differentiates chloroplast signatures (MGDG/DGDG, SQDG/PA) from PC/PE/TG. The tight ellipses and minimal overlap indicate that these class-level transformations are coordinated across samples rather than being driven by outliers as well. Hence, the biplot demonstrates that the transition to LI is elucidated by synergistic elevations in TG/MG and the PC/PE/PG axis, alongside a reduction in MGDG/SQDG/PA contributions, precisely mirroring the pattern observed in the total ion current (TIC) and subclass analyses.


We also examined within-plant lipid class ratios instead of absolute abundances to elucidate the comparison of C and LI across a multi-year, multi-field design. By aggregating molecular species into their parent classes and constructing all pairwise class ratios, we effectively minimized both batch noise and field variability while preserving the biochemical contrasts indicative of membrane remodeling. We applied an OPLS-DA model to the ratio value to identify the combinations of classes that most effectively differentiate the two conditions. The scores plot ($t_1$ vs.\ $o_1$) revealed a distinct and reproducible separation of C and LI along the primary predictive axis $t_1$, which accounted for 60.9\% of the X variance in our data, with the remaining variation captured orthogonally on $o_1$ (Fig \ref{fig:Fig2:OPLS} Panel C). The model exhibited ideal performance metrics with $R^2$Y $\approx$ 0.98 and $Q^2$ $\approx$ 0.98, signifying outstanding goodness-of-fit and predictive capacity, respectively. To prevent overfitting, 500 response-label permutations were executed. The permuted $R^2$Y and $Q^2$ values were centered close to zero with limited dispersion, whereas the observed statistics were positioned at the extreme right of their null distribution tails (exact permutation p($R^2$Y)=0.002, p($Q^2$)=0.005) Supplementary Figure \ref{fig:S6}. These evaluations support a stable, generalizable separation attributable to biological factors rather than stochastic structures within the data. For model interpretation, we ranked all ratios using variable importance (VIP) and retained those with VIP values exceeding 1 (Fig \ref{fig:Fig2:OPLS} Panel D, left). Two prominent features emerged. Firstly, ratios contrasting neutral glycerolipids with membrane classes, such as, TG- or DG-to-phospholipid and to galactolipid ratios, exhibited high VIP values, reflecting the expansion of neutral lipids noted in the TIC summaries. Secondly, ratios within the phospholipid framework (e.g., PC/PS, PE/PS, PG/PS) scored highly, indicating a synchronized rebalancing among headgroup families rather than a comprehensive increase or decrease in total phospholipids. For each VIP-selected ratio, we presented the paired distributions by lipid class collectively (Fig \ref{fig:Fig2:OPLS} Panel D, right). The ratios were categorized into sulfolipid, galactolipid, phospholipid, glycerolipid, lysophospholipid, and triacylglycerol. 

\subsubsection*{Sulfolipid ratios}
The LI samples exhibit relatively higher sulfolipid ratio values than in C samples. The PS/SQDG ratio displays a shift of $+1.11 \,\log_{10}$ (approximately $12.8\times$), the MG/SQDG ratio increases by $+0.70 \,\log_{10}$ (approximately $5.0\times$, with a Hodges--Lehmann shift of approximately $+0.70 \,\log_{10}$, and a 95\% confidence interval ranging from $0.69$ to $0.71$), and the PG/SQDG ratio shifts by $+0.42 \,\log_{10}$ (approximately $2.6\times$). These changes are accompanied with Cliff’s $\delta \approx +1.00$ and probability of superiority, also known as area under the curve (AUC), $\approx 100\%$, indicating a consistent superiority of nearly every LI sample over all Control samples for these ratios. Conversely, the inverse metric SQDG/TG demonstrates an opposing trend ($\delta \approx -1$, AUC $\approx 0\%$), illustrating that SQDG is also particularly reduced relative to neutral storage . These ratio modifications are also reflected by absolute class shifts. SQDG decreases from approximately $2.9\%$ to approximately $1.4\%$, whereas TG increases from approximately $2.7\%$ to approximately $5.4\%$ in LI samples (SuppTable). 

Under LI conditions, the ratios of phospho-/SQDG (such as PS/SQDG, PG/SQDG, PE/SQDG, PC/SQDG) and MG/SQDG exhibit significant increases, accompanied by almost complete group separation, as evidenced by Cliff’s δ values approaching +1 and an AUC nearing 100\%. Conversely, the opposite trend is observed for the SQDG/TG ratio, which decreases markedly, indicated by a δ of approximately $-$1 and an AUC close to 0\%. The mean values for classes transition in the same directional pattern, with SQDG diminishing from approximately 2.9\% to about 1.4\% and TG escalating from roughly 2.7\% to about 5.4\%. These coordinated alterations suggest a substantial depletion or turnover of SQDG within LI, coinciding with a concurrent expansion of the TG pool. Although the data regarding ratios and compositions are observational and do not inherently demonstrate metabolic flux or directionality, they align with models proposing that acyl carbons, liberated from plastid lipids, are subsequently re-esterified into extra-plastidic phospholipids and triglycerides.



%However, the presence of SQDG does not always correlate with photosynthetic ability
%R.A. Cedergren, R.I. Hollingsworth
%Occurrence of sulfoquinovosyl diacylglycerol in some members of the family rhizobiaceae
%Disruption of a gene essential for sulfoquinovosyldiacylglycerol biosynthesis in Sinorhizobium meliloti has no detectable effect on root nodule symbiosis


\subsubsection*{Galactolipid ratios} 
There is a minor reduction in MGDG and a slight increase in DGDG within LI (MGDG: $50.6\% \rightarrow 47.3\%$; DGDG: $18.8\% \rightarrow 19.2\%$ of TIC, SuppFig) at the class level. The ratio of DGDG to MGDG increases from approximately $0.37$ to $0.41$ ($\approx +9$--$10\%$ relative, SuppTable), suggesting a moderate shift towards the bilayer-forming galactolipid. Analysis of this ratio indicates that the shift is significant and systematic rather than attributable to random variations. All ratios of MGDG to phospholipids are elevated in the C, for example, MGDG/PS, MGDG/PG, MGDG/PE, and  MGDG/PC with median $\log_{10}$ differences between LI and C typically falling in the range of $0.3$–$1.0$ ($\approx 2\times$–$10\times$ lower in LI), with $q \ll 10^{-3}$, and effect sizes indicating nearly complete separation (Cliff’s $\delta \approx -1$, probability of superiority/AUC $\gtrsim 0.95$–$1.00$). Similarly, the DGDG/MG ratio favors C, consistent with a lower quantity of MG per unit DGDG in C (SuppFig). In contrast, MG/MGDG increases by approximately $+0.56 \,\log_{10}$ ($\approx 3.7\times$; HL CI tightly surrounds the estimate, $q \ll 10^{-3}$, $\delta \approx +1$, AUC $\approx 1.00$), and DGDG/MGDG increases by about $+0.29 \,\log_{10}$ ($\approx 1.9\times$;similar statistics). These patterns suggest an accelerated turnover of MGDG (more MG per MGDG) accompanied by a preference for DGDG (higher DGDG per MGDG). This is consistent with plastid membranes reducing the non-bilayer MGDG pool while modestly enriching the bilayer-stabilizing DGDG, a recognized strategy for maintaining thylakoid architecture under stress. These results support the presence of a coherent galactolipid remodeling program in LI rather than scattered, sample-specific effects.


\subsubsection*{Phospholipid ratios}
The C condition for PS-denominated ratios (PC/PS, PE/PS, PG/PS, PA/PS) demonstrates an approximate increase of $0.45$–$1.11 \,\log_{10}$ units in comparison to LI, which corresponds to an estimated range of $2.8$–$12.8\times$. The samples show an almost complete separation between groups, as indicated by Cliff’s $\delta \approx -1$, suggesting that nearly all C samples possess a higher PC/PS, PE/PS, PG/PS, PA/PS ratio than those observed in LI. From a biological stance, this suggests an enrichment of PS within the LI condition, 
while PC/PE/PG/PA are selectively reduced relative to PS. The class-composition percentages also corroborate these findings. Within the phospholipid pool, PC exhibits a slight reduction (from $79.9\%$ to $76.9\%$), whereas both PA and PS display an increase (PA: from $18.4\%$ to $20.8\%$; PS: from $<0.1\%$ to $0.2\%$). PG registers a slight increment (from $1.3\%$ to $1.5\%$). 

While the absolute changes in class percentages are minimal, the observed directional trends are strongly supported by ratio-based statistical analyses (e.g., PC/PS, PE/PS, PG/PS, PA/PS), which demonstrate significant effect sizes and an almost complete separation across numerous samples. The ratio data affirm that the redistribution of headgroups, favoring PS, is systematic rather than through some high value samples. Thus, there appears to be a relative shift in lipid metabolism that increases PS synthesis (e.g., via CDP-DAG/PS pathways), and/or a reduction in PS turnover, both of which would elevate PS relative to other phospholipids. 



\subsubsection*{Diacylglycerol / glycerolipid turnover ratios}
All DG-associated ratios experience a decrease under LI conditions. The DG/MG ratio shifts by $-0.48 \,\log_{10}$ (approximately $0.33\times$ the C), whereas the DG/PS ratio changes by $-0.89 \,\log_{10}$ (approximately $0.13\times$). Other DG-based comparisons (such as DG/TG, DG/PG, DG/PE) exhibit $q \ll 10^{-3}$, accompanied by Cliff’s $\delta \approx -1$ and a probability of superiority (AUC $\approx 0\%$), suggesting that nearly every LI sample ranks below almost every C sample. In contrast, MG ratios relative to membrane phospholipids exhibit an increase under LI conditions which include MG/PG, MG/PC, MG/PA, MG/PE, MG/PS, demonstrating median $\log_{10}$ shifts ranging from $+0.3$ to $+0.7$ ($\approx 2$–$5\times$), with $q \ll 10^{-3}$, $\delta \approx +1$, and AUC $\approx 100\%$. These statistics exhibit a near-complete separation between groups across various genotypes, effectively refuting the hypothesis of a small-outlier phenomenon. The class composition, as represented by the percentage of total ion current (TIC), reinstates this trend. TG levels increase (for example, from $\sim 2.7\% \rightarrow 5.4\%$ by class, and from $4.0\% \rightarrow 7.9\%$ by subclass), while DG levels experience a modest rise (from $20.3\% \rightarrow 21.9\%$), and MG levels decrease slightly (from $2.0\% \rightarrow 1.7\%$). Thus, even with a minor absolute increase in DG levels, this augmentation is less pronounced than that observed in TG and crucial phospholipids, thereby explicating the observed decline in DG/TG and DG/(PG/PE/PS) ratios. 


\subsubsection*{Effect-size and robustness metrics (reporting note)}
For each ratio we report three complementary statistics. 
(i) \textbf{Cliff's $\delta$} ($-1$ to $+1$) quantifies stochastic dominance: $\delta\!\approx\!+1$ means almost every LI value exceeds almost every Control value (and vice versa for $\delta\!\approx\!-1$). 
(ii) \textbf{Probability of superiority (AUC)} equals $P(\text{LI}>\text{C})$ and is numerically identical to the ROC area; AUC $\approx 100\%$ (or $0\%$) indicates near-complete separation. 
(iii) The \textbf{Hodges–Lehmann (HL) shift} is the robust median difference on the $\log_{10}$ scale (LI $-$ C); fold-change is $10^{\text{HL}}$. 
Multiple-testing control used Benjamini–Hochberg $q$-values.
Unless noted, all effects are on $\log_{10}$ ratios and confidence intervals are 95\% HL CIs.


\subsubsection*{Lyso-phospholipid ratios}
The majority of LPC-ratios (LPC/MG, LPC/PS, LPC/LPE, LPC/PE, LPC/PG) exhibit declines, with median shifts approximating $-0.53$ to $-0.02 \,\log_{10}$, $q \ll 10^{-3}$, and Cliff’s $\delta \approx -1$ (AUC $\approx 0\%$). This indicates that almost every LI sample is positioned below nearly every C. In contrast, ethanolamine-associated lysophospholipid such as LPE/MGDG shows a median shift of  $+0.56 \,\log_{10}$ ($\approx 3.7\times$; $q \ll 10^{-3}$; $\delta \approx +1$, AUC $\approx 100\%$), and LPE/PS exhibits a similarly positive shift with near-perfect separation. Meanwhile, DG/LPE declines ($\delta \approx -1$), indicating an increased presence of LPE relative to DG. Changes in composition corroborate this shift at the headgroup level: PC decreases from $79.9\%$ to $76.9\%$ of the phospholipid pool, whereas PE increases from $18.4\%$ to $20.8\%$; PS and PG both show modest increases (PS $<0.1\% \rightarrow 0.2\%$; PG $1.3\% \rightarrow 1.5\%$). The absolute LPC levels remain very small ($\sim 0.3$–$0.5\%$), ratio shifts after $z$-scoring serve as more reliable lipid changes. Taken together, the direction-consistent effects (VIP $\geq 1$; $q \ll 10^{-3}$; $|\delta| \approx 1$) support a reproducible LI-associated shift in the lysophospholipid balance from LPC toward LPE, rather than noise or outliers.


\subsubsection*{Triacylglycerol — endpoint sink of the re-acylation program}
The ratios of all lipid classes to TG, SQDG/TG, PS/TG, MGDG/TG, PA/TG, and LPC/TG, are decreased under LI conditions, as evidenced by high VIP ($\geq 1$), $q \ll 10^{-3}$, and Cliff's $\delta \approx -1$ (AUC $\approx 0\%$). This suggests near-complete separation among the genotypes, thereby excluding effects driven by outliers. The composition further affirms the ratio observation. TG approximately doubles (by class $\sim 2.7\% \rightarrow 5.4\%$; by subclass $\sim 4.0\% \rightarrow 7.9\%$), while the numerators either diminish or show only minor variations (SQDG $\sim 4.3\% \rightarrow 2.0\%$; MGDG $\sim 50.6\% \rightarrow 47.3\%$; PS $<0.1\% \rightarrow 0.2\%$; LPC $\sim 0.3\%$–$0.5\%$; PA $<0.1\%$). Since TG levels rise and most numerators do not correspondingly scale, all $X/$TG ratios decrease as theoretically anticipated. The consistent directional shifts in ratios towards TG, substantial nonparametric effect sizes, and compositional alterations indicate an expansion of the TG pool under LI conditions rather than random variability.


\begin{figure}[htbp]
  \centering
  \includegraphics[width=\textwidth]{fig/main/Fig2.png}
  \caption{Orthogonal projections to latent structures discriminant analysis (OPLS‐DA) results.  
    \textbf{(A)} Score plot showing Control (purple) vs LowInput (yellow) samples on the predictive component $t_1$ (51.4\% of $X$‐variance) versus the orthogonal component $\mathrm{to}_1$, with 95\% confidence ellipses and dashed axes at zero.  
    \textbf{(B)} Permutation test: overlaid histograms of 200 permuted R²Y (yellow) and Q² (purple) values, with the true‐model metrics indicated by dashed lines (both $p=0.005$), and an arrow marking the cumulative $X$‐variance explained (R²X = 0.514).}
  \label{fig:Fig2:OPLS}
\end{figure}

\subsection*{Genome-wide Association Studies of Lipids Associated with Control and Lowinput}
We conducted GWAS with three distinct data, (1) individual lipid species (Supp Table 2), (2) aggregates of lipid class abundances, and (3) all conceivable pairwise ratios of these aggregated classes. Utilizing a significance threshold of $\log_{10}(p)\ge7$ for individual lipids and $\log_{10}(p)\ge5$ for sums and ratios , we identified 2189 and 7363 genes associated with lipid species (see Supplementary Table 6 and 8), X and 10360  genes associated with the ratios of summed classes (refer to Supplementary Table 4) in a 25kb window for C and LI respectively. The full set of annotations is available in Supplementary Table 6,7,8, and 9. All the GWAS results can be obtained from the shiny app. Here, the candidate genes can be identified using (i) genes with the highest occurrences, (ii) individual or sum GWAS relating to a particular trait, and (iii) ratio GWAS that explains some mechanisms or biological process Fig \ref{fig:Fig3}. We divide the GWAS results for C and LI.


%Re rin gwas:
%PG(16:0/18:0)

\subsection*{Low Input GWAS}
\subsubsection*{Candidate Gene Identification using Genes with Highest Occurrences}
\subsubsection*{1. Phospholipid GWAS Identifies a Phosphate Starvation Response Gene}
Candidate genes were prioritized by analyzing GWAS results within the phospholipid lipid class. The Myb-like DNA-binding domain gene \texttt{SORBI\_3001G384300} was consistently identified. This gene exhibits homology with \textit{PHR1} (PHOSPHATE STARVATION RESPONSE 1) in rice, a principal regulator of phosphate homeostasis. It demonstrated associations with several phospholipid traits, namely PC(16:0/20:3), PC(16:0/22:5), PC(16:0/22:6), PC(18:1/20:1), PE(16:0/18:1), PC(18:1/24:1), PC(18:2/20:0), and PC(18:3/0:0), indicating a prospective correlation between phospholipid remodeling and phosphate starvation signaling.

\textit{PHR1} serves as a central transcription factor within plants, balancing the responses to phosphate (Pi) deprivation. It is categorized under the MYB-CC family of transcription factors and demonstrates a high level of conservation across both vascular plants and unicellular algae (Rubio et al., 2001). \textit{PHR1} exhibits specific binding affinity to a cis-regulatory element termed the P1BS (GNATATNC) motif, which resides in the promoters of numerous genes induced by Pi starvation, thereby facilitating their expression in conditions of Pi deficiency. These genes encompass those that encode phosphate transporters, signaling components, and enzymes that partake in metabolic adaptations to Pi scarcity (Bustos et al., 2010). Loss-of-function phr1 mutants display compromised expression of genes responsive to Pi starvation and a diminished accumulation of anthocyanins, starch, and sugars under conditions of Pi deficiency, along with modified Pi distribution between roots and shoots (Rubio et al., 2001; Bustos et al., 2010). In contrast, overexpression of PHR1 results in augmented Pi uptake and improved responses to Pi starvation (Nilsson et al., 2007). In addition to maintaining phosphorus homeostasis, PHR1 also plays a crucial role in regulating sulfate homeostasis, particularly under conditions of phosphate deficiency. It enhances the expression of the sulfate transporter gene SULTR1;3 and influences the translocation of sulfate from the aerial parts to the roots during phosphorus starvation. The observation that mutants in either phr1 or sultr1;3 demonstrate diminished sulfate transfer from shoots to roots suggests that PHR1 is integral to the interaction and coordinated regulation of P and sulfur homeostasis (Rouached et al., 2011). 

%SPX1 is a nuclear protein that interacts directly with PHR1 and functions as a phosphate-dependent inhibitor of PHR1 activity. This interaction is critically influenced by intracellular phosphate (Pi) concentrations; during conditions of Pi sufficiency, SPX1 binds to PHR1, thereby inhibiting its interaction with the P1BS motif in target gene promoters and consequently repressing Pi starvation responses. Conversely, under conditions of Pi deprivation, this interaction is attenuated, allowing PHR1 to activate its target genes (Puga et al., 2014). This mechanism establishes a molecular connection between Pi sensing and signaling, wherein Pi itself regulates the activity of the PHR1 transcription factor through SPX1. Notably, the Pi analog phosphite (Phi), which lacks metabolic capability but suppresses Pi starvation responses, can replicate Pi in promoting the SPX1-PHR1 interaction, further substantiating the hypothesis that SPX1 facilitates a direct perception of Pi levels (Puga et al., 2014).



%References

%Rubio V, Linhares F, Solano R, Martín AC, Iglesias J, Leyva A, Paz-Ares J. (2001). A conserved MYB transcription factor involved in phosphate starvation signaling both in vascular plants and in unicellular algae. Genes & Development, 15(16), 2122–2133.

%Bustos R, Castrillo G, Linhares F, Puga MI, Rubio V, Pérez-Pérez J, Solano R, Leyva A, Paz-Ares J. (2010). A central regulatory system largely controls transcriptional activation and repression responses to phosphate starvation in Arabidopsis. PLoS Genetics, 6(9), e1001102.

%Nilsson L, Muller R, Nielsen TH. (2007). Increased expression of the MYB-related transcription factor, PHR1, leads to enhanced phosphate uptake in Arabidopsis thaliana. Plant Cell and Environment, 30(11), 1499–1512.

%Rouached H, Secco D, Arpat AB, Poirier Y. (2011). The transcription factor PHR1 plays a key role in the regulation of sulfate shoot-to-root flux upon phosphate starvation in Arabidopsis. BMC Plant Biology, 11, 19.

%Puga MI, Mateos I, Charukesi R, Wang Z, Franco-Zorrilla JM, de Lorenzo L, Irigoyen ML, Masiero S, Bustos R, Rodríguez J, Leyva A, Rubio V, Sommer H, Paz-Ares J. (2014). SPX1 is a phosphate-dependent inhibitor of PHOSPHATE STARVATION RESPONSE 1 in Arabidopsis. Proceedings of the National Academy of Sciences of the United States of America, 111(41), 14947–14952.



\subsubsection*{2. DGAT1 Controls Triacylglycerol Storage in Response to Nitrogen Limitation and Cold}

We identified the gene \textit{SORBI\_3010G170000}, which encodes Acyl‐CoA:diacylglycerol acyltransferase 1 (DGAT1, analogous to Arabidopsis TG1), in five distinct GWASs: TG(18:1/18:3/22:0), TG(518:2/20:3/22:0), TG(18:2/18:2/18:4), TG(18:2/20:3/22:0), and TG(18:3/18:3/18:3). DGAT1 is responsible for the essential final conversion of DG into TG, which is a key lipid for carbon and energy storage in seeds and stress-affected vegetative tissues \cite{Zhang2009,Yang2011}. In Arabidopsis, low N levels result in the TG accumulation within leaves due to increased levels of DGAT1 and OLEOSIN1 \cite{Yang2011}. The ABA signaling pathway, involving the transcription factor ABI4, directly stimulates DGAT1 by interacting with CE1 elements (CACCG) in its promoter. In \emph{abi4} mutants, both DGAT1 stimulation and TG accumulation are reduced, emphasizing the significance of ABI4 during N deficiency \cite{Yang2011}. Additionally, DGAT1 is highly responsive to cold temperatures (4°C) and plays an essential role in freeze tolerance. Arabidopsis mutants deficient in \emph{dgat1} develop chlorosis and increased cell mortality under cold stress, with reduced TG but higher DG and PA levels \cite{Tan2018}. This elevated PA production induces RbohD-dependent ROS formation, causing oxidative stress. Increased DG kinase activity (DGK2/3/5) (GWAS results) further boosts PA, while the removal of \emph{dgk} genes restores cold tolerance, suggesting a balance between DGAT1 and DGK is essential for managing ROS and adapting to cold stress \cite{Tan2018}. In seeds, both DGAT1 and phospholipid:diacylglycerol acyltransferase 1 (PDAT1) are vital for optimal oil body development. \emph{dgat1} mutants have a 20–40\% decline in seed oil content (see Lipid annotation Section 5), whereas double mutants (\emph{dgat1/pdat1}) or RNAi lines demonstrate an 80\% decrease in TG, resulting in fertility and embryonic issues \cite{Zhang2009}. Overexpression of DGAT1 enhances seed weight and oil production, highlighting its crucial role in regulating TG levels throughout plant development \cite{Zhang2009,Yang2011}.



\subsubsection*{Candidate Gene Identification using Sum and Ratios of Lipids}
\subsubsection*{1. Sum of SQDG GWAS Identifies a Sulphate Assimilation Gene}
Our GWAS for the sum of SQDG identified a sulphate assimilation gene, called the adenylyl-sulfate kinase gene (APK3). APK3 is one of the four isoforms of APS kinase (adenosine 5'-phosphosulfate kinase) in Arabidopsis thaliana, an enzyme that plays a critical role in sulfur metabolism by phosphorylating adenosine 5'-phosphosulfate (APS) to produce 3'-phosphoadenosine 5'-phosphosulfate (PAPS), the active sulfate donor required for sulfation reactions in secondary metabolism (Mugford et al., 2009, p.1-2). Unlike the other APK isoforms, APK3 is uniquely localized in the cytosol, whereas APK1, APK2, and APK4 are plastid-localized (Mugford et al., 2009, p.4). The enzyme's activity influences sulfur partitioning between primary and secondary metabolism, particularly affecting the synthesis of sulfated secondary metabolites such as glucosinolates, which are important for plant defense (Mugford et al., 2009, p.1-2). Studies have shown that disruption of APK1 and APK2 leads to a significant reduction in glucosinolate levels and an increase in thiols, indicating that APKs regulate the availability of PAPS and thus control the flux toward secondary sulfated compounds (Mugford et al., 2009, p.2-3). However, the specific disruption of APK3, the cytosolic isoform, does not significantly affect primary sulfate assimilation or glucosinolate levels, suggesting a more specialized or possibly redundant role compared to plastidic APKs (Mugford et al., 2009, p.4). Furthermore, the C-terminal STAS domain of SULTR1;2, a sulfate transporter, interacts with OAS-TL and is involved in feedback regulation of sulfate transporter activity, but how APK3 activity integrates within this regulatory network remains unclear (Sum\_SQDG\_APK3\_S\_starvation.pdf, p.4). Overall, APK3 contributes to sulfur metabolism by modulating PAPS production in the cytosol, influencing sulfur flux balancing, but its precise regulatory role requires further elucidation.

%References:

%Mugford, S.G., Lee, B.-R., Koprivova, A., Matthewman, C.A., and Kopriva, S. (2009). Disruption of adenosine-5′-phosphosulfate kinase in Arabidopsis reduces levels of sulfated secondary metabolites. Plant Cell 21, 910–927. (Sum_SQDG_APK3_secondary_S.pdf, pp. 1–5)
%Kopriva, S., Mugford, S.G., Baraniecka, P., Lee, B.R., Matthewman, C.A., and Koprivova, A. (2012). Control of sulfur partitioning between primary and secondary metabolism in Arabidopsis. Frontiers in Plant Science, 3, 163. (Sum_SQDG_APK3_S_starvation.pdf, p.14)
%Takahashi, H. (2019). Sulfate transport systems in plants: functional diversity and molecular mechanisms underlying regulatory coordination. J. Exp. Bot. 70, 4075–4087. (Sum_SQDG_APK3_S_starvation.pdf, p.16) 


%\subsection*{SQDG Metabolism and Its Role in Phosphate‐Starvation Responses}

%In our GWAS of SQDG(32:0) levels, the top locus was \textit{SORBI\_3002G000600}, which encodes the plant ortholog of sulfoquinovosyltransferase (SQD2). SQDG is a negatively charged glycolipid (sulfoquinovose = 6‑deoxy‑6‑sulfonato‑glucose) that constitutes up to 10–20\% of chloroplast thylakoid lipids and is critical for stabilizing photosystem II, photosystem I, and cytochrome \emph{b}\(_6\)\emph{f} complexes \citep{Yu2002,Qin2015,Umena2011}.

%Biosynthesis proceeds in two enzymatic steps \citep{Yu2002,Sun2021}:
%\begin{enumerate}[label=(\arabic*)]
%  \item \textit{SQD1} (UDP‑sulfoquinovose synthase): 
%        \[
%           \mathrm{UDP\!-\!Glc} + \mathrm{SO_3^{2-}} \;\longrightarrow\; \mathrm{UDP\!-\!sulfoquinovose}
%        \]
%  \item \textit{SQD2} (sulfoquinovosyltransferase): 
%        \[
%           \mathrm{UDP\!-\!sulfoquinovose} + %\mathrm{diacylglycerol} \;\longrightarrow\; \mathrm{SQDG}
%        \]
%\end{enumerate}

%Under phosphate (Pi) starvation, plants degrade phospholipids (e.g.\ PG) to recycle Pi, while \textit{SQD1} and \textit{SQD2} are transcriptionally upregulated, leading to increased SQDG accumulation and preservation of thylakoid membrane functions \citep{Essigmann1998,Nakamura2013,Sun2021}. In rice, \textit{OsPHR2} directly activates \textit{OsSQD1} under Pi deficiency, and loss of \textit{OsPHR2} impairs SQDG levels, alters fatty‐acid composition, and reduces photosynthetic efficiency \citep{Sun2021}. Similarly, \emph{sqd2} mutants in \emph{Arabidopsis thaliana} are unable to synthesize SQDG and exhibit growth defects under low‐Pi conditions, underscoring the essential role of SQDG in replacing anionic phospholipids in the chloroplast \citep{Yu2002}.



\subsubsection*{2. Lipid Ratios identifies a Lecithin-Cholesterol Acyltransferase-like 1 Gene}
Two LCAT-like-1 genes on chromosome 3, \textit{SORBI\_3003G074500} and \textit{SORBI\_3003G074600}, were identified through GWAS with three distinct ratio phenotypes, (Sum\_AEG/Sum\_GalCer, Sum\_DG/Sum\_GalCer, Sum\_DGDG/Sum\_GalCer). The genes were in close proximity and were identified through the 25kb window. The phenotype is corroborated by 24 significant single nucleotide polymorphisms (SNPs), with the  SNP\_6349502 as the most significant with p-value 8.90 $\times 10^{-10}$. An additional LCAT-like gene on chromosome 1, \textit{SORBI\_3001G448800}, is associated with an extra lipid ratio (Sum\_Cer/Sum\_LPE), which harbors 84 significant SNPs with SNP\_72606991 having the best p-value of $9.33 \times 10^{-8}$. Studies have indicated that in yeast, Lro1 (LCAT-like) catalyzes this specific reaction, exerting substantial influence over the synthesis of TG. Its deletion results in a reduction of TG, whereas its overexpression leads to an increase in TG levels (Oelkers \textit{et al.}, JBC 2000). In plants, there are distinct PDAT orthologs that exhibit comparable chemical activities. 

Taken together, the multiple ratios converging on the same loci, overlap with OPLS-DA ratios, and the global distributional shifts in the lipidome (higher TG, depressed DG-anchored ratios, and reduced LPC-anchored ratios in LI), the genetic signal is \emph{consistent} with natural allelic variation at LCAT/PDAT-like enzymes. The lecithin-cholesterol acyltransferase–like (LCAT-like) family comprises enzymes such as phospholipid:diacylglycerol acyltransferases (PDATs). These enzymes facilitate the transfer of an acyl group from PC to diacylglycerol (DG), yielding triacylglycerol (TG) and lysophosphatidylcholine (LPC) through a CoA-independent mechanism.
\[
\text{PC} + \text{DG} \;\xrightarrow{\text{LCAT/PDAT}}\; \text{TG} + \text{LPC}.
\]


%\subsection*{Senescence‐linked lipid remodeling and the SAG39 candidate}

%Tier 1 – strongest mechanistic readouts (use these first)
%Sum_LPE / Sum_MGDG and Sum_LPC / Sum_MGDG
%Lysophospholipids (LPE/LPC) rise with PLA-mediated deacylation during membrane breakdown, while MGDG (core thylakoid galactolipid) falls as chloroplasts dismantle. These ratios jump when senescence intensifies—exactly the context where a SAD39 protease would be high.

%Sum_DGDG / Sum_PS and Sum_PG / Sum_PS
%DGDG/PG are plastid lipids; PS is extraplastidic/ER-derived. Senescence shifts lipid balance away from plastid membranes and toward extraplastidic pools. Expect these ratios to decrease with stronger senescence.

%Tier 2 – flux to storage / membrane remodeling (very good)
%Sum_DG / Sum_TG
%Senescing leaves divert DAG to TAG (lipid droplets). As storage rises, DG/TG tends to drop. A SAD39 allele marking stronger senescence should track that direction.
%Sum_PC / Sum_SQDG and Sum_PE / Sum_SQDG
%SQDG (plastid sulfolipid) diminishes with thylakoid loss; PC/PE (ER phospholipids) often hold or increase. These ratios typically increase with senescence.

%Our GWAS identified a senescence-specific cysteine protease SORBI\_3010G113600 (SAG39) as a candidate based on distinctive membrane lipid ratios (Supplementary Table), particularly LPE/MGDG and plastid/extraplastid contrasts such as DGDG/PS and PG/PS. These lipid ratios are mechanistically involved in chloroplast dismantling, a defining feature of senescence. Under low-input conditions, there is a decline in chloroplast thylakoid galactolipids like MGDG and DGDG (Fig), attributed to PLA-type deacylation. Conversely, lysophospholipids such as LPE and LPC accumulate as direct products of phospholipase activity (Jimbo \& Wada, 2023; Domínguez \& Cejudo, 2021). Therefore, the rise in LPE/MGDG suggests an active deacylation process of chloroplast membranes, leading to a reduction in thylakoid mass. This creates a biochemical environment where proteases such as SAG39 are involved in breaking down stromal and thylakoid proteins (Fig).

%Similarly, during leaf senescence, the ratio of DGDG to PS declines, signifying a redistribution of lipid pools from plastid-dominated galactolipids to extraplastidial phospholipids. DGDG is a crucial component of thylakoid membranes, responsible for maintaining the structure and stability of photosynthetic complexes. However, it undergoes active degradation during chloroplast dismantling, a pivotal event in senescence, mediated through the activity of galactosidases and lipases that liberate its constituent diacylglycerol and galactose (Domínguez \& Cejudo, 2021; Lee et al., 2009; Springer et al., 2016). This depletion of plastidial galactolipids causes a reduction in the numerator of the DGDG/PS ratio. In contrast, PS, an extraplastidial phospholipid primarily situated in the inner leaflet of the plasma membrane and endomembrane system, undergoes dynamic remodeling during senescence, specifically through the elongation of its acyl chains from approximately C37 to C41 (Li et al., 2014). This elongation is accelerated under conditions of stress and aging, potentially stabilizing membrane curvature or aiding in repair, whereas excessive elongation and eventual externalization of PS can also signal programmed cell death. As plastid membranes disassemble and extraplastidial membranes undergo remodeling, the relative abundance of PS increases (Supp Fig 5), contributing to the reduction in the DGDG/PS ratio. Consequently, a decreasing DGDG/PS ratio encapsulates both aspects of senescence-associated membrane remodeling: the enzymatic degradation of plastid galactolipids and the compositional and structural modifications within extraplastidial PS. This ratio serves as a reliable biochemical marker that signifies the transition from plastid lipid pools to extraplastid lipid pools, supporting nutrient recycling, membrane restructuring, and cellular reprogramming during leaf senescence (Domínguez \& Cejudo, 2021; Li et al., 2014).

%Thus, the presence of SAG39 in these ratio GWAS strengthens the interpretation that our low‐input lipidomic shifts are not incidental but part of a coordinated senescence program coupling lipid catabolism, proteolysis, and neutral‐lipid sequestration (Besagni \& Kessler, 2013; Wang et al., 2018; Domínguez & Cejudo, 2021; Jimbo & Wada, 2023).


%\subsection*{MG and MGDG Ratio GWAS Identifies a PG Biosynthesis Gene}
%Among the three stressors, anticipated lipid alterations establish a flux competition at the DAG/CDP-DAG hub, potentially driving MG/MGDG downward while rendering genotypic differences in PG synthesis highly significant. Under conditions of nitrogen deficiency, the biogenesis of thylakoid membranes decelerates, leading to a reduction in chlorophyll content, impairment of chloroplast ultrastructure, and a decrease in MGDG levels, particularly in highly unsaturated forms such as 34:6- and 36:6-MGDG, along with other thylakoid lipids. This results in a decline of the MG/MGDG ratio due to a diminishing denominator (MGDG), with MG often also experiencing a decline as a consequence of reduced thylakoid membrane construction or turnover. Phosphorus deficiency typically induces a substitution of phospholipids by galactolipids, resulting in PG depletion and an increase in non-phosphorus lipids such as MGDG/DGDG and SQDG. Nevertheless, this lipid remodeling relies on the same DAG/CDP-DAG precursor pool, whereby genetic variability in the metabolic pathway towards PG as opposed to galactolipids can alter the MGDG supply and thus affect the MG/MGDG ratio.

%Cold conditions specifically enhance the dependency on phosphatidylglycerol (PG) for the function of photosystems. PG plays a critical role in the structure, repair, and stability of photosystem II and I (PSII/PSI), and cold-tolerant plant species often adjust PG content and degree of unsaturation to sustain photosynthetic efficacy. The biosynthesis of plastidic PG follows a pathway from CDP-diacylglycerol (CDP-DAG) to phosphatidylglycerophosphate (PGP) to PG; mutations affecting this pathway, such as those observed in PGP1, result in photosynthetic impairments. In cold environments, plants tend to sustain or increase PG levels even under conditions of low phosphorus availability, as PG is functionally irreplaceable, with squamous glycolipid (SQDG) only capable of partially substituting for it. Cold stress also disrupts the fluxes of phosphatidylcholine (PC) and phosphatidic acid (PA) that converge at the same diacylglycerol (DAG) pool, thereby further affecting monogalactosyldiacylglycerol (MGDG) availability.

%Integrating these observed phenomena within the context of the low-input (LI) treatment reveals that reduced nitrogen levels lead to the contraction of thylakoid membranes, thereby decreasing MGDG; phosphate deficiency would typically suppress PG levels, yet low temperatures mitigate this effect by ensuring the maintenance of PG for the stabilization and repair of photosystems. Our data elucidate a net result — a decrease in MG and MGDG, and an increase (or preservation) of PG — which aligns with a cold-induced fortification of PG that overrides aspects of the low-phosphate substitution framework. This biochemical competition clarifies why allelic variations within the plastid PG-biosynthesis pathway (e.g., CDS/PGP) emerge as significant in GWAS: genotypes that channel increased DAG/CDP-DAG flux to PG consequently deplete the DAG available for MGDG, further reducing MG/MGDG levels and positioning the PG gene as the predominant association, despite PG itself not forming part of the ratio.

%Phosphatidylglycerol (PG) constitutes the primary phospholipid within chloroplasts and is integral to plant tolerance against chilling stress. It is exclusively synthesized through the prokaryotic pathway within chloroplasts, and it is essential for both the development of chloroplasts and their photosynthetic functionality (Nussberger et al., 1993; Hagio et al., 2002; Wada & Murata, 2007). The fatty acid composition of PG, particularly the proportion of high-melting-point molecular species (HMP-PG) enriched in 16:0, 18:0, and 16:1-trans, is closely associated with chilling sensitivity. Plants resistant to chilling typically possess less than 10 \% HMP-PG, whereas numerous chilling-sensitive species have levels exceeding 30 \% (Murata, 1983; Roughan, 1985; Wada & Murata, 2007). Elevated HMP-PG levels induce a gel-phase transition at reduced temperatures, thereby perturbing membrane fluidity and resulting in cellular damage (Murata & Yamaya, 1984). Genetic and transgenic investigations have substantiated that increased HMP-PG content instigates chilling sensitivity. For instance, the Arabidopsis fab1 mutant, which accumulates approximately 40–50 \% HMP-PG, experiences a collapse in photosynthesis and eventual death after prolonged exposure to cold (Wu & Browse, 1995; Barkan et al., 2006; Gao et al., 2015). The targeted reduction of HMP-PG in fab1 restores cold tolerance, thereby demonstrating causality (Gao et al., 2020). This accumulation of evidence emphasizes the role of PG biosynthesis genes as critical regulatory points under light and temperature stress. They influence the MG/MGDG ratio indirectly via precursor competition and directly determine the plant’s capacity to sustain photosynthetic proficiency and withstand combined nutrient and temperature challenges.

%References (as provided):
%Nitrogen_deficiency.pdf pp. 6–9; membrane_remodeling_phosphorus.pdf pp. 237–243; membrane_lipid_P_reuse.pdf pp. 13–14; glycerolipid_remodeling_P_starve.pdf p. 7; glycolipid_remodeling_nitrogen_phosphorus_deficiency.pdf p. 13; Cold_tolerance_barley.pdf pp. 7–9; Glycerolipid_freezing.pdf pp. 4–8; Photosynthesis_thylakoid_glycerolipid.pdf pp. 4–7; Cold_tolerance_maize.pdf pp. 8–9.
%Nussberger et al., 1993; Hagio et al., 2002; Wada & Murata, 2007; Murata, 1983; Roughan, 1985; Murata & Yamaya, 1984; Murata et al., 1992; Wolter et al., 1992; Moon et al., 1995; Ishizaki-Nishizawa et al., 1996; Wu & Browse, 1995; Barkan et al., 2006; Gao et al., 2015; Gao et al., 2020.


\subsubsection*{Candidate Gene Identification using Individual Lipid}
\subsubsection*{1. Alternative Oxidase Roles in Photoprotection and Nitrate Assimilation}

Through our GWAS focused on alpha-carotene, we  identified an alternative oxidase (AOX) gene. Alpha-carotene (\(\alpha\)-carotene), a secondary chloroplast carotenoid, is primarily located within the reaction centers of photosystem I (PSI) and photosystem II (PSII), with only minor quantities found in the peripheral light-harvesting complexes \citep{Young1989}. It bears structural similarity to \(\beta\)-carotene, absorbs blue-green light, and facilitates energy transfer to chlorophyll while concurrently quenching triplet chlorophyll and reactive oxygen species (ROS) to safeguard the photosynthetic apparatus from photooxidative damage under intense light stress. Its co-localization with \(\beta\)-carotene in pigment–protein complexes indicates a contributory role in stabilizing the core structures of PSII and PSI \citep{Young1989}. The mitochondrial AOX pathway offers a non-phosphorylating alternative to cytochrome oxidase, directly oxidizing ubiquinol to water, thereby preventing over-reduction of the photosynthetic electron transport chain \citep{Vishwakarma2015}. AOX1A, the dominant isoform in green tissues, plays a role in dissipating excess reducing equivalents produced by photosynthesis, supports non-photochemical quenching (NPQ), and collaborates with the chloroplast malate–oxaloacetate shuttle to sustain cellular redox homeostasis. Under conditions of stress, such as high light or drought, that inhibit the cytochrome pathway, AOX activity curbs ROS formation and maintains photosynthetic efficiency \citep{Vishwakarma2015}. In addition to its photoprotective function, AOX is vital for nitrate assimilation in plants. During NO\(_3^-\) reduction, the accumulation of reducing equivalents may lead to chloroplast over-reduction; AOX counters this by channeling excess reductants into mitochondrial respiration, thereby preventing oxidative stress and sustaining photosynthesis \citep{Gandin2014}. Studies involving \emph{aox1a} T-DNA insertion mutants in \emph{Arabidopsis thaliana} corroborate that AOX engages with nitrate assimilation pathways to uphold redox balance and optimize C-N metabolism under varying N conditions \citep{Gandin2014,Vishwakarma2015}.


%\subsection*{Beta‑Sitosterol GWAS Links Cellulose Synthase to Membrane Stability}
%In our GWAS of beta-sitosterol (BS), we detected a significant association peak at the cellulose synthase locus SORBI\_3003G049600, indicating that variations in this CesA gene may affect BS accumulation or its function in stabilizing membranes in sorghum. BS is a common phytosterol in plants that integrates into lipid bilayers to regulate membrane fluidity and stability. Although its exact role in cell wall structure is not fully understood, BS is suggested to protect cells from abiotic and biotic stress by enhancing plasma membrane integrity and potentially interacting with cytoplasmic and chloroplast membranes \citep{Sayeed2016}. Studies from Arabidopsis implies that BS plays a role in defense responses, yet a conclusive characterization of its role in cell wall mechanics remains necessary \citep{Sayeed2016}. The cellulose synthase (CesA) complexes are responsible for synthesizing the β-1,4-glucan chains of cellulose, the primary load-bearing polysaccharide in plant cell walls. In Arabidopsis, specific CesA isoforms form plasma-membrane rosettes to produce primary-wall (e.g., AtCesA1, 3, 6) and secondary-wall cellulose (e.g., AtCesA4, 7, 8), which support cell expansion, mechanical strength, and biomass accumulation \citep{Mueller1980,Somerville2006,Hu2018}. Mutations in CesA genes (e.g., \emph{rsw1}, \emph{prc1‑1}) result in decreased cellulose content, weakened cell walls, altered cell morphology, and reduced stress resistance \citep{Hu2018,Arioli1998,Persson2007,CanoDelgado2003,HernandezBlanco2007}. Although CesA primarily directs carbon towards cellulose production, downregulation or mutation of certain CesA genes can redirect carbon flux towards storage compounds. In Arabidopsis seeds, suppression of CesA leads to a slight reduction in cellulose content and prompts compensatory increases in non-cellulosic polysaccharides or proteins \citep{Hu2020}. It has been proposed that redirecting carbon from cell wall polysaccharides to seed storage proteins and oils may enhance nutritional quality, addressing the inverse relationship between seed oil and protein content \citep{Tomlinson2004,Ekman2008,Iyer2008,Shi2012,Tan2011,YoshieStark2008,Knowles1983}. 


\subsubsection*{2. Gibberellic Acid Response GWAS Identifies a MADS‑Box Regulator of Flowering Time}

In the gibberellic acid (GA) genome-wide association study (GWAS), we detected the \textit{SORBI\_3007G090421}. GA\(_3\) enhances floral initiation in short-day sorghum genotypes, predominantly when in conjunction with far-red light (FR). Williams and Morgan (1979) demonstrated that the combination of GA\(_3\) and FR results in an advancement of flowering by 30 to 80 days in early to intermediate maturity lines, and independently facilitates stem elongation \citep{Williams1979}. Lee \emph{et al.} (1998) further elucidated that photoperiod and phytochrome B are instrumental in regulating endogenous GA\(_1\)/GA\(_{20}\) rhythms, with altered GA peaks in \emph{phyB}-deficient genotypes being associated with early flowering under non-inductive day lengths \citep{Lee1998}. In our GA\(_3\) GWAS, the MADS-box transcription factor gene SORBI\_3007G090421 was identified. MADS-box proteins, particularly Type II C-function genes, are key regulators of floral organ identity and flowering time, whereas Type I MADS (e.g., \emph{AGL62}) affects endosperm development with consequential indirect effects on reproductive timing \citep{Paul2020}. Environmental temperature influences the effects of GA3 on development; Jabir and Mahmoud (2021) reported that elevated temperatures at planting dates, coupled with GA\(_3\) (100 ppm), expedited sorghum flowering, improved germination, and enhanced enzymatic activities for nutrient mobilization \citep{Jabir2021}. Williams and Morgan also observed genotype-specific temperature responses under controlled versus field conditions, casting light on temperature as a crucial element in GA3-mediated flowering regulation.


\subsubsection*{3. GWAS of Zeaxanthin Reveals High Light-inducible Protein}

Zeaxanthin is an essential carotenoid that plays a significant role in the photoprotection mechanisms of photosynthetic organisms, predominantly acting within the framework of the xanthophyll cycle. Under conditions of high light (HL) stress, violaxanthin undergoes enzymatic de-epoxidation to form antheraxanthin, which is further converted into zeaxanthin. This carotenoid is instrumental in dissipating excess excitation energy by quenching excited chlorophyll molecules. The process effectively averts the generation of deleterious reactive oxygen species (ROS), thus safeguarding photosystem II (PSII) from photoinhibition (Levin and Schuster 2023). Zeaxanthin associates with light-harvesting complexes, such as LHCII and certain LHC-like proteins, thereby facilitating non-photochemical quenching (NPQ) to efficiently transmute excess absorbed photonic energy into thermal energy (Levin and Schuster 2023).

Our GWAS for zeaxanthin has identified the gene SORBI\_3002G033800, which has an orthologous counterpart in Arabidopsis, referred to as One-helix proteins (OHPs). These OHPs share homology with the high light-inducible proteins (HLIPs) found in cyanobacteria. These functions as small chlorophyll a/b-binding proteins characterized by a single transmembrane helix with an LHC motif. OHPs are upregulated under high light conditions, playing a pivotal role in the biogenesis and repair of PSII. They transiently associate with PSII core proteins and temporarily bind chlorophyll pigments during the PSII repair cycle, shielding chlorophyll molecules from photooxidative damage by facilitating energy dissipation through the direct transfer between chlorophyll a and β-carotene (Levin and Schuster 2023). In Arabidopsis, mutations in OHP1 result in compromised chlorophyll accumulation, thylakoid architecture, and photosystem functionality, highlighting their essential role in photoprotection and photosynthetic efficiency (Levin and Schuster 2023).


%References:
%Levin, G., & Schuster, G. (2023). LHC-like Proteins: The Guardians of Photosynthesis. International Journal of Molecular Sciences, 24, 2503. 12568911
%Levin, G., Yasmin, M., Simanowitz, M.C., Meir, A., Tadmor, Y., Hirschberg, J., Adir, N., & Schuster, G. (2022). A Desert Green Alga That Thrives at Extreme High-Light Intensities Using a Unique Photoinhibition Protection Mechanism. bioRxiv. 911
%Myouga, F., Takahashi, K., Tanaka, R., Nagata, N., Kiss, A.Z., Funk, C., Nomura, Y., Nakagami, H., Jansson, S., and Shinozaki, K. (2018). Stable accumulation of photosystem II requires ONE-HELIX PROTEIN1 (OHP1) of the light harvesting-like family. Plant Physiology, 176(4), 2277–2291.
%Hey, D., and Grimm, B. (2018). ONE-HELIX PROTEIN2 (OHP2) is required for the stability of OHP1 and assembly factor HCF244 and is functionally linked to PSII biogenesis. Plant Physiology, 177(4), 1453–1472.

\begin{figure}[htbp]
  \centering
  \includegraphics[width=\textwidth]{fig/main/Fig3.png}
  \caption{\textbf{Low‐input lipid GWAS Manhattan plots}
    \textbf{(A)} Manhattan plots for five Phosphplipids PC(16:0/20:3),PC(16:0/22:5),PC(16:0/22:6),PC(16:1/20:1), and PE(16:1/18:1), all peaking at PHR1 locus (\textit{SORBI\_3001G384300}) in chromosome 1; the vertical green dashed line marks the SNP position for this gene.  
    \textbf{(B)} Manhattan plots for five TG species TG(18:1/18:3/22:0),TG(18:1/20:3/22:0),PC(18:2/18:2/16:4),TG(18:2/20:3/22:0), and TG(18:3/18:3/18:3), all peaking at DGAT1 locus (\textit{SORBI\_3010G170000}) in chromosome 10; the vertical green dashed line marks the SNP position for this gene. 
    \textbf{(C)} Sum of SQDG Manhattan plot, identifying an adenylate sulfate kinase (\textit{SORBI\_3005G195600}).  
    \textbf{(D)} \(\alpha\)‐Carotene Manhattan plot, identifying an alternative oxidase (\textit{SORBI\_3006G202500}).  
    \textbf{(E)} Gibberellic acid Manhattan plot, at a sugar transporter SORBI/_3010G030600 and a MADS‐box transcription factor locus (\textit{SORBI\_3007G090421}) .  
    \textbf{(F)} Zeaxanthin Manhattan plot, marking a chloroplast RNA‐binding protein (\textit{SORBI\_3001G357200}).  
    Green dots indicate SNPs within or near the highlighted genes in panels B–F. Panels A and B show traditional lipids (–log\textsubscript{10} $p$\(\geq\)7), and panels C–F show non‐traditional lipids (–log\textsubscript{10} $p$\(\geq\)5).}
  \label{fig:Fig3}
\end{figure}


%\subsection*{Lipid Changes under LowInput (LI)}
%The multi-year, multi-field experimental design introduced considerable environmental variability into the lipidomic profiles. Although spatial corrections (SpATS) and batch-effect normalization (SERRF) were employed, residual confounding factors necessitated a ratio-based analytical approach to compare the C and the LI conditions. Lipid class ratios, which demonstrate greater resilience to technical and environmental noise than absolute abundances, were prioritized to identify biologically conserved patterns of membrane adaptation. To mitigate these confounding effects, we concentrated on within-plant lipid class ratios, expected to be fairly consistent under a variety of cultivation environments, rather than absolute abundances. We aggregated individual molecular species into their respective lipid classes, computed all possible pairwise ratios on a linear scale, and subsequently applied OPLS-DA to identify the most discriminative ratios. The resulting scores plot (Fig. \ref{fig:Fig2:OPLS}) demonstrated a robust separation of C versus LI samples, and the model exhibited exemplary performance metrics (R²Y = 0.99; Q²Y = 0.82) alongside a highly significant CV-ANOVA (p \textless 0.001) (Supp Figure, supp table). Hotelling’s T² analysis confirmed the absence of extreme outliers, and a 1,000-permutation test dismissed overfitting, indicating that our observations are unlikely attributable to chance. From the complete ratio set, 21 features surpassed a conservative VIP threshold of 1. Each of these high VIP ratios was then subjected to univariate testing (adjusted p \textle 0.05) and cross-verified against established pathways of membrane remodeling under cold and nutrient stress. The two-tier selection approach prioritizing statistical power and biological relevance yielded a defined set of lipid-class ratios (supp table) which may differentiate between C and LI. These lipid ratios are used for the further studying the effects of C and LI. 





%SORBI_3006G214500 - Lecithin cholesterol acyltransferase like 4
%SORBI_3001G448800 - Lecithin cholesterol acyltransferase like 1
%SORBI_3001G041900 - PNLPA
%SORBI_3001G042901 - Adenylate isopentanyltransferases


%\subsection*{1. Membrane Lipid Remodeling under LowInput (LI)}

%\subsubsection*{1.1 Sulfolipid (SQDG) Collapse Under LI Stress Adaptation}


\subsection*{Random forest SHAP identifies lipid predictors of phenotypic variation in SAP}
We examined the Sorghum Association Panel (SAP) under controlled conditions, modeling plant height and flowering time (Supplementary Table). Lipid intensities transformed as $\log_{10}(x+1)$, were median-centered, and, alongside each SAP phenotype, were residualized based on population PCs to mitigate confounding effects due to population structure. A random forest model was developed using an 80/20 train-test division, stratified by $k$-means clusters within PC, and optimized through 5-fold cross-validation to minimize the root mean square error (RMSE). This model was evaluated on the untrained dataset (Supplementary Table 9). For lipid ranking interpretation, we calculated exact TreeSHAP values (\texttt{treeshap}) for each individual lipid. The global importance was quantified as the mean absolute SHAP value, which served as the basis for determining all feature rankings. This same methodological framework under "control" conditions was subsequently extended to additional SAP phenotypes, such as  beta-carotene, grain number, thousand-grain weight, and zeaxanthin, utilizing datasets supplied by other research studies to facilitate cross-trait comparisons of lipid predictors. For testing, we selected our plant height data to compare with Boatwright's (ref) SAP plant height dataset. 

Fig.\ref{fig:Fig4}A shows the plant height phenotype before and after PC residualization. There is a striking transformation post PC residualization. The distribution compresses into a constrained, mean-zero residual with reduced variability, in contrast to the raw phenotype, which spans a wide, right-shifted spectrum. This observation suggests that a considerable portion of the variation in plant height is associated with the population structure captured by the PCs. Removing this component was important to model the lipid prediction to ensure that subsequent models do not erroneously interpret genetic-background differences as biological variation. Similar analysis was done for all the phenotypes that were tested. Fig.\ref{fig:Fig4}B shows the before and after PC residualization for lipid TG(10:0/10:0/10:0). There is only a small shift in density characterized by a slight recentering and minor change in variance, while the overall shape remains unimodal and similar to the original distribution for the lipid. However, some lipids have modest changes. It indicates that the population structure accounts for only a minimal portion of the variability. Thus, the majority of the signal is preserved post-adjustment. Collectively, the panels demonstrate that the PC residualization effectively removes strong structure from the phenotype while maintaining lipid abundances mostly intact, which is important for predictive modeling and TreeSHAP interpretation.

The random-forest regressor model for plant height demonstrated consistent generalization across five folds of CV on the training dataset (Fig. \ref{fig:Fig4}C). The mean performance across the folds is characterized by RMSE $=27.85$, MAE $=17.99$, and $R^2=0.680$ (dashed lines), with only minor variations between folds, suggesting that the accuracy is not reliant on any single partition. Evaluation on the test set (Fig.\,\ref{fig:Fig4}D) resulted in RMSE $=24.213$ and $R^2=0.688$. The predicted residual phenotypes align closely with the best fit line, showing slight compression at the extremes but lacking any apparent systematic bias. This outcome verifies that the combination of PC residualization and stratified splitting produced a model capable of generalizing beyond the training data.

To elucidate the interpretations of the fitted forest model, TreeSHAP values were calculated for all lipid entities for plant height. As depicted in Fig.,\ref{fig:Fig4}D, a sample-level beeswarm plot is superimposed with a bar illustrating the global mean importance corresponding to the same ordered features. Each point within the plot represents an individual sample. The x-axis indicates the SHAP contribution to the residual prediction of plant height, while color signifies the standardized lipid abundance. Points on the right-hand side suggest samples where increased lipid abundance increases the prediction, whereas points on the left imply the converse. 

Regarding plant height, the feature ranking is enriched for medium–chain TGs, such as TG(10{:}0/10{:}0/10{:}0), TG(10{:}0/12{:}0/16{:}0), TG(12{:}0/12{:}0/14{:}0), TG(12{:}0/12{:}0/16{:}1), with many additional medium–chain TGs among the top hits. Medium–chain fatty acids are rapidly funneled through $\beta$-oxidation, providing a ``fast-burn'' carbon/ATP source. This pattern indicates that taller plants are those with larger, more accessible neutral-lipid reserves that can be mobilized to meet the steep carbon and energy demands of stem elongation (cell wall biosynthesis, lignification, and meristematic growth). \textit{Gibberellic acid} appears among the important predictors but at a lower rank than the TG suite, suggesting a division of labor. GA acts as the developmental trigger for elongation, while the realized growth magnitude is constrained by fuel availability. Several phospho/galacto-lipids (e.g., phosphatidylcholine, phosphatidylethanolamine, monogalactosyldiacylglycerol, sulfoquinovosyldiacylglycerol) demonstrate bidirectional distributions around zero, reflecting context-dependent effects across diverse genotypes. The full list is shown in Supplementary Table.

Regarding flowering time, the model evaluation metrics were not as good as the plant height with Rsq 0.00268 (Supplementary Table). However, we were still able to identify lipids that explained flowering time. The signal is dominated by coordinated changes in signaling lipids, storage lipids, and membrane builders. The top feature is PA(16:0/18:2), a classic second messenger made rapidly by PLD/DGK during stress and hormone responses. PA can modulate growth hubs (e.g., TOR/SnRK1), which aligns with the biology of the floral transition turn down vegetative growth, execute a developmental program. Its prominence suggests PA is a key integrator of cues that move the apex toward flowering. Gibberellic acid appears among the top predictors, matching its well-known role in promoting flowering. The full list is shown in Supplementary Table. 

Fig.,\ref{fig:Fig4}E shows complementary evidence derived from Pearson correlations between phenotypes and lipids for the identical features. The observed correlation pattern aligns with the SHAP analysis in terms of directionality. Medium-chain TGs exhibit a strong positive correlation with residual plant height, while a number of membrane lipids display weaker or mixed correlation signs. Consistency between panels D and E corroborates a coherent mechanism, in which accessible neutral-lipid reserves promote stem elongation, while also identifying lipids whose effects are contingent upon the underlying state.

\subsection*{Cross study prediction and transferability of lipid features}
To evaluate the predictive accuracy of our lipidomics for phenotypes documented in independent studies, we utilized plant height data from Lucas Boatwright’s sorghum experiment (ref), which was conducted under similar growth conditions, as the response variable, with our lipid profiles serving as predictors. The performance of the model on this external dataset was characterized by metrics of \emph{RMSE} = 18.6, \emph{MAE} = 13.3, Pearson’s $r = 0.636$, $R^2 = 0.405$, and bias = $+5.41$, closely matching the accuracy observed in our field trials (Supplementary Table~4). TreeSHAP analysis identified several top-ranked lipids common to both the external and our datasets, indicating that the most informative lipid signals are biologically consistent across different experimental settings. This concordance motivated the application of the same analytical pipeline to additional SAP phenotypes assessed under similar conditions including stem diameter, \textbeta-carotene, lutein, zeaxanthin, grain number per primary panicle, thousand-grain weight, and grain yield per primary panicle—and the results, including per-phenotype metrics and prominent SHAP features are explained below (Supplementary Table 5).




%\subsubsection*{1. Plant height}
%\emph{Medium-chain triacylglycerols:} The feature ranking is enriched for medium–chain TGs, such as TG(10{:}0/10{:}0/10{:}0), TG(10{:}0/12{:}0/16{:}0), TG(12{:}0/12{:}0/14{:}0), TG(12{:}0/12{:}0/16{:}1), with many additional medium–chain TGs among the top hits. Medium–chain fatty acids are rapidly funneled through $\beta$-oxidation, providing a ``fast-burn'' carbon/ATP source. This pattern indicates that taller plants are those with larger, more accessible neutral-lipid reserves that can be mobilized to meet the steep carbon and energy demands of stem elongation (cell wall biosynthesis, lignification, and meristematic growth).

%\emph{Hormonal and signaling context}
%\textit{Gibberellic acid} appears among the important predictors but at a lower rank than the TG suite, suggesting a division of labor. GA acts as the developmental trigger for elongation, while the realized growth magnitude is constrained by fuel availability. Consistent with this, classical lipid signals (\textit{PA(16{:}0/18{:}2)}, and sphingolipid nodes (\textit{Cer(d18{:}2/20{:}1)}; \textit{SPB 18{:}0;2OH}; \textit{SPHINGANINE}) register as supporting cues rather than the leading axis.

%\emph{Membrane remodeling and turnover markers}
%Chloroplast and extraplastid membrane lipids contribute secondary signal: \textit{MGDG(18{:}2/18{:}2)}, \textit{SQDG(16{:}0/18{:}2)}, \textit{PC(18{:}1/22{:}1)}, and \textit{PC(18{:}1/18{:}2)} are consistent with the need to expand and remodel membranes during rapid cell proliferation. Carotenoid-derived volatiles (\textit{apocarotenal}; \textit{$\alpha$-ionone}, $0.320$) and sterols (\textit{$\gamma$-sitosterol}, $0.337$) likely report redox/stress status and membrane organization accompanying fast growth.


%\subsubsection*{2. Flowering Time}
%The signal is dominated by coordinated changes in signaling lipids, storage lipids, and membrane builders i.e., a whole-system metabolic shift rather than a single pathway.

%\emph{Phospholipids}
%The top feature is PA(16:0/18:2), a classic second messenger made rapidly by PLD/DGK during stress and hormone responses. PA can modulate growth hubs (e.g., TOR/SnRK1), which aligns with the biology of the floral transition turn down vegetative growth, execute a developmental program. Its prominence suggests PA is a key integrator of cues that move the apex toward flowering.

%Features such as DG(16:0/18:0) (0.068), PE(18:1/18:2) (0.056), and PC(14:0/14:0) (0.064) indicate broad membrane reconfiguration. DG is a hub precursor for both storage and membrane lipids; PE/PC composition shifts are expected when tissues change identity (vegetative → reproductive) and when trafficking between ER and plastids ramps up.

%\emph{Hormones}
%Gibberellic acid appears among the top predictors, matching its well-known role in promoting flowering. The apocarotenoid α-ionone (0.051) hints at carotenoid breakdown/signaling intersecting with developmental timing—consistent with stress/redox cues that often accompany the switch.

%\emph{Triacylglycerol (TG) }
%A long list of TGs e.g., TG(14:0/18:1/18:2), TG(18:1/18:2/18:2), TG(18:0/18:2/18:2), TG(12:0/12:0/14:0), TG(10:0/10:0/10:0) signals mobilization of carbon/energy. Breaking down storage lipids can feed β-oxidation and supply building blocks for rapidly growing floral tissues. The mixture of medium-chain and PUFA-containing TGs suggests active lipolysis and remodeling rather than a single static pool.

%\emph{Sphingolipid control of cell fate and stress}
%High-ranking ceramide (Cer(d18:2/20:1)) and its bases (SPB 18:0;2OH; sphinganine) point to sphingolipid signaling. These molecules are central to programmed cell death, stress, and senescence—processes that accompany meristem reprogramming and resource reallocation when plants commit to flowering.


%\emph{Additional signaling lipids}
%MG(20:4) (0.090) and the ether lipid AEG(o-18:4/16:1) (0.045) point to active lipase/β-oxidation routes and potential oxidative-stress buffering, respectively—both plausible in a high-demand transition.


\subsubsection*{1. Stem diameter:} 
Plastid membrane lipids exhibited the highest ranking, prominently featuring SQDG(18:3/18:3), with DGDG(16:0/18:1) also being noteworthy. Growth-related signals, including gibberellic acid and the oxylipin 9,12,13-TriHODE, were among the most significant factors, alongside PA(16:0/18:2) and several medium/long-chain TGs. This suggests an association between the remodeling of plastid anionic/galactolipids, oxylipin signaling, and stem thickening.

\subsubsection*{2. \textbeta‐Carotene:}
%\emph{Neutral-lipid storage context}. 
Many of the strongest predictors are TGs for example TG(12:0/16:0/18:2), TG(14:0/18:3/18:3), TG(12:0/18:2/18:3), TG(12:0/18:1/18:1), TG(12:0/18:1/18:3), and TG(8:0/16:1/18:1). Because carotenoids partition into lipid droplets, a TG-rich, PUFA-bearing background likely provides both solubilization and sequestration capacity, stabilizing β-carotene at higher levels. In short, TG abundance/composition behaves like a storage capacity proxy for carotenoid load.
%\emph{Direct pathway sentinels and stress/redox markers}. 
Molecules like apocarotenal (a carotenoid breakdown product), \textbeta-carotene itself, and \textalpha-carotene appear among the top features, providing an internal validity check that the model is capturing carotenoid metabolism. Ubidecarenone, (CoQ10), another isoprenoid, implicates shared precursor and redox state across isoprenoid branches. Oxylipin trans-EKODE and \textgamma-linolenoyl ethanolamide (NAE 18:3) indicate stress and redox signaling commonly co-regulated with antioxidant carotenoids. The full list is tabulated in Supplementary Table. 

%\emph{Plastid membrane setting}. Galactolipids characteristic of thylakoid membranes—DGDG(18:0/18:3), DGDG(18:2/18:3), and MGDG(18:2/18:2)—also rank highly. β-Carotene is synthesized and housed in the thylakoid; the MGDG/DGDG profile (especially its PUFA content) tunes membrane fluidity and the local environment of carotenoid enzymes. The model is effectively “reading” chloroplast membrane remodeling as a correlate of carotenoid output.

%\emph{Extra-plastid phospholipids that interface with plastids}. Several PCs/PEs—PC(18:1/24:0), PC(18:0/18:1), PC(18:1/20:4), PC(16:1/16:1), PS(18:0/18:2), PE(18:0/18:2), PC(16:0/18:2)—point to ER/plasma-membrane processes that supply acyl chains and signals to plastids. Very-long-chain and PUFA-rich PCs often mark active acyl editing and ER \leftrightarrow. plastid lipid exchange, which in turn shapes thylakoid composition and carotenoid biosynthesis.


\subsubsection*{3. Lutein} 
%\emph{Carotenoid network}
The model puts lutein itself at the top , with strong support from zeaxanthin , $\alpha$-carotene , and $\beta$-cryptoxanthin. These molecules are reading out the carotenoid biosynthetic and functional pathway. $\alpha$-carotene feeds the $\varepsilon$,$\beta$ branch that yields lutein. Zeaxanthin is a partner xanthophyll in photoprotection (xanthophyll cycle). $\beta$-cryptoxanthin sits in the $\beta$-branch and often co-varies under high-light/redox cues. In short, the model isn’t latching onto a single compound—it’s capturing the co-regulated carotenoid module that travels with high lutein.
%\emph{Apocarotenoids and N-acylethanolamides}
Apocarotenal and γ-linolenoyl ethanolamide (NAE 18:3) indicate active carotenoid turnover and lipid signaling. Apocarotenoids are oxidative cleavage products of carotenoids and can act as signaling molecules. NAEs and related oxylipin routes often escalate under stress, conditions that also upregulate photoprotective xanthophylls. Their presence suggests lutein levels reflect not just synthesis and storage, but dynamic turnover and feedback via redox/stress pathways.
%\emph{Triacylglycerols}
A dense block of TGs is among the top drivers such as TG(12:0/18:1/18:1), TG(12:0/12:0/18:2), TG(18:1/20:1/22:1), TG(12:0/16:0/18:2), TG(12:0/18:1/18:3). Lutein is fat-soluble and partitions into hydrophobic phases; abundant/PUFA-rich TGs indicate lipid-droplet/plastoglobule capacity that can solubilize and protect lutein. The repeated appearance of medium- to long-chain and PUFA-containing TGs points to droplets with favorable packing and fluidity for xanthophyll accommodation.
%\emph{Phospholipids}
%Multiple phospholipids rise in importance—PC(18:1/18:1) (0.00195), PC(16:0/18:1) (0.00163), PC(16:1/16:1) (0.00144), PC(16:0/22:6) (0.00139), plus PE(16:0/18:2) (0.00145) and PS(18:0/18:2) (0.00154). PCs/PE/PS are ER-centric membrane lipids that participate in acyl editing and ER↔plastid lipid exchange. Their chain-length/unsaturation signatures often mirror membrane remodeling under high light and development, which in turn influences thylakoid composition—the very milieu where lutein is bound. The signal here says: extra-plastid membrane state and inter-organelle lipid traffic are part of the lutein phenotype.


%\emph{Sphingolipid and very-long-chain context}
%The appearance of Cer(24:1) (0.00161) points to sphingolipid signaling/membrane organization, which can modulate photosynthetic membranes and stress responses. Together with very-long-chain PCs (e.g., PC 24:0/22:6 contexts), this supports a picture of membrane microdomain remodeling that coincides with high lutein states.


\subsubsection*{4. Zeaxanthin}
%\emph{The xanthophyll cycle}
A defining result is the prominence of the xanthophyll cycle intermediate \textit{antheraxanthin}, indicating that zeaxanthin variation is driven by enzyme-level conversion dynamics (VDE/ZE). Genotypes with higher antheraxanthin{\,$\leftrightarrow$\,}zeaxanthin flux under stress/light show the highest zeaxanthin.
%\emph{Hormonal link: gibberellic acid (GA)}
%\textit{Gibberellic acid} ranks among the top predictors, suggesting coordination between developmental programs and photoprotection. GA may (i) mark growth stages demanding enhanced NPQ capacity or (ii) cross-talk with stress pathways that activate the xanthophyll cycle. This provides a concrete, testable axis linking hormone status to zeaxanthin.
%\emph{Thylakoid membrane context}
Chloroplast lipids \textit{DGDG(18{:}0/18{:}2)} and multiple \textit{SQDG} species highlight that thylakoid composition predicts zeaxanthin capacity. Because zeaxanthin binds antenna complexes in PSII, the galacto-/sulfolipid matrix sets the biophysical environment for both accumulation and rapid turnover.
%\emph{Substrate/flux readiness}
%High-ranking \textit{DG(16{:}0/16{:}0)}, \textit{DG(18{:}0/18{:}2)}, and \textit{MG(18{:}3)} indicate that glycerolipid backbone availability and flux support (i) membrane maintenance/remodeling and (ii) neutral-lipid synthesis, both conducive to zeaxanthin homeostasis.
%\emph{Storage/sequestration theme}
%Multiple triacylglycerols (TGs) \textit{TG(12{:}0/16{:}0/18{:}3)}, \textit{TG(14{:}0/18{:}3/18{:}3)}, \textit{TG(12{:}0/18{:}1/18{:}2)}, \textit{TG(14{:}0/16{:}0/18{:}1)} point to neutral-lipid capacity (lipid droplets/plastoglobules) and PUFA-rich environments that stabilize hydrophobic pigments.
%\emph{Turnover and redox markers}
%Signals such as \textit{apocarotenal} reflect active carotenoid turnover and redox state, consistent with zeaxanthin’s rapid, reversible role in non-photochemical quenching.

%\subsubsection*{5. Grain number per primary panicle:}
%The top feature, MG(20:4), is a lipid-turnover flag: monoacylglycerols rise when lipases remodel membranes. Together with high-ranking DG(16:0/18:0) and DG(16:1/18:3), plus head-group lipids (PC(14:0/18:2), PC(16:1/16:1), PS(18:0/18:2), PE(18:0/18:2)), the profile points to intense membrane recycling and resynthesis—exactly what’s needed during floret initiation, meiosis, and rapid cell division in the developing panicle.

%\emph{Energy supply from mitochondria}
%The prominence of cardiolipin CL(18:1) is a strong mitochondrial signal. CL scaffolds respiratory complexes, so its importance implies that ATP production capacity in the panicle tissues constrains final grain set (sink strength).

%\emph{Specialized neutral-lipid pools}
%Unlike plant height (which favored medium-chain TGs), grain number elevates TGs with long/very-long chains—e.g., TG(16:0/18:1/22:0), TG(18:0/18:2/22:0), TG(18:1/20:0/22:1), TG(18:0/18:0/20:1). These species are typical of reproductive tissues (tapetum, pollen coat, cuticle), suggesting roles in structural provisioning and signaling rather than generic fuel.

%\emph{Developmental signaling and fate decisions} 
%Sphingolipid nodes—SPB 18:0;2OH (2.55) and sphinganine (2.39)—plus the apocarotenoid α-ionone (3.18) indicate lipid-derived signals that likely help determine which florets abort vs. set grain. DAGs also serve as second messengers, reinforcing a signal-integration layer atop membrane remodeling.

%\emph{Panicle (chloroplast) contribution} 
%The sulfolipid SQDG(16:0/14:0) (2.73) and PG(18:0/16:0) (2.20) tie grain number to plastid membrane status—consistent with panicle photosynthesis supporting early grain set.

%\subsubsection{Thousand‐grain weight:} 
%\emph{Antioxidant shield (key insight)} The #1 phospholipid PC(18:1/20:4) (0.0193) and the #2 α-tocopherol (vitamin E) (0.0172) anchor a strong redox/antioxidant signal. Tocopherol guards PUFA-rich oils against peroxidation during grain filling; its high importance suggests genotypes with more robust antioxidant buffering can pack and retain more biomass. Supporting isoprenoid/redox markers—ubidecarenone (CoQ) (0.0127), β-carotene (0.0128), zeaxanthin (0.0124), and apocarotenal (0.0154)—reinforce that oxidative control is a limiting step for final seed mass.

%\emph{Membrane machinery for filling} Multiple head-group and lyso-lipids—PC(14:0/18:2) (0.0166), PC(16:0/18:0) (0.0151), PC(14:0/14:0) (0.0127), PC(16:0/20:3) (0.0126), PC(18:1/18:1) (0.0121), PC(18:2/0:0) (0.0116), LPE(16:0) (0.0169), PE(16:0/0:0) (0.0149), DG(16:0/16:0) (0.0133)—point to active acyl editing (Lands cycle) and Kennedy-pathway flux. Practically, this is the ER/oil-body assembly line: continuous PC↔LPC/LPE cycling and a healthy DAG pool let the endosperm expand membranes and build oil-body monolayers at high throughput—directly supporting higher TGW.

%\emph{Oil deposition (dense mass)} A suite of TAGs—TG(14:0/18:1/18:2) (0.0160), TG(18:2/18:2/18:4) (0.0160), TG(12:0/12:0/18:1) (0.0141), TG(16:0/16:0/18:2) (0.0140), TG(16:1/20:1/20:2) (0.0134), TG(16:0/18:3/18:3) (0.0127), TG(16:0/18:2/18:3) (0.0117), TG(14:0/16:0/16:0) (0.0128)—tracks oil-body load and composition. Heavier seeds correlate with TAG profiles enriched for PUFAs (storage density) with some medium/long-chain balance that likely reflects efficient packaging and synthesis.

%\emph{Bioenergetics and plastid support} Cardiolipin CL(18:1) (0.0122) flags mitochondrial respiratory capacity, i.e., ATP supply for biosynthesis during filling. Plastid lipids SQDG(16:0/14:0) (0.0139) and SQDG(18:1/18:3) (0.0130) tie TGW to chloroplast function in maternal/green tissues and developing caryopses—consistent with continued photoassimilate supply and fatty-acid synthesis during grain fill.

%\emph{Signaling & turnover} MG(18:2) (0.0171) and MG(18:0) (0.0131) indicate active lipase/transferase cycles that feed DAG/TAG synthesis and remodel membranes, while phytosphingosine (0.0128) hints at sphingolipid-mediated stress/quality control during storage deposition.

%Heavy seeds come from lines that assemble oil bodies fast (PC↔lyso-PC/LPE, DAG flux), pack a lot of TAG, and defend those PUFAs with a strong antioxidant network (α-tocopherol ± carotenoids/CoQ), powered by solid mitochondrial and plastid capacity (CL, SQDG).


%\subsubsection{Grain yield per primary panicle:} 
%What “yield” looks like in lipids. Yield integrates the whole pipeline—source photosynthesis, developmental timing, sink capacity, and protection. Your RF + TreeSHAP profile reflects exactly that blend: plastid source strength, developmental coordination, membrane machinery, and targeted carbon allocation.

%\emph{Photosynthetic source strength (plastid capacity)} High importance for chloroplast lipids—DGDG(18:0/18:3) and SQDG(16:1/18:3)—together with β-carotene points to carbon fixation and thylakoid function as rate-limiting for final yield. MGDG(16:0/18:1) and Ubiquinone-9 (mitochondrial redox) reinforce the idea that robust plastid/mitochondrial energy systems underpin strong source supply to the panicle sink.

%\emph{Developmental signaling & coordination (the top signal)} α-Ionone is the #1 feature—an apocarotenoid volatile from carotenoid cleavage—consistent with apocarotenoid signaling coordinating development and resource allocation. Alongside gibberellic acid, LPE(16:0), and MG(20:4), the model captures a hormone–lipid signaling axis that likely tunes spikelet/grain set and filling dynamics.

%\emph{Cellular machinery & protection (build fast, keep it stable)} Strong PE/DG signals—PE(16:0/0:0), PE(16:0/20:4), PE(16:0/18:0), DG(16:0/18:0), DG(16:0/18:2)—indicate membrane biogenesis and acyl-editing throughput (Kennedy/Lands cycles) as constraints during filling. β-Sitosterol and phytosphingosine point to membrane stability and quality control—keeping expanding endomembranes and oil-body monolayers functional under oxidative load.

%\emph{Strategic carbon allocation (reproductive-biased TAGs)} The TAGs associated with yield—e.g., TG(18:0/18:2/22:0), TG(16:0/18:1/22:1), TG(16:0/16:0/18:1), TG(14:0/16:0/18:1), TG(18:1/18:3/22:0)—skew toward longer-chain/PUFA mixes typical of reproductive tissues and developing seeds. This looks less like generic “bulk fuel” (plant height) and more like targeted provisioning to the panicle for efficient oil/starch deposition.

%One-liner. High-yield lines pair strong plastid/mitochondrial source capacity with apocarotenoid/hormonal coordination, high-throughput membrane assembly & protection, and targeted TAG provisioning to the panicle.



\begin{figure}[!ht]
  \centering
  \includegraphics[width=\linewidth]{fig/main/Fig4.png}
  \caption{\textbf{Random-forest pipeline: residualization, model performance, and SHAP-based interpretation.}
  \textbf{A}, Kernel-density plots of the phenotype (example: plant height) before (purple) and after PC residualization (yellow). Residualization collapses population-structure variation and recenters values near zero while retaining spread/ordering needed for prediction. 
  \textbf{B}, Example lipid (TG(10:0/10:0/10:0)) before (purple) and after PC residualization (yellow). Changes are modest relative to the phenotype, indicating that lipid variance is largely biological once PC effects are removed. 
  \textbf{C}, Five-fold cross-validation on the training set for the tuned random forest (1{,}000 trees); points show per-fold RMSE, MAE, and $R^2$, with dashed lines marking fold means, demonstrating stable generalization across folds. 
  \textbf{D}, Held-out test performance: RF predictions versus observed residual phenotype, with the 1:1 line shown (dashed). The model attains RMSE $\approx 24.2$ and $R^2 \approx 0.69$. 
  \textbf{E}, TreeSHAP interpretation of the final model. Left, beeswarm: each point is a sample’s SHAP value for the top features (color encodes standardized abundance); positive values increase the predicted residual phenotype. Right, global ranking by mean absolute SHAP identifies the most phenotype-informative lipids (notably several short/medium-chain TG species alongside selected membrane lipids such as PC/PE, MGDG, and SQDG). 
  Together, the panels show that PC residualization removes confounding, the RF model generalizes, and SHAP pinpoints the lipid species that most strongly drive predictions.}
  \label{fig:fig4_rf_shap}
\end{figure}





\subsection*{Integrated LION, LINEX2, and GWAS Analyses Reveal Lipid Remodeling Under Lowinput}
Across three orthogonal analyses, reaction topology (LINEX2), lipid ontology (LION), and quantitative genetics (GWAS), LI consistently points to lipolysis–re-acylation that reallocates acyl chains from phospholipid membranes into neutral triglyceride storage while plastids tune their anionic surface by turning over SQDG. In the LINEX2 sub-network (Fig.\ref{fig:Fig5} \textbf{A–B}), the most connected transitions form a TG$\leftrightarrow$DG core coupled to acyl-editing links between phospholipids and neutral lipids (e.g., \mbox{PE + DG $\rightleftharpoons$ LPE + TG}, \mbox{PC + DG $\rightleftharpoons$ LPC + TG}, \mbox{TG $\rightleftharpoons$ DG} and \mbox{DG + MG $\leftrightarrow$ TG}) and to a plastid branch that interconverts DG and SQDG via sulfoquinovose (e.g., \mbox{SQDG  $\rightleftharpoons$ DG}). LION enrichment reinforces this same architecture in regards to terms related to glycerolipids, lipid droplets, and neutral headgroups which are elevated in LI, whereas multiple glycerophospholipid classes and membrane biophysical signatures (intrinsic curvature, lateral diffusion, transition temperature) trend downward (Fig.\ref{fig:Fig1_lipid_class} \textbf{B}). The class/subclass compositions match these directions. In Fig.\ref{fig:Fig1_lipid_class} \textbf{A}, TG increases (2.7\%$\rightarrow$5.4\%) with a modest DG rise (13.8\%$\rightarrow$14.8\%), PE rises (5.8\%$\rightarrow$6.6\%), while plastid SQDG decreases (2.9\%$\rightarrow$1.4\%), MGDG decreases (34.5\%$\rightarrow$32.1\%), PC shows a mild decrease (25.6\%$\rightarrow$24.8\%), DGDG is essentially flat (12.7\%$\rightarrow$13.0\%). Subclass bars (Fig.\ \textbf{S5 B–C}) also reiterates this. Glycerolipids show TG up (4.0\%$\rightarrow$7.9\%) and DG up (20.3\%$\rightarrow$21.9\%) with MGDG and SQDG down (50.6\%$\rightarrow$47.3\%; 4.3\%$\rightarrow$2.0\%), and the phospholipid pool tilts from PC toward PE (PC 79.9\%$\rightarrow$76.9\%; PE 18.4\%$\rightarrow$20.8\%), alongside small PG/LPC increases (LPE, PA $<0.1\%$). Together, these orthogonal signals point to acyl flux from structural phospholipids (especially PC) and plastid galactolipids/sulfolipids toward neutral TG, with PE and lysophospholipid edits mediating the transfer.

Morever, GWAS identifies genes directly onto the enriched edges. A patatin-like lipase, \textit{PNPLA1} (SORBI\_3001G041900), associates with twelve TG species (e.g., TG(18{:}1\_18{:}2\_18{:}3), TG(16{:}0\_18{:}1\_22{:}0)), fitting the TG$\rightarrow$DG$+$FA arm of the cycle and explaining the slight DG accumulation that feeds downstream edits or plastid demand (ref). Re-esterification back to TG is anchored by \textit{DGAT1}/TAG1 (SORBI\_3010G170000) and \textit{DGAT3} (SORBI\_3009G034600), whose hits span MG, DGDG(16{:}0\_18{:}3), and multiple TGs. The \textit{DGAT3} bias toward polyunsaturated TGs is consistent with the smooth trends versus chain length/unsaturation (Fig.\ \textbf{C–D}) characteristic of storage-oriented remodeling. On the plastid branch, \textit{SQD2} (SORBI\_3001G427300) maps to SQDG(16{:}0\_16{:}0 / 16{:}0\_18{:}1 / 16{:}0\_18{:}3) (and one TG), exactly the DG$\rightarrow$SQDG step; the observed SQDG decrease with low-input suggests heightened turnover/substitution that spares phosphate and adjusts the thylakoid surface charge while competing with DGAT for DG. A \textit{PLD} locus (SORBI\_3005G222500) linked to DG and TG species provides a membrane-to-DG route via PA$\rightarrow$DG (PAP), supplying substrate both for DGAT (storage) and \textit{SQD2} (plastid). Finally, two broad LCAT-like associations—\textit{LCAT-like 4} (SORBI\_3006G214500; 22 hits across MG, DG, TG, PC/PE/PG, MGDG/DGDG, SQDG) and \textit{LCAT-like 1} (SORBI\_3001G103800; PC and TG hits)—fit the lysophospholipid acyl-shuttling implicit in the \mbox{LPC $\leftrightarrow$ DG} and \mbox{PG + DG $\rightleftharpoons$ LPC + TG} edges and help explain the modest PC$\downarrow$/PE$\uparrow$ tilt.

In sum, low-input triggers a coherent remodeling program in which PNPLA1-mediated TG breakdown, DGAT1/3-mediated TG synthesis, PLD/PAP-derived DG supply, LCAT-like acyl editing, and plastid \textit{SQD2} draw on the same DG node. The net effect is redistribution of acyl chains from phospholipids and plastid galacto/sulfolipids into neutral TG, with plastids concurrently substituting SQDG for phosphate-demanding anionic phospholipids. The integrated evidence—enriched reaction paths (Fig.\ \textbf{A–B}), ontology shifts (Fig.\ \textbf{B}), class/subclass compositions (Fig.\ \textbf{1A}, \textbf{S5B–C}), and species-level smooths (Fig.\ \textbf{C–D})—all point to this same lipolysis–re-acylation switch and SQDG cycling as hallmarks of the low-input state.


\begin{table}[!ht]
  \centering
  \footnotesize
  \setlength{\tabcolsep}{4pt}
  \renewcommand{\arraystretch}{1.2}
  \caption{\textbf{Key enriched reactions in the LINEX2 sub-network (low-input vs.\ control)}}
  \label{tab:linex_reactions}
  \begin{tabularx}{\linewidth}{@{}%
      p{0.18\linewidth}
      p{0.22\linewidth}
      p{0.26\linewidth}
      X
    @{}}
    \toprule
    \textbf{Pathway} & \textbf{Stoichiometry} & \textbf{Putative enzyme(s)} & \textbf{Interpretation (low-input)} \\
    \midrule
    Lipolysis axis
      & DG $\rightarrow$ MG + FA
      & LIPE-like lipase
      & Provides MG for re-esterification or signalling. \\
    & MG + FA $\rightarrow$ TG
      & \textit{PNPLA3} (triacylglycerol synthase)
      & \textbf{$\downarrow$ flux}: storage synthesis suppressed. \\
    & TG $\rightarrow$ DG + FA
      & \textit{PNPLA1} (SORBI\_3001G041900)
      & \textbf{$\uparrow$ lipolysis}: dominant driver of DG pool. \\
    \addlinespace
    Phospholipid recycling
      & PE + DG $\rightleftharpoons$ LPE + TG
      & LRO1-type acyltransferase
      & Membrane PE shuttles acyl chains to TG. \\
    \addlinespace
    Plastid sulfolipid (SQDG) cycling
      & DG + UDP–sulfoquinovose $\rightarrow$ SQDG + UDP
      & \textit{SQD2} (sulfoquinovosyldiacylglycerol synthase)
      & Builds anionic sulfolipid in thylakoid membranes; phosphate-sparing replacement of phospholipids. \\
    & SQDG + H$_2$O $\rightarrow$ DG + sulfoquinovose
      & Putative sulfoquinovosidase (\textit{YihQ}-like)
      & SQDG turnover can release DG for TG/phospholipid remodeling; adjusts anionic lipid pool under stress. \\
    \addlinespace
    Alternative TG conversions
      & PE $\rightarrow$ DG + PI (acyl transfer)
      & EPTB (phosphoethanolamine phosphotransferase)
      & Recycles PE headgroups into TG $\Rightarrow$ DG cascade. \\
    & TG $\rightarrow$ RHEA:32843
      & Unspecified lipase
      & Alternative TG hydrolysis branch. \\
    & LPC $\leftrightarrow$ DG
      & Unspecified transferase
      & LPC\,$\leftrightarrow$\,DG interconversion at droplet surface. \\
    \bottomrule
  \end{tabularx}
\end{table}



\begin{figure*}[t]
  \centering
  \includegraphics[width=\textwidth]{fig/main/Fig5.png} % <- your composite with A–D
  \caption{\textbf{Network enrichment and chain/unsaturation trends under low input.}
  \textbf{(A)} LINEX$^2$ lipid–enzyme network enrichment using \emph{ratio} contrasts between lipid classes; nodes are lipids (circles) colored by class (MG, DG, TG, SQDG) and enzymes (triangles), edges are curated conversions. Node size scales with absolute enrichment score.
  \textbf{(B)} LINEX$^2$ enrichment using \emph{absolute differences} (LowInput$-$Control) at the species level; same glyphs and legend as in (A).
  \textbf{(C)} Condition effect versus total chain length: points are individual lipid species and the blue line is a LOESS smoother ($\pm$~95\% CI in gray) of $\Delta Z = Z_{\mathrm{LowInput}} - Z_{\mathrm{Control}}$ plotted against the sum of acyl carbons. The curve dips below zero for mid chain lengths (C\,$\sim$\,34–50) and rises toward very long chains (C\,$\sim$\,56–60), consistent with loss of membrane lipids and relative enrichment of long-chain TAG. 
  \textbf{(D)} Condition effect versus total unsaturation (sum of double bonds). The downward trend indicates preferential depletion of highly unsaturated species under low input. 
  For (C–D), intensities were TIC-normalized, log-transformed with a small sample-wise pseudocount, $Z$-scored within condition and lipid, and then differenced (LowInput$-$Control) per species; the smoother was fit with LOESS (span~0.9). MG, DG, TG, SQDG: mono/di/triacylglycerol and sulfoquinovosyldiacylglycerol.}
  \label{fig:Fig5}
\end{figure*}







\subsection*{Integrated Lipid‐Metabolism Network, Ontology Enrichment, and GWAS Analaysis Idenfies Key Lipid Transitions Under Low Input}


\noindent \textbf{Network-level shifts (LINEX2).}
The reaction sub-network enriched in low-input samples is dominated by a lipolysis–re-acylation axis that shuttles acyl chains between membranes and storage (Fig.\ \textbf{A–B}). Three features stand out:
\begin{enumerate}\itemsep3pt
  \item \textbf{TG $\leftrightarrow$ DG cycling.} Reactions converting triacylglycerol (TG) to diacylglycerol (DG) {+} free fatty acid (FA), and the reverse re-esterification of MG/DG back to TG, are over-represented, indicating active remodeling of the neutral-lipid pool.
  \item \textbf{Phospholipid $\leftrightarrow$ neutral-lipid crosstalk.} The edge \mbox{PE + DG $\rightleftharpoons$ LPE + TG} suggests a PE-based acyl-transfer route that moves acyl chains from membrane glycerophospholipids into TG (a Lands–cycle–like edit). The \mbox{LPC $\leftrightarrow$ DG} connection is consistent with lysophospholipid acyl transfer at the droplet surface.
  \item \textbf{Plastid anionic-lipid cycling.} \mbox{DG + UDP–sulfoquinovose $\rightarrow$ SQDG + UDP}, and the reverse hydrolytic route \mbox{SQDG + H$_2$O $\rightarrow$ DG + sulfoquinovose}, indicate dynamic sulfoquinovosyldiacylglycerol (SQDG) turnover—i.e., plastid membranes swapping phosphate-containing anionic lipids for SQDG and feeding DG back to the glycerolipid pool when needed.
\end{enumerate}

\noindent \textbf{Ontology-level consequences (LION).}
LION enrichment mirrors the network picture (Fig.\ \textbf{B}). Terms increased in low-input include \emph{glycerolipids} (DG/TG), \emph{lipid droplet}/\emph{lipid storage}, and \emph{headgroup with neutral charge}, consistent with accumulation of neutral storage lipids. Terms decreased include \emph{glycerophospholipids} (PC/PE subclasses) and membrane biophysical signatures (e.g., intrinsic curvature, lateral diffusion/transition-temperature terms), pointing to depletion or acyl editing of structural phospholipids. Together, these ontologies argue for redistribution of acyl chains from zwitterionic/anionic membrane lipids into storage TG, alongside plastid-specific adjustments.

\noindent \textbf{Genetic anchors (GWAS) that map onto the same reactions.}
GWAS hits pin enzymes to the enriched edges and explain species-level patterns:
\begin{itemize}\itemsep3pt
  \item \textbf{TG lipolysis $\rightarrow$ DG + FA: \textit{PNPLA1} (SORBI\_3001G041900).} Associations to 12 TG species (e.g., TG(18:1\_18:2\_18:3), TG(16:0\_18:1\_22:0)) implicate a patatin-like lipase in the TG$\rightarrow$DG arm highlighted by LINEX2, shifting the DG/TG balance and feeding DG into downstream editing or SQDG synthesis.
  \item \textbf{DG + acyl–CoA $\rightarrow$ TG: \textit{DGAT1}/TAG1 (SORBI\_3010G170000) and \textit{DGAT3} (SORBI\_3009G034600).} Hits spanning MG, DGDG(16:0\_18:3) and multiple TGs (with \textit{DGAT3} biased to polyunsaturated TGs) anchor re-esterification back to TG and help explain the chain/unsaturation smooths (Fig.\ \textbf{C–D}).
  \item \textbf{DG + UDP–sulfoquinovose $\rightarrow$ SQDG + UDP: \textit{SQD2} (SORBI\_3001G427300).} Associations with SQDG(16:0\_16:0/16:0\_18:1/16:0\_18:3) (and one TG) fit the plastid branch; \textit{SQD2} builds sulfolipid in thylakoids, likely sparing phosphate and tuning anionic surface charge under low-input while competing with DGAT for the DG substrate.
  \item \textbf{PC/PE $\rightarrow$ PA + headgroup: \textit{PLD} (SORBI\_3005G222500).} Links to DG and TG species suggest PLD-derived PA, dephosphorylated to DG (via PAP), supplies DG for DGAT or for \textit{SQD2}—exactly the membrane-to-storage trafficking seen in LINEX2 and LION.
  \item \textbf{Lysophospholipid acyl transfer / acyl editing: LCAT-like enzymes.} 
  \textit{LCAT-like 4} (SORBI\_3006G214500; 22 hits across MG, DG, TG, PC/PE/PG, MGDG/DGDG, SQDG) and \textit{LCAT-like 1} (SORBI\_3001G103800; PC and TG hits) are consistent with phospholipid$\rightarrow$neutral-lipid acyl flow and the \mbox{LPC $\leftrightarrow$ DG} / \mbox{PG + DG $\rightleftharpoons$ LPC + TG} edges.
\end{itemize}

\noindent \textbf{Composition evidence (stacked bars) supports the model.}
Class-level composition (Fig.\ \textbf{1A}) shows \textbf{TG increase} (2.7\% $\rightarrow$ 5.4\%), \textbf{DG slight increase} (13.8\% $\rightarrow$ 14.8\%), \textbf{PE increase} (5.8\% $\rightarrow$ 6.6\%), and \textbf{SQDG decrease} (2.9\% $\rightarrow$ 1.4\%), with \mbox{MGDG decrease} (34.5\% $\rightarrow$ 32.1\%), mild \mbox{PC decrease} (25.6\% $\rightarrow$ 24.8\%), and DGDG $\sim$stable (12.7\% $\rightarrow$ 13.0\%). 
Subclass panels corroborate this: glycerolipids (S5B) show \textbf{TG up} (4.0\% $\rightarrow$ 7.9\%) and \textbf{DG up} (20.3\% $\rightarrow$ 21.9\%), with \textbf{MGDG down} (50.6\% $\rightarrow$ 47.3\%) and \textbf{SQDG down} (4.3\% $\rightarrow$ 2.0\%); phospholipids (S5C) show a \textbf{PC$\downarrow$ / PE$\uparrow$ tilt} (PC 79.9\% $\rightarrow$ 76.9\%; PE 18.4\% $\rightarrow$ 20.8\%), small PG and LPC increases, LPE/PA $<0.1\%$.

\noindent \textbf{Synthesis and interpretation.}
Across three orthogonal layers, low-input drives a coherent remodeling program: (\emph{i}) enhanced TG$\leftrightarrow$DG cycling; (\emph{ii}) PE-mediated acyl transfer into TG and LPC/DG interconversion; and (\emph{iii}) plastid SQDG cycling that competes with DGAT for DG. The composition and ontology shifts (neutral lipids and droplets up; phospholipid/biophysical signatures down; SQDG/galactolipids reduced) and the genetic anchors (\textit{PNPLA1}, \textit{DGAT1/3}, \textit{SQD2}, \textit{PLD}, LCAT-like enzymes) converge on the same mechanism. We propose that, under low-input, plants reallocate acyl chains from structural phospholipids to storage TG while plastids substitute SQDG for phosphate-demanding anionic phospholipids. The species-level smooths versus chain length and unsaturation (Fig.\ \textbf{C–D}) are consistent with storage-oriented remodeling into variably unsaturated, sometimes longer-chain TGs, with relative depletion/editing of membrane PCs/PEs and plastid galactolipids.








%--------------------------------------------------------------------
\bibliographystyle{plainnat}
\bibliography{lipid_refs}



\section*{Discussion}

\subsection*{Sphingolipid Structural and Signaling Reprogramming Under Low Input Stress}
The proportion of sphingolipids increases under LI (17.3\%) compared to C (0.7\%), which is observed in cold acclimation responses observed in plants (ref), although such a high increase has not been reported. Across various species, exposure to cold typically enhances total sphingolipids, especially complex glycosylated forms such as GIPCs, while reducing glucosylceramides (GalCer) (Supplementary Figure \ref{fig:S5}) and selectively remodels long-chain bases via $\Delta$8 desaturases to maintain membrane fluidity at low temperatures  (ref). Although our treatment involves nutrient limitation (N and P) also rather than just cold exposure, the increased abundance of sphingolipids under LI conditions implies that nutrient-limited conditions may initiate a broader stress-related function in which sphingolipids function both as structural stabilizers and as signaling hubs. The stress-dependent modulation of ceramide pools and their phosphorylated derivatives (e.g., phytosphingosine-P, ceramide-P) provides a mechanism for influencing cell-fate decisions and transcriptional responses. Nitric oxide signaling is known to interact with these phosphorylation cycles and could contribute to the LI response (ref). Similarly, lipid signaling nodes associated with nutrient stress, particularly PLD/PA dynamics under P or N limitation, function in the same membranes and are likely to coordinate with sphingolipid homeostasis during remodeling. Thus, the observed increase in sphingolipids under LI conditions aligns with a conserved acclimation strategy enhancing membrane integrity and modulating lipid-mediated signaling to reduce the effects of stress while integrating nutrient-responsive pathways. 

\subsection*{Galactolipid-Mediated Chloroplast Remodeling and Stress Signaling}
Galactolipids exhibit relative compositional stability in both C and LI (SuppTable). In DGDG, the predominant 18:2/18:4 species accounts for approximately 63.9 to 65.4\%, while the 16:0/18:3 species experiences a modest decrease from 27.6 to 24.2\%. In MGDG, the 18:3/18:3 species slightly increases from 87.3 to 89.3\%, with other species remaining relatively minor. Such constancy aligns with the predictions found in existing literature regarding chloroplast membrane composition (ref). MGDG, characterized by its cone-shaped and non-bilayer forming propensity, contributes to the curvature necessary for stacked thylakoids, whereas DGDG, which forms bilayers, provides lamellar stabilization (ref). Both MGDG and DGDG are associated with PSI/PSII complexes, leading to a strong functional constraint on their local composition (galactolipid\_chloroplast.pdf; galactolipid\_photosynthesis.pdf). Changes in these lipid pools due to stress result in severe phenotypic consequences. \textit{Mgd1} mutation nearly eradicates MGDG, producing albino and photosynthesis defective embryos, whereas the \textit{dgd1} mutation disrupts DGDG, reduces PSII quantum yield, causes chloroplasts to round with regions devoid of thylakoids, and incites jasmonate overproduction (galactolipid\_photosynthesis.pdf; galactolipid\_chloroplast.pdf). An increased MGDG:DGDG ratio is alone sufficient to provoke jasmonate accumulation and alter chloroplast morphology, while a partial restoration of this ratio ameliorates growth and chloroplast structure (galactolipid\_chloroplast.pdf). Thus, there is an active homeostatic mechanism regulating both the MGDG:DGDG ratio and the pronounced 18:3 enrichment of the plastid galactolipid acyl profile.

Under nutrient stress, plants therefore may remodel elsewhere first. During phosphate limitation, DGDG expands into extraplastidic membranes while thylakoids preserve their MGDG/DGDG balance. Similarly, plastids adjust surface charge mainly via SQDG turnover rather than by altering galactolipids (galactolipid\_chloroplast.pdf). Cold and other stresses commonly tweak unsaturation to maintain fluidity rather than overhaul headgroup composition, which is consistent with our slight enrichment of MGDG(18:3/18:3) with compensatory, tiny shifts in minor species. Thus, despite different fields/experiments, our near-identical MGDG/DGDG (0.37 vs./ 0.41) profiles are a proof-of-concept for this model. Plants protect the load-bearing plastid galactolipid scaffold that supports photosystems and regulates JA, while shunting LI remodeling into extraplastidic pools (e.g., PC/PE acyl editing, DG/TG buffering) and anionic tuning via SQDG. 

NOTE:: So here, I can explain about the others PCs DGs/TGs as well. but explained below. everything is related. i think we can have like an overall mechanism at the end.  


%References:

%Chun-Wei Yu et al., "Increased ratio of galactolipid MGDG : DGDG induces jasmonic acid overproduction and changes chloroplast shape," New Phytologist, 2020, galactolipid\_chloroplast.pdf, pp.1-9.
%Koichi Kobayashi et al., "Galactolipid synthesis in chloroplast inner envelope is essential for proper thylakoid biogenesis, photosynthesis, and embryogenesis," PNAS, 2007, galactolipid\_photosynthesis.pdf, pp.1-6.


\subsection*{Phospholipid Species Turnover Suggests Active Acyl Editing Under LowInput}

Several independent analyses indicate higher PE turnover (relative to PC) in LI. LPE ratios increase considerably such as, LPE/MGDG shifts by  $+0.56 \,\log_{10}$ ($\approx 3.7\times$), with Cliff’s $\delta \approx +1$ and probability of superiority (AUC $\approx 100\%$), i.e., near-complete separation of LI from C. LPE/PS shows a similarly positive shift with $q \ll 10^{-3}$. In contrast, LPC ratios decline (LPC/MG, LPC/PS, LPC/LPE, LPC/PE, LPC/PG; medians $-0.53$ to $-0.02 \,\log_{10}$), with 
$\delta \approx -1$ and AUC $\approx 0\%$, placing almost every LI sample below nearly every C sample. 
Similarly, DG/LPE decreases ($\delta \approx -1$), indicating more LPE relative to DG in LI. 
These statistics argue for a systematic tilt toward the ethanolamine branch of acyl-editing 
rather than random variability or batch effects.

The alterations in lipid composition are in agreement as well. Within the class of phospholipids, PC exhibits a decrease ($\sim 79.9\% $ to $ \sim 76.9\%$), whereas PE shows an increase ($\sim 18.4\% $ to $ \sim 20.8\%$) under LI conditions (Supplementary Figure \ref{fig:S5}). On a broader total lipidome, PC demonstrates a decline ($25.6\% $ to $ 24.8\%$), while PE displays an increase ($5.8\% $ to $ 6.6\%$). The absolute quantities of LPC and LPE remain minor ($\sim 0.3$–$0.5\%$ and $<0.1\%$, respectively). Hence, the analysis of ratio distributions, which are evaluated on the $\log_{10}$ scale and scored according to $z$, serves as a more sensitive indicator of inherent turnover.

Taken together, LPE $\uparrow$, LPC $\downarrow$, DG/LPE $\downarrow$, large $|\log_{10}|$ effects with $\delta$ near $\pm 1$ and AUC spanning 0–100\%, plus PC $\downarrow$/PE $\uparrow$, the LI lipidome is consistent with a PE-centered Lands’ cycle (i.e., relatively greater formation and rapid consumption of LPE compared with LPC). This is consistent with a PE-biased acyl-editing equilibrium. The convergence of statistics and class shifts supports the interpretation. Moreover, GWAS also provides two key genes for the mechanism. Two loci identifying as lysophospholipid acyltransferase (LPEAT-like) demonstrate significant signals. On chromosome 3, \textit{SORBI\_3003G145900} (UniProt C5XIQ8; LPEAT1-like, PTHR23063) is associated with PC(18:2\_20:4) (most significant SNP\_14996372, $p = 4.20 \times 10^{-8}$). On chromosome 9, \textit{SORBI\_3009G108500} (UniProt A0A1B6P8D4; LPEAT1-like, PTHR23063) is associated with Sum\_DG/Sum\_MG and Sum\_GalCer/Sum\_MG (most significant SNP\_43518103, $p = 6.90 \times 10^{-9}$ and $5.48 \times 10^{-8}$). Given that LPEAT catalyzes the conversion from LPE $\rightarrow$ to PE, allelic variation at LPEAT-like loci is anticipated to modulate the LPE$\leftrightarrow$PE equilibrium. This modulation, in conjunction with the Kennedy pathway (DG + CDP-ethanolamine $\rightarrow$ → PE), is also expected to influence the DG reserves. Under such circumstances, DG levels may be reduced compared to MG during the lipolysis-re-acylation process, making DG/MG-based phenotypes a plausible indicator of PE-associated remodeling. Thus, the signals from LPEAT are perceived as consistent with representations of the LI ratio signature (LPE $\uparrow$, DG/LPE $\downarrow$, PC $\downarrow$/PE $\uparrow$), with the understanding that an association does not equate to causal inference.




Lipid Class Remodeling under Low-Input Stress
Low-input conditions (e.g. nutrient limitation) trigger broad shifts in the plant lipidome relative to controls. Glycerolipids remain the dominant lipid category but their overall share of total lipids drops (from ~64\% in controls to ~52\% under stress)[1]. Membrane glycerophospholipids similarly decrease slightly (~30\% to 25\%), whereas sphingolipids surge dramatically from a trace (~0.2\% of total) to ~17\% under LowInput stress[1]. This indicates a substantial enrichment of sphingolipid content during stress, likely contributing to more rigid, high-phase-transition membranes. Minor lipid classes (sterols, betaine lipids, free fatty acids, etc.) remain a small fraction (together ~5–7\%) with only subtle changes[2][3]. These profile shifts suggest a reallocation of lipid resources: neutral storage lipids expand at the expense of some membrane lipids, and stress-induced sphingolipid accumulation may reinforce membrane structural integrity.

At the subclass level, the LowInput regime clearly favors neutral acylglycerols over chloroplast galactolipids. Triacylglycerols (TG) nearly double in relative abundance (rising from ~2.7\% to 5.4\% of total ion signal), accompanied by a modest increase in diacylglycerols (DG, ~13.8\% to 14.8\%)[4]. Conversely, chloroplast galactolipids show attenuation: monogalactosyldiacylglycerol (MGDG) declines from ~34.5\% to 32.1\%, and sulfoquinovosyldiacylglycerol (SQDG, a sulfur-containing plastid lipid) drops by roughly half (2.9\% to 1.4\%), while digalactosyldiacylglycerol (DGDG) remains essentially unchanged[5]. In other words, the glycerolipid profile shifts from being dominated by thylakoid galactolipids to containing a higher proportion of neutral storage lipids[3][6]. Glycerophospholipid composition is largely preserved – phosphatidylcholine (PC) stays the major phospholipid (~80\% of the phospholipid pool in both conditions), phosphatidylethanolamine (PE) increases slightly (from ~18\% to 21\%), and phosphatidylglycerol (PG) remains minor (~1.3\%→1.5\%)[7]. Notably, lysophospholipids (lysophosphatidylcholine, LPC; and lysophosphatidylethanolamine, LPE) and other minor phospholipids (phosphatidic acid, phosphatidylserine) persist at very low levels (<0.5\%), indicating that the bilayer-forming lipid core is maintained despite the stress[8]. Overall, the low-input condition induces a conserved membrane architecture (PC-rich bilayers) but with a parallel redirection of fatty carbon into storage lipids (DG/TG), and a reduction in chloroplast glycolipids, reflecting a shift from membrane maintenance toward energy-rich neutral lipid reservoirs[9][10].
Importantly, specific lipid subclasses show signatures of active remodeling via acyl editing. Under LowInput stress, the LPC pool becomes more diverse in its acyl composition: the dominant LPC(16:0) species loses relative abundance (dropping from ~73\% to ~63\% of the LPC pool), while previously minor species like LPC(18:3) and LPC(18:2) emerge or increase (with one new LPC species reaching ~6.9\% of the LPC fraction, whereas it was undetected in controls)[11]. This broadening of the LPC profile is a classic hallmark of phospholipase A₂ activity cleaving various acyl chains from PC/PE, producing a spectrum of lyso-phospholipids. In contrast, the monoacylglycerol (MG) pool loses diversity and becomes dominated by just a few species under stress: several low-abundance MGs present in control (e.g. MG(16:1), MG(18:0), MG(20:4)) drop below detection in LowInput samples[12]. The remaining MG signal concentrates in three high-flux species – MG(18:3), MG(18:1) and MG(12:0) – which together constitute the bulk of the MG pool[13]. This contraction of MG diversity, coupled with the expansion of LPC species, strongly suggests an accelerated Lands’ cycle-like lipid turnover: phospholipids are deacylated by phospholipases to yield various LPCs, and the freed acyl groups are rapidly re-esterified through MG intermediates into DG and TG. Indeed, the near-disappearance of many MG species implies MG is a transient intermediate that is quickly consumed by acyltransferases (MGAT and DGAT enzymes) to form DG and eventually TG[14]. Consistent with this, we observe net increases in DG and TG levels under stress, reflecting a rerouting of fatty acyl flux away from membrane phospholipids and into neutral lipid storage[9]. In summary, LowInput stress elicits a coordinated remodeling of lipid classes: membrane structural lipids (phospholipids and galactolipids) are partially sacrificed or modified (with modest PC↓, MGDG↓, SQDG↓) in favor of accumulating storage triacylglycerols and stress-associated sphingolipids, with lysolipid accumulation and MG depletion signaling an active lipid recycling mechanism to maintain essential membranes while reallocating carbon and nutrients.
GWAS Associations Reflecting Stress-Adaptive Lipid Traits
Genome-wide association studies (GWAS) in this sorghum panel highlight genetic loci linked to the observed lipid changes, often corresponding to known stress-response pathways. Notably, a MYB-like transcription factor gene (SORBI\_3001G384300) was repeatedly identified in associations with multiple phospholipid traits[15][16]. This gene is homologous to the phosphate-starvation response regulator PHR1 in other plants, a master controller of Pi homeostasis. Its allelic variation showed significant linkage with several phosphatidylcholine (PC) molecular species (e.g. PC 16:0/20:3, 16:0/22:5, 18:1/20:1, etc.)[17][18]. The implication is that natural variants in a phosphate-starvation signaling pathway (PHR1) modulate membrane phospholipid composition under low-input conditions. In other words, genotypes better able to sense or respond to Pi deficiency maintain distinct PC profiles, suggesting that phospholipid remodeling is genetically coupled to phosphorus stress adaptation[15][19]. This finding dovetails with known Pi-starvation lipid responses (phospholipid levels are tightly regulated as plants scavenge internal Pi from membrane lipids) and suggests that the LowInput treatment’s effect on phospholipids is at least partly orchestrated by P-starvation signaling networks.

Similarly, the GWAS pinpointed a key metabolic enzyme governing triacylglycerol accumulation: Acyl-CoA:diacylglycerol acyltransferase 1 (DGAT1, gene SORBI\_3010G170000). DGAT1 was identified as a significant hit in association with at least five distinct TAG species[20][21], emphasizing its central role in the formation of TAG under stress. DGAT1 catalyzes the final step of triacylglycerol biosynthesis (acylating DG to form TG) and is known to be upregulated during carbon overflow or stress conditions in plants[22]. The allelic variation in DGAT1 correlating with higher levels of specific TAGs (including polyunsaturated species like TAG 18:3/18:3/18:3) suggests that some genotypes have a greater capacity to channel excess fatty acids into storage lipids when nutrients are limited. Indeed, in Arabidopsis and other models, DGAT1 activity is pivotal for TAG accumulation in nitrogen deprivation and is induced by ABA signaling during stress[23][24]. The GWAS result thus reinforces that nitrogen limitation and cold-stress adaptation are linked to TAG metabolism: natural sorghum variants with more active DGAT1 alleles may accumulate more TAG in leaves, which is a strategy to remobilize and store carbon when growth is constrained by low N (or to protect membranes during chilling by sequestering fatty acids into TAG)[22][24]. In line with this, DGAT1 is also crucial for freezing tolerance; loss-of-function mutants fail to produce adequate TAG and suffer lipid imbalances (excess DG and PA causing ROS stress) under cold[24][25]. Thus, the genetic association of DGAT1 with TAG levels in our study directly ties a molecular mechanism of lipid plasticity (TAG synthesis) to stress resilience traits (N-deficiency survival and cold tolerance).
Additional GWAS hits further illustrate how lipid changes under LowInput are genetically connected to nutrient stress responses. For example, variation in a sulfur assimilation gene encoding adenylyl-sulfate kinase (APK3) was found to associate with the summed abundance of the sulfolipid SQDG[26][27]. SQDG serves as a phosphorus-free substitute for phospholipids in Pi-starved chloroplasts, and APK enzymes control the supply of activated sulfate (PAPS) for sulfolipid biosynthesis and other sulfated metabolites. The identification of APK3 (a cytosolic APS kinase isoform) in conjunction with SQDG levels suggests that sulfate metabolism genes can modulate sulfolipid accumulation[28][29]. Under low-input (likely low-P) conditions, genotypes with different APK3 activity may differ in how strongly they ramp up sulfolipid production to replace phospholipids and maintain membrane function. This aligns with the idea that phosphorus and sulfur nutrient pathways are co-regulated: PHR1 not only controls Pi-starvation responses but also affects sulfate uptake and allocation[30][31], and here we see both a P-sensing gene (PHR1 homolog) and an S-assimilation gene (APK3) emerging as determinants of membrane lipid remodeling. Together, these genetic links to P and S pathways underscore that lipidomic plasticity under nutrient stress is underpinned by classical nutrient homeostasis regulators (the plant “starvation” response toolkit).
Beyond nutrient stress, the GWAS approach captured loci tying lipid-related metabolites to broader stress adaptation and developmental processes. For instance, an alternative oxidase (AOX) gene was uncovered through associations with $\alpha$\-carotene levels in the panel[32][33]. While $\alpha$\-carotene is a carotenoid (not a membrane lipid), its levels and roles in photoprotection are intertwined with plastid lipid environment and oxidative stress. AOX in mitochondria provides an auxiliary electron sink to prevent over-reduction of the chloroplast under high light or drought stress, thereby protecting the photosynthetic machinery[32][34]. The co-variation of an AOX gene with a chloroplast carotenoid suggests that natural genetic variation in redox-balancing mechanisms can influence pigment (and possibly lipid) composition under stress. This highlights a broader integration: lipid changes (like thylakoid lipid saturation or xanthophyll cycle activity) work in concert with respiratory flexibility (AOX) to mitigate photooxidative damage[35][36]. In a similar vein, our GWAS of other stress-related metabolites (notably hormonal or photoprotective compounds) identified regulators like a MADS-box transcription factor affecting gibberellin-mediated flowering time, and a high-light inducible protein linked to zeaxanthin levels[37][38]. While these latter findings extend beyond membrane lipids, they reinforce the notion that the LowInput condition engages a network of genetic pathways. Many of these pathways (nutrient signaling, hormone responses, redox control) have downstream effects on or crosstalk with lipid metabolism. For example, delayed flowering under stress or enhanced NPQ (non-photochemical quenching) via carotenoids can be seen as adaptive reallocations of resources, akin to the lipid reallocations we directly observed. In summary, the GWAS results connect the lipidomic phenotypes to specific genes whose known functions (phosphate sensing, lipid storage synthesis, sulfur metabolism, oxidative stress mitigation) validate and enrich our understanding of the biochemical adaptation to low-input stress.
Molecular Network Insights into Lipid Remodeling Pathways
Molecular networking of the lipidomics data provided structural context to the myriad lipid changes, revealing that LowInput-induced alterations occur in coordinated clusters corresponding to biochemical pathways. A feature-based MS/MS network (constructed via GNPS or similar) grouped related lipid ions into clusters based on spectral similarity, effectively creating a map of lipid families and their interconversions. Within this network, a highly connected subcluster centered on diacylglycerol–triacylglycerol relationships emerged as a focal point of remodeling. This cluster contained the nodes for DG and TG species linked by intense connections, and it extended bridges to phospholipid and lyso-phospholipid nodes, mirroring known acyl-editing reactions. For example, the network shows DG–TG forming a core hub, with edges connecting DG to PC/LPC and PE/LPE in a cycle: reactions of the form PC + DG ↔ LPC + TG and PE + DG ↔ LPE + TG are indicated by the co-clustering of those species[39]. In essence, spectral evidence groups a PC and a TG with a DG and an LPC, suggesting that these compounds share fragment ions or neutral losses consistent with a biochemical transformation (loss/addition of an acyl chain). Likewise, DG and MG feature in edges like DG + MG ↔ TG, corresponding to MGAT/DGAT activity linking monoacylglycerols into the DAG–TAG pool[39]. The annotation propagation in this cluster was powerful: several unknown MS features could be putatively identified as triacylglycerols or related intermediates because they clustered with library-confirmed TAGs, all exhibiting the characteristic fatty acyl fragment pattern (e.g. neutral loss of C18:3 fatty acid). Thus, the network not only recapitulated the DG–TG axis of neutral lipid storage, but also enhanced compound identification by leveraging the shared fragmentation signatures within the cluster.

Another notable cluster in the molecular network corresponded to plastid lipid interconversions. Here, spectral features of sulfoquinovosyldiacylglycerol (SQDG) grouped alongside diacylglycerols, indicating a connection between SQDG and DG species[40]. This reflects the metabolic step of SQDG synthesis from DG: in plastids, the enzyme SQD2 transfers a sulfonate-containing headgroup onto DG to form SQDG. The network’s DG–SQDG adjacency (an edge SQDG ⇌ DG) is consistent with that known reaction, and in our data this connection was strengthened under LowInput. The presence of both SQDG and DG nodes in a cluster, with shared fragments (e.g. a common C16/C18 fatty acyl pair fragment), allowed annotation propagation as well – confirming the identity of several SQDG analogues by their clustering with known SQDG species. The network also revealed that SQDG-related nodes became relatively less intense in LowInput samples (in line with SQDG’s measured decrease), but the DG nodes in the same cluster remained, hinting that DG might accumulate or be diverted elsewhere (like into TG) instead of into SQDG under stress. This cluster-level view aligns with the biological interpretation that plastids under nutrient stress adjust anionic lipid content by modulating SQDG turnover. In effect, the molecular network visualizes a branch point: one route leads from DG into SQDG (sulfolipid for P-limited chloroplast membranes), and another route feeds DG into TG (storage or other uses), and LowInput conditions appear to favor the latter route.
Overall, the feature-based networking corroborated that lipid changes were not random but organized along metabolic pathways. Clusters of glycolipids (e.g. the thylakoid galactolipids MGDG/DGDG) showed tight grouping and relatively invariant connectivity between Control and LowInput, reflecting their conserved roles and minor compositional change. In contrast, the clusters associated with lipid catabolism and reassembly – such as the DAG/TAG cluster with its lyso-PC/PE links – were enriched for LowInput-specific changes (many nodes in these clusters showed increased intensity or new appearance in LowInput). This suggests that the stress condition activates entire modules of co-regulated lipid reactions. By propagating annotations across these modules, we identified sets of lipids moving in concert: for example, a dozen TG species and their corresponding DG partners all increased together, which is consistent with a systematic shift of equilibrium toward TAG formation. Likewise, a group of LPC species all rose in LowInput relative to their paired PCs, indicating a general increase in phospholipid turnover via PLA₂. Such patterns seen in the network reinforce the biochemical interpretation that a lipolysis–reacylation cycle is at play. The GNPS network essentially maps this cycle: one cluster connects phospholipids to lyso-phospholipids (lipase action) and to glycerides (acyltransferase action), encapsulating the Lands cycle and TAG pathway in graph form[39][41]. In summary, the molecular networking approach allowed us to visualize and verify that LowInput stress-induced lipidomic changes cluster into distinct pathways – notably a DG–TG–(L)PC axis and a DG–SQDG axis – and to spread known identifications to unknown yet related lipids within those pathways. This provides a deeper, pathway-centric understanding of the lipidomic remodeling, complementing the class-wise quantification.
Integrated Perspective: Lipid Plasticity, Genetic Loci, and Network Connectivity
All lines of evidence converge on a coherent model of lipidomic plasticity under LowInput stress. The chemical profile data show that under nutrient stress, plants reallocate fatty acids from membrane phospholipids and glycolipids into neutral storage lipids, while adjusting membrane composition to maintain function. Crucially, three orthogonal analyses – metabolic network topology, lipid ontology enrichment, and quantitative genetics – all point to the same mechanistic shift: a concerted lipolysis–re-esterification program that channels acyl chains away from structural phospholipids into triacylglycerols, coupled with plastid membrane remodeling to conserve phosphorus[42][39]. In the reaction network analysis, we identified a sub-network enriched in LowInput conditions dominated by the cycle of triglyceride breakdown and resynthesis: a DG–TG hub linked to edges representing phospholipase (PC→LPC) and acyltransferase (DG→TG) reactions[39]. Consistently, lipid ontology (LION) analysis found functional categories like “glycerolipids”, “lipid storage droplets”, and neutral lipid headgroups to be over-represented in the LowInput lipidome, whereas descriptors of fluid, polyunsaturated membranes and phospholipid-rich architecture were diminished[43][44]. This indicates the LowInput profile functionally skews toward thicker, more rigid membranes and carbon storage, aligning with the measured increases in DG/TG and decreases in polyunsaturated MGDG/SQDG. The direct lipid measurements agree: for example, TAG content doubles while MGDG and SQDG fall off under stress, and even within phospholipids a slight shift from PC toward PE (and a rise in lysophospholipid levels) is observed, signifying ongoing phospholipid editing[5][45]. Together, these pieces portray a stress adaptation strategy wherein membrane lipid degradation and recycling feed a buildup of energy-rich reserves and protective compounds. In parallel, the chloroplasts replace some phospholipids with sulfolipids (or at least turn over SQDG rapidly), presumably to spare phosphorus and modulate thylakoid charge properties[46][47]. The end result is a restructured lipidome that favors survival under nutrient scarcity: membranes become less fluid but remain intact, and excess energy is stored in TAGs rather than in membrane expansion.
Notably, the integrative approach allowed us to tie specific genes and enzymes to each major lipid transformation, reinforcing the causality of this remodeling program. Many of the top GWAS candidate genes map directly onto the reactions highlighted by the network analysis. For example, a patatin-like phospholipase (PNPLA1, SORBI\_3001G041900) was associated with a suite of TAG species[48], consistent with PNPLA1’s role in hydrolyzing triglycerides to release DG and fatty acids. This suggests that genotypic differences in TAG breakdown (the TG→DG + FA step of the cycle) can influence how much DG accumulates for re-routing into other lipids[49][50]. On the reassembly side, DGAT1 (identified earlier) and a related acyltransferase DGAT3 (SORBI\_3009G034600) were both significant GWAS hits, with alleles linked to multiple TG species as well as MG and even a DGDG species[51]. DGAT1/3 essentially anchor the DG→TG reaction; interestingly, DGAT3 showed a bias toward producing highly polyunsaturated TAGs, matching the observation that the new TAGs accumulated under stress tend to include C18:3-rich combinations[52][53]. This implies a genetically controlled preference for sequestering polyunsaturated fatty acids into TAG (possibly to protect them from peroxidation in membranes), a trait that would be beneficial during stress. On the plastid branch of the network, the enzyme SQD2 (sulfoquinovosyl diacylglycerol synthase, SORBI\_3001G427300) was identified as a candidate gene influencing SQDG levels[54]. Its locus was linked not only to several SQDG molecular species (e.g. SQDG 16:0/16:0, 16:0/18:3) but even to one TAG species[55], underscoring the competition for DG between the chloroplast SQDG pathway and cytosolic TAG pathway. The fact that SQDG decreases in LowInput while TAG increases suggests that limited DG might be preferentially driven toward storage rather than toward anionic membrane lipids when nutrients are scarce[56]. The SQD2 genetic association validates this trade-off, as natural variation in SQD2 could affect how aggressively a plant invests DG into sulfolipid versus allowing it for TAG production (with a more active SQD2 possibly buffering SQDG levels at the expense of TAG). Additionally, a phospholipase D (PLD) locus (SORBI\_3005G222500) was linked to DG and TG traits[57], aligning with the membrane-to-DG route in the network: PLD produces phosphatidic acid (PA) from PC, which is then dephosphorylated (by PAP enzyme) to yield DG. This PLD→PA→DG route supplies substrate that can be used either by DGAT for TAG synthesis or by SQD2 for SQDG synthesis[58]. The GWAS hit suggests that genetic differences in PLD/PAP activity modulate the flux of carbon from membrane phospholipids into the DG pool available for downstream remodeling. Finally, two LCAT-like acyltransferases (lecithin:cholesterol acyltransferase homologs) were found to have broad pleiotropic associations across many lipid classes[59]. One LCAT-like gene (SORBI\_3006G214500) showed 22 distinct lipid trait associations, spanning MG, DG, TG, multiple phospholipids (PC/PE/PG), galactolipids, and SQDG[60]. Another (SORBI\_3001G103800) was linked specifically to PC and TG levels[61]. These enzymes are hypothesized to participate in acyl editing, perhaps by re-acylating lyso-phospholipids (analogous to animal LCAT working on PC). Their broad impact fits the role of shuttling acyl groups between the lyso pools and DG/TG: in the network, they correspond to the LPC ↔ DG and PG + DG ↔ LPC + TG connections that facilitate exchange of acyl chains between phospholipids and neutral lipids[61][62]. The LCAT-like genes’ associations help explain subtle compositional shifts like the slight PC decrease and PE increase under stress, as enhanced acyl flux through lyso-PC could preferentially refill PE or other classes[63][45]. In summary, the GWAS findings map neatly onto the enzymatic steps of the lipid network deduced from the biochemical data. This cross-validation – seeing genetic evidence for PNPLA1, DGAT1/3, SQD2, PLD/PAP, LCAT, etc. – lends strong support to the model that low-input stress triggers a coordinated lipid recycling mechanism governed by these key regulators.
In conclusion, the integration of lipidomic profiles, GWAS loci, and network analysis paints a comprehensive picture of how sorghum adapts its lipid metabolism to low-resource conditions. LowInput stress induces a plastic reconfiguration of the lipidome: polyunsaturated membrane lipids are partially diverted into storage triacylglycerols (guarding against membrane damage and storing carbon), lysolipids accumulate as by-products of accelerated acyl turnover, and chloroplasts fine-tune their anionic lipids (like SQDG) to balance nutrient economy and membrane charge. This remodeling is not random but rather reflects an orchestrated response, as evidenced by enrichment of specific lipid ontologies and the emergence of a central DAG/TAG metabolic hub in the network[42][64]. The identification of multiple stress-related genes controlling these lipid changes – from nutrient signaling hubs (PHR1) to metabolic enzymes (DGAT, SQD2, PLD) – further demonstrates that lipidomic plasticity under stress is under genetic control and aligned with broader stress-response pathways. Essentially, natural genetic variation in sorghum appears to modulate the efficiency of this lipid rerouting program, which in turn may influence stress tolerance (e.g. genotypes that better preserve phosphate by altering lipids, or accumulate TAG to prevent free fatty-acid-induced damage, could have a survival advantage). The concurrence of biochemical data and GWAS implies that the lipid changes observed are adaptive, not merely symptomatic. Finally, by leveraging molecular networks, we validated that entire groups of functionally related lipids shift together, reinforcing the concept of lipid metabolic network resilience – the plant maintains critical membrane functions by dynamically reallocating lipid building blocks between compartments. This systems-level understanding, supported by multi-evidence cross-validation, reveals how low-input stress resilience in plants is tightly linked to agile lipid metabolism, and highlights candidate genetic regulators that could be targeted to improve stress tolerance via metabolic engineering or breeding[65][66]. Such insights underscore the power of integrating omics data: the coupling of lipidomic remodeling to genotype and network context has illuminated a metabolic survival strategy and the genes that underlie it, providing a robust framework for discussing plant stress adaptations at the lipid metabolism level.



\section*{Conclusion}

because we can For more information, see \nameref{S1_Appendix}.

\section*{Supporting information}

% Include only the SI item label in the paragraph heading. Use the \nameref{label} command to cite SI items in the text.
\paragraph*{S1 Fig.}
\label{S1_Fig}
{\bf Bold the title sentence.} Add descriptive text after the title of the item (optional).

\paragraph*{S2 Fig.}
\label{S2_Fig}
{\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 File.}
\label{S1_File}
{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Video.}
\label{S1_Video}
{\bf Lorem ipsum.}  Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Appendix.}
\label{S1_Appendix}
{\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\paragraph*{S1 Table.}
\label{S1_Table}
{\bf Lorem ipsum.} Maecenas convallis mauris sit amet sem ultrices gravida. Etiam eget sapien nibh. Sed ac ipsum eget enim egestas ullamcorper nec euismod ligula. Curabitur fringilla pulvinar lectus consectetur pellentesque.

\section*{Acknowledgments}
Me, myself and I

\nolinenumbers

% Either type in your references using
% \begin{thebibliography}{}
% \bibitem{}
% Text
% \end{thebibliography}
%
% or
%
% Compile your BiBTeX database using our plos2015.bst
% style file and paste the contents of your .bbl file
% here. See http://journals.plos.org/plosone/s/latex for 
% step-by-step instructions.
% 
\begin{thebibliography}{10}


\bibitem[Dall’Osto \emph{et~al.}(2012)]{DallOsto2012}
Dall’Osto, L., Cazzaniga, S., Bressan, M., Paleček, D., Židek, K., Jennings, R.~C., \& Bassi, R. (2012).  
Zeaxanthin protects plant photosynthesis by modulating chlorophyll triplet yield in specific light‐harvesting antenna subunits.  
\emph{Journal of Biological Chemistry}, 287(10), 6180–6190.

\bibitem[Demmig‐Adams \emph{et~al.}(2020)]{DemmigAdams2020}
Demmig‐Adams, B., Adams, W.~W., III, \& Holzwarth, A.~R. (2020).  
Zeaxanthin, a molecule for photoprotection in many different environments.  
\emph{Molecules}, 25(1), 100.

\bibitem[Guardini \emph{et~al.}(2020)]{Guardini2020}
Guardini, Z., Bressan, M., Caferri, R., Bassi, R., \& Dall’Osto, L. (2020).  
Identification of a pigment cluster catalysing fast photoprotective quenching response in CP29.  
\emph{Nature Plants}, 6, 1261–1273.


\bibitem[Williams and Morgan(1979)]{Williams1979}
Williams, E.~A., \& Morgan, P.~W. (1979).  
Floral initiation in sorghum hastened by gibberellic acid and far‐red light.  
\emph{Planta}, 145, 269–272.

\bibitem[Lee \emph{et~al.}(1998)]{Lee1998}
Lee, I.~J., Foster, K.~R., \& Morgan, P.~W. (1998).  
Photoperiod control of gibberellin levels and flowering in sorghum.  
\emph{Plant Physiology}, 116, 1003–1011.

\bibitem[Paul \emph{et~al.}(2020)]{Paul2020}
Paul, P., Dhatt, B.~K., Miller, M., \emph{et~al.} (2020).  
MADS78 and MADS79 are essential regulators of early seed development in rice.  
\emph{Plant Physiology}, 182, 933–948.

\bibitem[Jabir and Mahmoud(2021)]{Jabir2021}
Jabir, D.~A.~A., \& Mahmoud, M.~R. (2021).  
The effect of temperature stress associated with different planting dates and levels of gibberellic acid on the growth of sorghum spring.  
\emph{IOP Conference Series: Earth and Environmental Science}, 923, 012090.

\bibitem[Young and Britton(1989)]{Young1989}
Young, A.~J., \& Britton, G. (1989).
\newblock The distribution of alpha-carotene in the photosynthetic pigment-protein complexes of higher plants.
\newblock \emph{Plant Science}, 64, 179–183.

\bibitem[Vishwakarma \emph{et~al.}(2015)]{Vishwakarma2015}
Vishwakarma, A., Tetali, S.~D., Selinski, J., Scheibe, R., \& Padmasree, K. (2015).
\newblock Importance of the alternative oxidase (AOX) pathway in regulating cellular redox and ROS homeostasis to optimize photosynthesis during restriction of the cytochrome oxidase pathway in \emph{Arabidopsis thaliana}.
\newblock \emph{Annals of Botany}, 116(4), 553–566.
\newblock \doi{10.1093/aob/mcv066}

\bibitem[Gandin \emph{et~al.}(2014)]{Gandin2014}
Gandin, A., Dinakar, C., McDonald, A.~E., \& Vanlerberghe, G.~C. (2014).
\newblock Cooperation between the AOX pathway and nitrate assimilation to maintain optimal photosynthesis by regulating the accumulation of reducing equivalents.
\newblock \emph{Plant Physiology}.
  
\bibitem[Sayeed \emph{et~al.}(2016)]{Sayeed2016}
Sayeed, M.~S.~B., \emph{et~al.} (2016).  
Critical analysis on characterization, systemic effect, and therapeutic potential of beta‑sitosterol.  
\emph{Medicines}.

\bibitem[Mueller and Brown(1980)]{Mueller1980}
Mueller, S.~C., \& Brown, R.~M., Jr. (1980).  
Evidence for an intramembrane component associated with a cellulose microfibril synthesizing complex in higher plants.  
\emph{Journal of Cell Biology}, 84(3), 315–326.

\bibitem[Somerville(2006)]{Somerville2006}
Somerville, C. (2006).  
Cellulose synthesis in higher plants.  
\emph{Annual Review of Cell and Developmental Biology}, 22, 53–78.

\bibitem[Hu \emph{et~al.}(2018)]{Hu2018}
Hu, H., \emph{et~al.} (2018).  
Cellulose synthase mutants distinctively affect cell growth and cell wall integrity for plant biomass production in Arabidopsis.  
\emph{Plant Cell Physiology}, 59(6), 1142–1154.

\bibitem[Arioli \emph{et~al.}(1998)]{Arioli1998}
Arioli, T., \emph{et~al.} (1998).  
Molecular analysis of cellulose biosynthesis in Arabidopsis.  
\emph{Science}, 279, 717–720.

\bibitem[Persson \emph{et~al.}(2007)]{Persson2007}
Persson, S., \emph{et~al.} (2007).  
Genetic evidence for three unique components in primary cell‐wall cellulose synthase complexes.  
\emph{Proceedings of the National Academy of Sciences USA}, 104(39), 15566–15571.

\bibitem[Cano‐Delgado \emph{et~al.}(2003)]{CanoDelgado2003}
Cano‐Delgado, A.~I., \emph{et~al.} (2003).  
Reduced cellulose synthesis invokes lignification and defense responses in Arabidopsis thaliana.  
\emph{Plant Journal}, 34(4), 351–362.

\bibitem[Hernández‐Blanco \emph{et~al.}(2007)]{HernandezBlanco2007}
Hernández‐Blanco, C., \emph{et~al.} (2007).  
Impaired cellulose synthesis enhances disease resistance in Arabidopsis.  
\emph{Plant Cell}, 19(3), 890–903.

\bibitem[Tomlinson \emph{et~al.}(2004)]{Tomlinson2004}
Tomlinson, K., \emph{et~al.} (2004).  
Effects of inhibiting cellulose biosynthesis on nitrogen, phosphorus, and sulfur metabolism in Brassica napus.  
\emph{Plant Physiology}, 134(2), 568–577.

\bibitem[Ekman \emph{et~al.}(2008)]{Ekman2008}
Ekman, D., \emph{et~al.} (2008).  
Carbon partitioning during secondary wall biosynthesis in Arabidopsis stems.  
\emph{Plant Journal}, 53(3), 425–436.

\bibitem[Iyer \emph{et~al.}(2008)]{Iyer2008}
Iyer, P.~V.~V., \emph{et~al.} (2008).  
Alteration of cellulose and lignin in Arabidopsis via RNAi of cellulose synthase genes.  
\emph{Molecular Plant}, 1(2), 212–220.

\bibitem[Shi \emph{et~al.}(2012)]{Shi2012}
Shi, D., \emph{et~al.} (2012).  
Genetic analysis of Arabidopsis cellulose mutants reveals secondary cell wall defects.  
\emph{Plant Physiology}, 158(4), 1587–1595.

\bibitem[Tan \emph{et~al.}(2011)]{Tan2011}
Tan, J., \emph{et~al.} (2011).  
Enhancing seed protein content by down‐regulating cellulose synthesis in rice.  
\emph{Plant Biotechnology Journal}, 9(7), 834–842.

\bibitem[Yoshie‐Stark \emph{et~al.}(2008)]{YoshieStark2008}
Yoshie‐Stark, Y., \emph{et~al.} (2008).  
Manipulation of cell wall composition to increase seed protein content in maize.  
\emph{Journal of Agricultural Food Chemistry}, 56(11), 3981–3988.

\bibitem[Knowles(1983)]{Knowles1983}
Knowles, N.~R. (1983).  
Carbohydrate and protein accumulation during seed development in peas.  
\emph{Plant Physiology}, 72(1), 45–50.

\bibitem[Hu \emph{et~al.}(2020)]{Hu2020}
Hu, H., \emph{et~al.} (2020).  
Manipulating cellulose synthase for seed storage protein improvement.  
\emph{Plant Cell Reports}, 39(5), 607–619.


\bibitem{Yu2002}
Yu, B., Xu, C., \& Benning, C. (2002).  
\emph{Arabidopsis disrupted in SQD2 encoding sulfolipid synthase is impaired in phosphate-limited growth.}  
\textit{Proceedings of the National Academy of Sciences USA}, 99, 5732–5737.

\bibitem{Sun2021}
Sun, Y., Song, K., Liu, L., \emph{et al.} (2021).  
\emph{Sulfoquinovosyl diacylglycerol synthase 1 impairs glycolipid accumulation and photosynthesis in phosphate-deprived rice.}  
\textit{Journal of Experimental Botany}, 72(18), 6510–6523.

\bibitem{Qin2015}
Qin, X., Suga, M., Kuang, T., \& Shen, J. R. (2015).  
\emph{Structural basis for energy transfer pathways in the plant PSI-LHCI supercomplex.}  
\textit{Science}, 348, 989–995.

\bibitem{Umena2011}
Umena, Y., Kawakami, K., Shen, J. R., \& Kamiya, N. (2011).  
\emph{Crystal structure of oxygen-evolving photosystem II at 1.9 Å resolution.}  
\textit{Nature}, 473, 55–60.

\bibitem{YuBenning2003}
Yu, B., \& Benning, C. (2003).  
\emph{Anionic lipids are required for chloroplast structure and function in Arabidopsis.}  
\textit{The Plant Journal}, 36, 762–770.

\bibitem{Essigmann1998}
Essigmann, B., Güler, S., Narang, R. A., Linke, D., \& Benning, C. (1998).  
\emph{Phosphate availability affects thylakoid lipid composition and the expression of SQD1, a gene required for sulfolipid biosynthesis in Arabidopsis thaliana.}  
\textit{Proceedings of the National Academy of Sciences USA}, 95, 1950–1955.

\bibitem{Nakamura2013}
Nakamura, Y. (2013).  
\emph{Phosphate starvation and membrane lipid remodeling in seed plants.}  
\textit{Progress in Lipid Research}, 52, 43–50.

\bibitem{Yang2011}
Yang, Y., Yu, X., Song, L., \& An, C. (2011).  
\emph{ABI4 Activates DGAT1 Expression in Arabidopsis Seedlings during Nitrogen Deficiency.}  
\textit{Plant Physiology}, 156(2), 874–883.

\bibitem{Tan2018}
Tan, W.-J., Yang, Y.-C., Zhou, Y., Huang, L.-P., Xu, L., Chen, Q.-F., Yu, L.-J., \& Xiao, S. (2018).  
\emph{DIACYLGLYCEROL ACYLTRANSFERASE and DIACYLGLYCEROL KINASE Modulate Triacylglycerol and Phosphatidic Acid Production in the Plant Response to Freezing Stress.}  
\textit{Plant Physiology}, 177(4), 1304–1316.

\bibitem{Zhang2009}
Zhang, M., Fan, J., Taylor, D. C., \& Ohlrogge, J. B. (2009).  
\emph{DGAT1 and PDAT1 Acyltransferases Have Overlapping Functions in Arabidopsis Triacylglycerol Biosynthesis and Are Essential for Normal Pollen and Seed Development.}  
\textit{The Plant Cell}, 21(12), 3885–3901.

\end{thebibliography}

%==========================================
%   Start the Supplementary Material section
%==========================================
\FloatBarrier
\section*{Supplementary Material}
\beginsupplement


%========================================================
%  Supplementary Figure S1 – TIC traces
%========================================================
\begin{figure}[htp]
  \centering
  \includegraphics[width=\textwidth]{fig/supp/SuppFig1.png}
  \caption{
    Total ion current (TIC) traces for all injections in the lipidomics run. 
    {\bf(A)} Control samples (top panel). 
    {\bf(B)} Lowinput samples (bottom panel).
  }
  \label{fig:S1}
\end{figure}



%========================================================
%  Supplementary Figure S2 - SERFF RSD & PCA results
%========================================================
\begin{figure}[htp]
  \centering
  % Combined panel A (left) and B (right) in one image file
  \includegraphics[width=\textwidth]{fig/supp/SuppFig2.png}
  \caption{
    SERRF-normalized quality metrics across injections. 
    {\bf(A)} Control samples: per-feature RSD distributions (boxplot) and PCA of QC vs.\ biological samples. 
    {\bf(B)} Low-Input samples: same metrics after SERRF correction. 
    Red and green dots represents blank and QC respectively. Both panels demonstrate tight RSDs and clear separation of QC from biological samples in PC1/PC2.
  }
  \label{fig:S2}
\end{figure}








%========================================================
%  Supplementary Figure 3: Spatial Analysis 
%========================================================
\begin{figure}[htp]
  \centering
  \includegraphics[width=\textwidth]{fig/supp/SuppFig3.png}
  \caption{
    Spatial‐analysis diagnostics for the lipid TG(10:0/10:0/10:0) from the SpATS model. 
    {\bf(A)} Control: 3D spatial‐trend surface (row vs.\ column displacement). 
    {\bf(B)} Control diagnostics: (i) raw data, (ii) fitted values, (iii) residuals, (iv) fitted spatial trend, (v) genotypic BLUPs, (vi) histogram of BLUPs. 
    {\bf(C)} Low‐Input: 3D spatial‐trend surface. 
    {\bf(D)} Low‐Input diagnostics, as in (B).
  }
  \label{fig:S3}
\end{figure}





%\begin{figure}[htp]
%  \centering

  % ---------- row 1 ----------
  %\begin{subfigure}[t]{0.48\textwidth}
  %  \includegraphics[width=\linewidth]{fig/supp/SuppFig_3A_Lipid_Counts.png}
  %  \caption{Number of lipid \textit{species}.}
  %  \label{fig:S3A}
  %\end{subfigure}\hfill
  %\begin{subfigure}[t]{0.48\textwidth}
  %  \includegraphics[width=\linewidth]{fig/supp/SuppFig_3B_trad_nontrad_counts.png}
  %  \caption{Number of lipid \textit{classes}.}
  %  \label{fig:S3B}
  %\end{subfigure}

  %\vspace{1em}

  % ---------- row 2 (centred) ----------
  %\begin{subfigure}[t]{0.55\textwidth}
  %  \centering
  %  \includegraphics[width=\linewidth]%{fig/supp/SuppFig_3C_Lipid_Overlap_Venn_traditional.png}
    %\caption{Shared and unique lipid species.}
    %\label{fig:S3C}
  %\end{subfigure}

  %\caption{Overview of lipid coverage in Control and Low-Input samples.}
  %\label{fig:S3}
%\end{figure}



%========================================================
% Supplementary Figure S4 - Lipid species/class count   (panels A, B, C, and D)
%========================================================
\begin{figure}[htp]
  \centering
  \includegraphics[width=\textwidth]{fig/supp/SuppFig4.png}
  \caption{
    Summary of lipid coverage in Control vs.\ Lowinput runs. 
    {\bf(A)} Number of detected lipid species per major class.
    {\bf(B)} Breakdown of species counts within the two largest classes: Glycerolipid and Glyverophospholipid.
    {\bf(C)} Venn diagram of all lipid species: 184 species (∼58.2\%) are shared, while Control only (65, 20.6\%) and Lowiput only (67, 21.2\%) show a small number of unique detections in each run.
  }
  \label{fig:S4}
\end{figure}


%========================================================
% Supplementary Figure S5 - TIC - Class, Glycerolipid, Glycerophospholipid
%========================================================
\begin{figure}[htp]
  \centering
  \includegraphics[width=\textwidth]{fig/supp/SuppFig5.png}
  \caption{
    Distribution of total ion current (TIC) by lipid category in Control vs.\ Lowinput samples. 
    {\bf(A)} Major lipid classes as \% of TIC: glycerolipids dominate (~64 \% Control, ~52 \% Lowinput), followed by glycerophospholipids (~30 \% vs 25 \%). Sphingolipids sees the highest change.
    {\bf(B)} Breakdown of the glycerolipid pool: mono- and di-galactosyldiacylglycerols, diacylglycerols and triacylglycerols together account for most of the glycerolipid TIC, with nearly identical subclass proportions in both runs. 
    {\bf(C)} Breakdown of the glycerophospholipid pool: phosphatidylcholines (>79 \%), phosphatidylethanolamines (~18 \%), and minor head-groups (LPC, PS, etc.) together explain the glycerophospholipid TIC. The results are very consistent between Control and Lowinput.
  }
  \label{fig:S5}
\end{figure}



%========================================================
%  Supplementary Figure S4 - Lipid ratio contrasts under low-P
%========================================================
\begin{figure}[htp]
  \centering
  % Adjust width fraction as needed (e.g., 0.8\textwidth or \textwidth)
  \includegraphics[width=0.8\textwidth]{fig/supp/SuppFig_4_lipid_ratio_linear_lowP.png}
  \caption{$\Delta$Z-score contrasts for lipids under LI. 
    The panel shows violin+boxplots for metrics such as \textitt{SQDG-Spared}, \textitt{LPC-PC}, and \textitt{LPE-PE} under Control versus LowInput conditions. 
    Stars denote significance levels (***: $p<0.001$, **: $p<0.01$, *: $p<0.05$) from appropriate statistical tests. 
    A negative $\Delta$Z in SQDG\_Spared indicates sulfolipid is not upregulated relative to galactolipids and PG; 
    \textitt{LPC-PC} is not significantly changed, whereas \textitt{LPE-PE} and composite Lyso\_activity shift toward values consistent with selective PE deacylation.}
  \label{fig:S4_lipid_ratio_lowP}
\end{figure}

%========================================================
%  Supplementary Figure S5 - TIC Proportions for LPC and LPE
%========================================================
\begin{figure}[htp]
  \centering
  % Adjust width fraction as appropriate, e.g., 0.6\textwidth or \textwidth
  \includegraphics[width=0.7\textwidth]{fig/supp/SuppFig_5_TIC_LPC_LPE.png}
  \caption{Total ion current (TIC) proportions of lysophosphatidylcholine (LPC) and lysophosphatidylethanolamine (LPE) under Control and Low-P conditions. The plot displays relative TIC share of LPC versus LPE; stars denote significance levels (e.g., *: $p<0.05$, **: $p<0.01$) from appropriate tests. An increase in the LPE fraction and corresponding decrease in LPC under low-P suggests selective deacylation of PE for P salvage, while PC-derived LPC remains relatively stable.}
  \label{fig:S5_TIC_LPC_LPE}
\end{figure}



\end{document}


